{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Librerías\n",
    "from prody import *\n",
    "import csv\n",
    "from prody import * \n",
    "from Bio import pairwise2, SeqIO\n",
    "import os\n",
    "import glob"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Nombre de la proteina de interes\n",
    "prot_name = 'fxa'\n",
    "# Secuencia P28482 (ERK2_HUMAN)\n",
    "uniprot_id = \"P00742\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MGRPLHLVLLSASLAGLLLLGESLFIRREQANNILARVTRANSFLEEMKKGHLERECMEETCSYEEAREVFEDSDKTNEFWNKYKDGDQCETSPCQNQGKCKDGLGEYTCTCLEGFEGKNCELFTRKLCSLDNGDCDQFCHEEQNSVVCSCARGYTLADNGKACIPTGPYPCGKQTLERRKRSVAQATSSSGEAPDSITWKPYDAADLDPTENPFDLLDFNQTQPERGDNNLTRIVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTRGLPKAKSHAPEVITSSPLK\n"
     ]
    }
   ],
   "source": [
    "# Secuencia de la CDK2 de UniProt\n",
    "fasta = SeqIO.read(open(F'../data/{uniprot_id}.fasta'),'fasta')\n",
    "seq_prot = str(fasta.seq)\n",
    "print(seq_prot)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append(r'../.')\n",
    "from modules.find_gaps import find_gaps\n",
    "# Recibe una secuencia (con gaps como \"-\") y un valor r para valorar el gap en +/- r posiciones "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total de PDB IDs a modelar: 136\n"
     ]
    }
   ],
   "source": [
    "# Los IDs de los cristales están en el archivo\n",
    "import pandas as pd\n",
    "import glob\n",
    "file_path = glob.glob(F\"../data/TABLA_MTDATA_{prot_name.upper()}_*.json\")[0]\n",
    "df_prot = pd.read_json(file_path)\n",
    "pdbids_list = df_prot.set_index('PDB_ID').index\n",
    "print(\"Total de PDB IDs a modelar:\", len(pdbids_list))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Directiros y parámetros base\n",
    "struct_dir = F'../../FILES/CRYSTALS/PROT_{prot_name.upper()}_CHAINS/'\n",
    "tail_pdb = '_A.pdb' # extensión del archivo pdb del cristal\n",
    "tail_model = '_MODLL' # epiteto del archivo del modelo final, la extensión .pdb la agrega modeller\n",
    "pdbs_model_dir = F\"../../FILES/CRYSTALS/PROT_{prot_name.upper()}_MODELOS_modeller_NoPrep/\" # Carpeta de salida para los modelos\n",
    "\n",
    "# Creamos el directorio de salida si es que no existe\n",
    "import os\n",
    "if not os.path.exists(pdbs_model_dir):\n",
    "    os.makedirs(pdbs_model_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# FIND GAP FUNCTION WAS CORRECTED: find_gaps"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "from modeller import * # licencia modeller necesaria\n",
    "import shutil\n",
    "# vi /home/joel/anaconda3/envs/mds/lib/modeller-9.22/modlib/modeller/config.py\n",
    "# MODELIRANJE\n",
    "from modeller.automodel import *\n",
    "\n",
    "\n",
    "def run_modeller(pdb_code, seq_prot, num_res_window = 0, force_modeling = False,\n",
    "                max_var_iterations = 500, repeat_optimization = 2):\n",
    "    '''función para modelar gaps en estructuras cristalográficas utilizando Modeller'''\n",
    "    ###########################\n",
    "    if not force_modeling:\n",
    "        if os.path.isfile(pdbs_model_dir + pdb_code + tail_model + \".pdb\"):\n",
    "            return print(\"El modelo ya existe en el directorio\")\n",
    "    ###########################\n",
    "    try:\n",
    "        # Se lee el archivo pdb. La selección omite residuos negativos.\n",
    "        # Además omite los residuos no estandar, o con modificaciones postraduccionales como el TPO (Tirosina fosfatada)\n",
    "        stc_prot = parsePDB(struct_dir \n",
    "                            + pdb_code + tail_pdb).select('not nonstdaa and resid > 0') \n",
    "        chaid = stc_prot.getChids()[0] # Obtiene el id de la cadena\n",
    "        ref_hv = stc_prot.getHierView()[chaid] # Se obtiene sólo la cadena\n",
    "        seq_cry = ref_hv.getSequence() # Se obtiene la secuencia de la estructura cristalográfica \n",
    "        print(seq_cry)\n",
    "    except:\n",
    "        return print(\"Error al abrir y modelar:\", pdb_code)\n",
    "    ########################### \n",
    "    # Pregunta si la longitud y la secuencias de Uniprot y de la estructura son iguales\n",
    "    same_seq = len(seq_cry) == len(seq_prot) and seq_cry == seq_prot\n",
    "    \n",
    "    # Nombre de los cabezales de las secuencia deben coincidir con el de los archivos de entrada y salida\n",
    "    crys_file_name = pdb_code + tail_pdb\n",
    "    model_file_name = pdb_code + tail_model\n",
    "    \n",
    "    # si same_seq es verdadero, se omite hacer el modelado pues la estructura está completa\n",
    "    if same_seq:\n",
    "        shutil.copy(struct_dir + pdb_code + tail_pdb, \n",
    "                    pdbs_model_dir + pdb_code + tail_model + \".pdb\")\n",
    "        return print(\"La proteína \" + pdb_code + \" ya está completa\") \n",
    "        # Si le hacen falta átomos a un residuo fuera del gap, esto se corregirá \n",
    "        # con PDB2PQR y pdb4amber en un fase posterior al modelado.\n",
    "    ###########################     \n",
    "    print(\"Modelando proteína \" + pdb_code)\n",
    "    # Obtener la secueNcia de la estructura a modelar y guardarla en un archivo.\n",
    "    # Alineamiento global, se penalizan con -10 la abertura de nuevos gaps, se obtiene el mejor alineamiento.\n",
    "    alignment = pairwise2.align.globalxs(seq_cry, seq_prot, -10, -0.5, gap_char='-')[0]\n",
    "    # Secuencias del alineamiento\n",
    "    algn1_struc = alignment[0] # Secuencia alineada de la estructura cristalográfica\n",
    "    algn2_seq = seq_prot # Secuencia completa de UniProt\n",
    "\n",
    "    \n",
    "\n",
    "    ''' NECESARIO: There should be 10 fields separated by colons \":\".\n",
    "    Please check the file to make sure your sequences end with the '*' character.\n",
    "    Nomenclaturas de los campos del header: \n",
    "    https://salilab.org/modeller/8v2/manual/node176.html'''\n",
    "\n",
    "    # HEADERS (la secuencia del cristal va primero, luego la secuencia completa)\n",
    "    struc_header = \"structureX:\" + crys_file_name + \":.:\" + chaid + \":.:\" + chaid + \":.:.:.:\"\n",
    "    seq_header = \"sequence:\" + model_file_name + \":.:.:.:.:.:.:.:\"\n",
    "    \n",
    "    # Crea el archivo de alineamiento con la estructura requerida por modeller\n",
    "    alg_filename = pdb_code + \".alg\"\n",
    "    with open(alg_filename, \"w\") as handle:\n",
    "        handle.write(\"\\n>P1;%s\\n%s\\n%s*\\n>P1;%s\\n%s\\n%s*\\n\" % (crys_file_name, struc_header, algn1_struc, \n",
    "                                                               model_file_name, seq_header, algn2_seq))\n",
    "    ##########################\n",
    "    # Ejecuta find_gaps para obtener los posibles gaps de la secuencia\n",
    "    gaps = find_gaps(algn1_struc, r = num_res_window)\n",
    "    num_gaps = gaps[\"num_gaps\"]\n",
    "    gap_i = gaps[\"gap_window\"]\n",
    "    print(gaps)\n",
    "    #gap_i = [[gap - 1 for gap in gap_i[0]]]\n",
    "    \n",
    "    \n",
    "    # Se crea el string que necesita Modeller para definir al primer gap\n",
    "    s = \"self.residue_range('\" + str(gap_i[0][0]) + \"', '\" + str(gap_i[0][1]) + \"')\"\n",
    "\n",
    "    # A pesar de que sólo haya un gap, se ejecuta el ciclo, \n",
    "    # si hay más de uno, se extiende el String para incluir los demás\n",
    "    for i in range(1, num_gaps):\n",
    "        s = s + \", \" + \"self.residue_range('\" + str(gap_i[i][0]) + \"', '\" + str(gap_i[i][1]) + \"')\" #nótese la coma\n",
    "    ##########################\n",
    "    ''' SE EJECUTA MODELLER'''\n",
    "    ##########################\n",
    "    # Se crea un nuvo ambiente de modeller\n",
    "    env = environ()\n",
    "    env.io.atom_files_directory = ['.', struct_dir]\n",
    "    \n",
    "    # Se modifica la clase MyModel de Modeller para fijar los residuos ya existentes en el cristal\n",
    "    # Estos residuos no se modelarán ni sus átomos cambiarán de posición\n",
    "    # NECESARIO para poder capturar los valores obtenidos por el rango de gaps\n",
    "    \n",
    "    MyModel_code = \"\"\"\n",
    "class MyModel(automodel):\n",
    "    def select_atoms(self):\n",
    "        return selection(\"\"\" + s + \"\"\")\n",
    "\n",
    "    def special_patches(self, aln):\n",
    "\n",
    "        self.patch(residue_type='DISU', residues=(self.residues['7:'], self.residues['12:']))\n",
    "        self.patch(residue_type='DISU', residues=(self.residues['27:'], self.residues['43:']))\n",
    "        self.patch(residue_type='DISU', residues=(self.residues['156:'], self.residues['170:']))\n",
    "        self.patch(residue_type='DISU', residues=(self.residues['181:'], self.residues['209:']))\n",
    "\n",
    "        \n",
    "\"\"\" # Al estar dentro del loop la identación de este string es importante\n",
    "    exec(MyModel_code, globals()) # Se lleva a cabo la modificación a MyModel para agregar la región fijada\n",
    "    # Se intancia el objeto MyModel\n",
    "    a = MyModel(env, alnfile = alg_filename, # Lee el archivo fasta creado y guardado en el directorio actual\n",
    "                      knowns = crys_file_name, # Archivo pdb crys, que coincide con el id en el archivo fasta\n",
    "                      sequence = model_file_name) # Nombre del modelo\n",
    "    a.starting_model= 1\n",
    "    a.ending_model  = 1\n",
    "    # Se definen los parámetros de refinamiento dle modelo:\n",
    "    # Más info sobre el refinamiento: https://salilab.org/modeller/9.21/manual/node19.html\n",
    "    a.library_schedule = autosched.slow # Originalmente comentado\n",
    "    a.max_var_iterations = max_var_iterations #*500\n",
    "    a.md_level = refine.slow # Nivel del refinamiento\n",
    "    \n",
    "    a.repeat_optimization = repeat_optimization # Número de repeticiones para la optimización\n",
    "    \n",
    "    a.make()\n",
    "\n",
    "    ###########################\n",
    "    # Renombra el archivo pdb a pdb_code + _full.pdb\n",
    "    # DEBE ser el único archivo pdb en el directorio '.'\n",
    "    model_file = glob.glob('./' + pdb_code + '*.pdb')[0] # Nombre del archivo pdb\n",
    "    \n",
    "    os.rename(model_file, pdbs_model_dir + pdb_code + tail_model + \".pdb\") # Mueve el pdb a la carpeta model_pdbs\n",
    "    # Elimina los archivos extra generados - Todos empiezan con el codigo del PDB ID\n",
    "    for f in glob.glob(pdb_code + \"*\"):\n",
    "        os.remove(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "seq_prot = 'IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTRGLPKAK'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "241"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(seq_prot)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "El modelo ya existe en el directorio\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1ezq\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26343    24640\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.0448\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24640     377\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       90\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          34.0182\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.84826E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.669   1.669      4.8290       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  47.249  47.249      11.608       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.549   1.549      1.4895       1.000\n",
      " 5 Soft-sphere overlap restraints     :      90       0      0   0.001   0.001     0.23128E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.103   0.103     0.27067       1.000\n",
      "10 Distance restraints 2 (N-O)        :      44       0      0   0.187   0.187     0.96276       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.064   3.064     0.99658       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  76.882  76.882     0.67215       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  60.466  60.466      1.0312       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  82.634  82.634      2.7435       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  85.294  85.294      3.0740       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      23       0      0   0.594   0.594     0.14429       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  29.809 113.344      6.0889       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1ezq_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1352.4152\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -119.78 -125.20   11.26    0.62  -63.00  177.74   27.93\n",
      "    1        235R 235R N   CA   1853 1854  150.47  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -133.60 -167.20   75.37    2.32   82.20  174.70   13.65\n",
      "    2        236G 236G N   CA   1864 1865  107.13  174.60                    8.50\n",
      "    3   6833 238P 239K C   N    1881 1883  -81.07  -70.20   23.80    1.94  -62.90  161.06   19.99\n",
      "    3        239K 239K N   CA   1883 1884  119.23  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -81.11  -68.20   18.16    1.15  -62.50  162.10   27.52\n",
      "    4        240A 240A N   CA   1892 1893  158.08  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    3    8   24   39   74   89  112  144  221  239  273\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1ezq_MODLL.B99990001.pdb        34.01815\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1f0r\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26312    24609\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24609     380\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      108\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          36.6424\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.15719       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.564   1.564      4.6134       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.215  49.215      12.049       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.381   1.381      1.0034       1.000\n",
      " 5 Soft-sphere overlap restraints     :     108       0      0   0.000   0.000     0.22016E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.118   0.118     0.43547       1.000\n",
      "10 Distance restraints 2 (N-O)        :      44       0      0   0.212   0.212      1.4108       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.836   2.836     0.85390       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  78.234  78.234      1.9116       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  66.553  66.553     0.45558       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      1  83.905  83.905      3.8744       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0   6.649   6.649      1.3864       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.377   0.377     0.83588E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  23.880 127.114      8.3421       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   0.688   0.688     0.62942E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1f0r_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1460.5486\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -111.91 -125.20   28.84    1.59  -63.00  160.35   25.04\n",
      "    1        235R 235R N   CA   1853 1854  166.19  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   86.04   78.70    7.97    0.45   82.20  171.54    8.13\n",
      "    2        236G 236G N   CA   1864 1865 -163.00 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -80.79  -70.70   16.03    1.57  -63.50  171.22   22.84\n",
      "    3        237L 237L N   CA   1868 1869  129.14  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -80.93  -70.20   19.25    1.17  -62.90  163.81   22.30\n",
      "    4        239K 239K N   CA   1883 1884  156.39  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -69.31  -68.20    3.78    0.27  -62.50  170.33   28.25\n",
      "    5        240A 240A N   CA   1892 1893  148.91  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1267 1286   2.266\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    2   11   14   41   72   94  117  160  185  256  268\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1f0r_MODLL.B99990001.pdb        36.64244\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1f0s\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26578    24875\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2295 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.4271\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24875     389\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          59.1386\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.65869E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.780   1.780      6.0904       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  50.310  50.310      13.989       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       1      1   3.217   3.217      7.5189       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.001   0.001     0.13479E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.292   0.292      3.4330       1.000\n",
      "10 Distance restraints 2 (N-O)        :      51       0      0   0.292   0.292      2.6493       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.456   4.456      2.1079       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  59.838  59.838      2.1370       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  77.812  77.812     0.55278       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  73.651  73.651      1.7546       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 104.831 104.831      2.2410       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.836   0.836     0.35976       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  28.346 116.009      16.226       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1f0s_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1458.5183\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6574 232M 232M C   CA   1835 1830   13.72    0.00   13.72    4.83    0.00   13.72    4.83\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864  106.88   78.70   61.76    0.92   82.20  132.76    7.66\n",
      "    1        236G 236G N   CA   1864 1865  138.95 -166.10                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868 -118.85 -108.50   14.08    0.63  -63.50 -174.78   29.13\n",
      "    2        237L 237L N   CA   1868 1869  142.04  132.50                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -70.17  -70.20    5.47    0.40  -62.90  175.88   22.45\n",
      "    3        239K 239K N   CA   1883 1884  134.93  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892 -143.10 -134.00    9.53    0.23  -62.50 -172.49   35.05\n",
      "    4        240A 240A N   CA   1892 1893  149.80  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3   14   22   41   68   96  111  165  167  266  277\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1f0s_MODLL.B99990001.pdb        59.13857\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTAAEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1fax\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26400    24697\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24697     393\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       89\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          52.4386\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.11082       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.293   2.293      8.2015       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      2  56.962  56.962      15.752       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.209   1.209     0.49352       1.000\n",
      " 5 Soft-sphere overlap restraints     :      89       0      0   0.002   0.002     0.33207E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.137   0.137     0.84692       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.226   0.226      1.1930       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.062   2.062     0.40097       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  65.158  65.158    -0.24139       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  40.831  40.831     0.80685E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  81.558  81.558      2.3139       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  53.013  53.013      1.2826       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      32       0      0   0.212   0.212     0.50740E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      2  49.368 133.607      21.057       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      34       0      0   0.696   0.696     0.86399       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1fax_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1354.4819\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6828 233K 234T C   N    1844 1846  117.98   55.90   93.33   10.84   55.90   93.33   10.84\n",
      "    1        234T 234T N   CA   1846 1847  109.19   39.50                   39.50\n",
      "    2   6831 236G 237L C   N    1866 1868 -125.88 -108.50   20.80    0.92  -63.50 -174.33   29.64\n",
      "    2        237L 237L N   CA   1868 1869  143.92  132.50                  -41.20\n",
      "    3   6832 237L 238P C   N    1874 1876  -53.62  -58.70    5.82    0.54  -64.50  179.80   14.00\n",
      "    3        238P 238P N   CA   1876 1877  -33.33  -30.50                  147.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -70.66  -70.20    4.66    0.33  -62.90  174.34   23.07\n",
      "    4        239K 239K N   CA   1883 1884  145.04  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -76.93  -68.20   16.88    1.09  -62.50  160.01   26.95\n",
      "    5        240A 240A N   CA   1892 1893  159.75  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2    8   35  118   94  121  171  173  219  287\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1fax_MODLL.B99990001.pdb        52.43864\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1fjs\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26618    24915\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1856 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24915     402\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      101\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          44.4125\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.24636       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.694   1.694      5.2915       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  45.810  45.810      11.427       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.859   1.859      2.2134       1.000\n",
      " 5 Soft-sphere overlap restraints     :     101       0      0   0.003   0.003     0.12593       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      48       0      0   0.210   0.210      2.1512       1.000\n",
      "10 Distance restraints 2 (N-O)        :      35       0      0   0.404   0.404      2.8298       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.536   2.536     0.68259       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0   8.448   8.448     0.12249       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  71.024  71.024      1.5238       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  79.105  79.105      3.4034       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  86.997  86.997      2.5078       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.325   0.325     0.73796E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      3  24.828 136.881      10.535       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      27       0      0   0.633   0.633      1.2789       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1fjs_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1444.0736\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -125.16 -125.20   47.21    2.28  -63.00  145.08   23.84\n",
      "    1        235R 235R N   CA   1853 1854 -172.19  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -69.55  -80.20   18.34    0.47   82.20 -146.16    6.72\n",
      "    2        236G 236G N   CA   1864 1865  159.17  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -68.69  -70.70    2.40    0.17  -63.50  178.59   25.10\n",
      "    3        237L 237L N   CA   1868 1869  140.28  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -60.04  -70.20   10.27    0.71  -62.90  179.69   23.49\n",
      "    4        239K 239K N   CA   1883 1884  138.86  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -64.44  -68.20    4.32    0.42  -62.50  171.67   28.23\n",
      "    5        240A 240A N   CA   1892 1893  147.45  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    7   13   39   76   92  149  134  209  259  310\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1fjs_MODLL.B99990001.pdb        44.41251\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1g2l\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26575    24872\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1856 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.0705\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24872     369\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       69\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          82.9452\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.004   0.004     0.31924       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.014   2.014      6.4022       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  54.749  54.749      14.069       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.453   1.453     0.95462       1.000\n",
      " 5 Soft-sphere overlap restraints     :      69       0      0   0.002   0.002     0.37661E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.153   0.153      1.0902       1.000\n",
      "10 Distance restraints 2 (N-O)        :      48       0      0   0.240   0.240      1.7179       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   1.879   1.879     0.33304       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  52.270  52.270      1.2625       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  68.449  68.449     0.86280       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  57.285  57.285      1.6799       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  95.441  95.441      1.2716       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.515   0.515     0.12599       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       4      2  42.841 127.957      52.326       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      15       0      0   0.836   0.836     0.49222       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1g2l_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1067.1836\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -141.62  -62.90  103.62   12.96  -62.90  103.62   12.96\n",
      "    1        233K 233K N   CA   1837 1838   26.59  -40.80                  -40.80\n",
      "    2   6830 235R 236G C   N    1862 1864 -106.92  -80.20   32.48    1.32   82.20 -114.73    7.55\n",
      "    2        236G 236G N   CA   1864 1865 -167.44  174.10                    8.50\n",
      "    3   6833 238P 239K C   N    1881 1883  -67.44  -70.20    4.67    0.39  -62.90  175.09   22.99\n",
      "    3        239K 239K N   CA   1883 1884  144.17  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -73.51  -68.20   11.98    0.79  -62.50  163.43   27.34\n",
      "    4        240A 240A N   CA   1892 1893  156.04  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3   11   19   44   70  112  123  158  224  242  269\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1g2l_MODLL.B99990001.pdb        82.94515\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1g2m\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26350    24647\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1863 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.1442\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.2560\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24647     394\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       86\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          55.7327\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.004   0.004     0.28427       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.951   1.951      6.3621       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      2  52.846  52.846      13.006       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.145   1.145     0.56984       1.000\n",
      " 5 Soft-sphere overlap restraints     :      86       0      0   0.000   0.000     0.27316E-03   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.184   0.184      1.6410       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.459   0.459      4.9738       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   1.773   1.773     0.29641       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  46.865  46.865     0.18078       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  77.447  77.447      1.2436       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  29.505  29.505      1.7044       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 109.653 109.653      2.3413       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      31       0      0   0.293   0.293     0.48551E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       3      2  29.581 105.743      22.918       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      35       0      0   0.289   0.289     0.16200       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1g2m_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1225.7495\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868  -73.11  -70.70   27.57    2.00  -63.50  150.04   21.40\n",
      "    1        237L 237L N   CA   1868 1869  169.07  141.60                  -41.20\n",
      "    2   6833 238P 239K C   N    1881 1883  -87.96  -70.20   18.14    1.24  -62.90  176.91   24.42\n",
      "    2        239K 239K N   CA   1883 1884  144.07  140.40                  -40.80\n",
      "    3   6834 239K 240A C   N    1890 1892  -68.28  -68.20    1.46    0.11  -62.50  172.44   28.55\n",
      "    3        240A 240A N   CA   1892 1893  146.76  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    4   14   44   73  103  139  142  228  204  274\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1g2m_MODLL.B99990001.pdb        55.73272\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTRGLPKAK\n",
      "Modelando proteína 1hcg\n",
      "{'num_gaps': 1, 'gap_lengths': [5], 'gap_list': [[135, 139]], 'gap_window': [[133, 141]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26703    25000\n",
      "72 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      72\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25000     657\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      142\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          81.2498\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      74       0      0   0.004   0.004     0.29986       1.000\n",
      " 2 Bond angle potential               :     100       0      0   1.590   1.590      4.6658       1.000\n",
      " 3 Stereochemical cosine torsion poten:      41       0      4  58.977  58.977      22.716       1.000\n",
      " 4 Stereochemical improper torsion pot:      30       0      0   1.612   1.612      2.3725       1.000\n",
      " 5 Soft-sphere overlap restraints     :     142       0      0   0.002   0.002     0.40966E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     123       0      0   0.374   0.374      4.1398       1.000\n",
      "10 Distance restraints 2 (N-O)        :     120       0      2   0.553   0.553      13.435       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      10       0      0   5.272   5.272      2.4361       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  57.558  57.558      1.6695       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  84.732  84.732      1.9873       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0 126.820 126.820      4.2788       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  22.719  22.719      1.4591       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      72       0      0   0.895   0.895      1.2119       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       5      4  30.644 120.306      18.910       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      57       0      0   0.588   0.588      1.6276       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1hcg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1696.3763\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6729 134H 135E C   N    1059 1061 -114.05 -117.80    3.79    0.13  -63.60 -176.39   22.11\n",
      "    1        135E 135E N   CA   1061 1062  136.24  136.80                  -40.30\n",
      "    2   6730 135E 136K C   N    1068 1070 -120.38 -118.00   21.00    0.99  -62.90  169.29   25.42\n",
      "    2        136K 136K N   CA   1070 1071  159.97  139.10                  -40.80\n",
      "    3   6732 137G 138R C   N    1081 1083 -139.66 -125.20   15.74    0.49  -63.00 -171.64   30.69\n",
      "    3        138R 138R N   CA   1083 1084  146.84  140.60                  -41.10\n",
      "    4   6733 138R 139Q C   N    1092 1094  -98.55 -121.10   24.80    0.77  -63.80  173.21   23.78\n",
      "    4        139Q 139Q N   CA   1094 1095  129.38  139.70                  -40.30\n",
      "    5   6734 139Q 140S C   N    1101 1103 -130.61 -136.60   45.27    2.17  -64.10  156.19    9.84\n",
      "    5        140S 140S N   CA   1103 1104  106.32  151.20                  -35.00\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    65  824   1.996\n",
      "  serious non-bonded atom clash:   459  461   2.153\n",
      "  serious non-bonded atom clash:   488  498   2.245\n",
      "  serious non-bonded atom clash:   559 1860   2.030\n",
      "  serious non-bonded atom clash:   887  925   2.288\n",
      "  serious non-bonded atom clash:   889  925   2.100\n",
      "  serious non-bonded atom clash:  1508 1535   1.953\n",
      "  serious non-bonded atom clash:  1614 1704   2.254\n",
      "  serious non-bonded atom clash:  1822 1846   2.235\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    2    4    4    9   13   39   36   78  100  131  168  203  260  274\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1hcg_MODLL.B99990001.pdb        81.24976\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1ioe\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26672    24969\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24969     395\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       90\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          59.3193\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.005   0.005     0.40055       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.251   2.251      8.4222       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  59.653  59.653      16.047       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.474   1.474      1.1577       1.000\n",
      " 5 Soft-sphere overlap restraints     :      90       0      0   0.014   0.014      1.9835       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      49       0      0   0.190   0.190      1.6029       1.000\n",
      "10 Distance restraints 2 (N-O)        :      49       0      0   0.367   0.367      4.1048       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   5.111   5.111      2.4649       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0   8.936   8.936     0.15520       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  83.125  83.125      1.7207       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  20.766  20.766      1.3753       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  41.003  41.003      1.6106       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      36       0      0   0.335   0.335     0.89478E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      2  27.875 147.708      18.048       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      27       0      0   0.327   0.327     0.13721       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1ioe_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1333.8763\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6828 233K 234T C   N    1844 1846   70.23   55.90   36.29    3.61 -124.80  179.46   10.73\n",
      "    1        234T 234T N   CA   1846 1847   72.83   39.50                  143.50\n",
      "    2   6830 235R 236G C   N    1862 1864  -86.90  -80.20   41.20    1.30   82.20 -131.45   17.66\n",
      "    2        236G 236G N   CA   1864 1865 -145.25  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -64.35  -70.70   12.85    0.80  -63.50  171.63   23.78\n",
      "    3        237L 237L N   CA   1868 1869  130.42  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -58.35  -70.20   13.27    0.83  -62.90  175.30   23.02\n",
      "    4        239K 239K N   CA   1883 1884  134.44  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -74.21  -68.20   11.12    0.71  -62.50  164.85   27.61\n",
      "    5        240A 240A N   CA   1892 1893  154.66  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    3   12   24   35   63   88  124  155  192  219  265  296\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1ioe_MODLL.B99990001.pdb        59.31935\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqe\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26705    25002\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25002     376\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       93\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          51.7281\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.21064       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.269   2.269      8.2829       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  53.985  53.985      14.704       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.199   1.199     0.63425       1.000\n",
      " 5 Soft-sphere overlap restraints     :      93       0      0   0.004   0.004     0.16690       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.309   0.309      3.8234       1.000\n",
      "10 Distance restraints 2 (N-O)        :      43       0      0   0.381   0.381      3.0162       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   3.480   3.480      1.1428       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  46.346  46.346     0.15718       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0   4.197   4.197    -0.46419       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  54.665  54.665     0.96493       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0   3.280   3.280      1.0262       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      34       0      0   0.310   0.310     0.10513       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      2  31.861 142.618      16.341       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      20       0      0   1.308   1.308      1.6165       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqe_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1221.8816\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6828 233K 234T C   N    1844 1846   55.26   55.90   50.39    3.48 -124.80 -172.25   10.98\n",
      "    1        234T 234T N   CA   1846 1847   89.89   39.50                  143.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -66.50  -70.70    6.09    0.39  -63.50  178.41   24.60\n",
      "    2        237L 237L N   CA   1868 1869  137.19  141.60                  -41.20\n",
      "    3   6832 237L 238P C   N    1874 1876  -69.90  -58.70   13.61    1.15  -64.50  170.05   12.98\n",
      "    3        238P 238P N   CA   1876 1877  -22.77  -30.50                  147.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -81.24  -70.20   11.14    0.78  -62.90  178.23   24.19\n",
      "    4        239K 239K N   CA   1883 1884  141.91  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -69.35  -68.20    1.92    0.12  -62.50  172.40   28.59\n",
      "    5        240A 240A N   CA   1892 1893  146.84  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   150  169   2.293\n",
      "  serious non-bonded atom clash:   314 1458   2.101\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    1    4    9   14   35   55   82  130  165  170  230  269  297\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqe_MODLL.B99990001.pdb        51.72806\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqf\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26840    25137\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25137     393\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       78\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          67.4972\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.004   0.004     0.34640       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.152   2.152      7.7364       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      4  58.671  58.671      17.055       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.089   1.089     0.42870       1.000\n",
      " 5 Soft-sphere overlap restraints     :      78       0      1   0.021   0.021      4.1535       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      50       0      0   0.307   0.307      4.4938       1.000\n",
      "10 Distance restraints 2 (N-O)        :      52       0      0   0.392   0.392      4.6595       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.350   2.350     0.52088       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0   6.623   6.623    -0.55366       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      1  23.064  23.064      1.4626       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  87.600  87.600      4.3445       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  56.850  56.850      1.1869       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      32       0      0   0.319   0.319     0.67533E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       4      2  30.015 122.800      21.489       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      25       0      0   0.284   0.284     0.10574       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqf_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1138.5784\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6828 233K 234T C   N    1844 1846   65.92   55.90   45.96    3.97 -124.80  179.31   10.59\n",
      "    1        234T 234T N   CA   1846 1847   84.36   39.50                  143.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -66.42  -70.70   26.62    1.86  -63.50  156.56   21.57\n",
      "    2        237L 237L N   CA   1868 1869  115.33  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -90.46  -70.20   28.92    2.42  -62.90  162.92   19.81\n",
      "    3        239K 239K N   CA   1883 1884  119.77  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892 -109.17 -134.00   26.04    0.62  -62.50 -174.09   32.93\n",
      "    4        240A 240A N   CA   1892 1893  139.15  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   506  511   2.285\n",
      "  serious non-bonded atom clash:  1377 1380   2.297\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    2   13   25   44   80  117  131  146  192  219  251  338\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqf_MODLL.B99990001.pdb        67.49720\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqg\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26742    25039\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25039     408\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       82\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          60.1904\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.002   0.002     0.11434       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.803   1.803      5.1633       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  58.406  58.406      14.811       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.229   1.229     0.73937       1.000\n",
      " 5 Soft-sphere overlap restraints     :      82       0      0   0.011   0.011      1.2039       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      48       0      0   0.171   0.171      1.5096       1.000\n",
      "10 Distance restraints 2 (N-O)        :      55       0      0   0.275   0.275      2.8222       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.026   2.026     0.38722       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  48.573  48.573    -0.10744       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  74.838  74.838      1.0903       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  99.442  99.442      3.1651       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  78.615  78.615      1.9729       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      31       0      0   0.219   0.219     0.28864E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      2  34.765 129.059      26.829       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      40       0      0   0.487   0.487     0.46117       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1397.2178\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -107.29  -62.90   66.53    7.95  -62.90   66.53    7.95\n",
      "    1        233K 233K N   CA   1837 1838    8.76  -40.80                  -40.80\n",
      "    2   6830 235R 236G C   N    1862 1864 -129.42 -167.20   48.26    0.94   82.20 -158.67   15.55\n",
      "    2        236G 236G N   CA   1864 1865  144.57  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -80.24  -70.70   19.34    1.20  -63.50  161.25   23.38\n",
      "    3        237L 237L N   CA   1868 1869  158.42  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -100.19 -118.00   18.17    0.75  -62.90 -179.57   25.61\n",
      "    4        239K 239K N   CA   1883 1884  142.67  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -73.73  -68.20    9.71    0.62  -62.50  166.20   27.80\n",
      "    5        240A 240A N   CA   1892 1893  153.28  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    7   14   27   44   80  132  143  164  229  241  317\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqg_MODLL.B99990001.pdb        60.19040\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqh\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26712    25009\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25009     405\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       89\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          79.8656\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.002   0.002     0.12079       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.935   1.935      6.2784       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  54.969  54.969      14.537       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.081   1.081     0.53394       1.000\n",
      " 5 Soft-sphere overlap restraints     :      89       0      0   0.000   0.000     0.13889E-03   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      50       0      0   0.144   0.144     0.81012       1.000\n",
      "10 Distance restraints 2 (N-O)        :      60       0      0   0.280   0.280      2.9010       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.462   2.462     0.57164       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  41.455  41.455     0.13201       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  52.075  52.075    -0.50483       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  60.012  60.012      1.1382       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  92.810  92.810      2.8101       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      31       0      0   0.371   0.371     0.78930E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       6      2  38.519 152.097      49.509       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      30       0      0   0.824   0.824     0.94957       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqh_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1264.5864\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -132.44  -62.90   82.45   10.92  -62.90   82.45   10.92\n",
      "    1        233K 233K N   CA   1837 1838    3.48  -40.80                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846   60.20   55.90   54.32    4.12 -124.80 -178.04   10.61\n",
      "    2        234T 234T N   CA   1846 1847   93.65   39.50                  143.50\n",
      "    3   6830 235R 236G C   N    1862 1864  -86.36  -80.20   13.78    0.32   82.20 -114.91    7.45\n",
      "    3        236G 236G N   CA   1864 1865 -173.58  174.10                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868 -100.02 -108.50   28.23    1.59  -63.50  163.51   24.93\n",
      "    4        237L 237L N   CA   1868 1869  159.42  132.50                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883  -69.09  -70.20    6.22    0.42  -62.90  175.19   22.42\n",
      "    5        239K 239K N   CA   1883 1884  134.28  140.40                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -79.25  -68.20   15.54    0.98  -62.50  163.74   27.69\n",
      "    6        240A 240A N   CA   1892 1893  156.22  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2   11   18   31   75   93  117  170  189  188  266  304\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqh_MODLL.B99990001.pdb        79.86558\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqi\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26650    24947\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24947     395\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          68.1352\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.005   0.005     0.44455       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.414   2.414      9.3485       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  57.635  57.635      16.353       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   0.909   0.909     0.32082       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.003   0.003     0.76337E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.295   0.295      3.8463       1.000\n",
      "10 Distance restraints 2 (N-O)        :      50       0      0   0.511   0.511      5.8296       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.437   2.437     0.56029       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  58.843  58.843      1.0887       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  60.785  60.785     0.63401       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      1  93.164  93.164      3.5865       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  92.406  92.406      2.4094       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      32       0      0   0.252   0.252     0.52133E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      2  31.613 140.952      19.446       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      32       0      0   1.662   1.662      4.1392       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqi_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1421.6903\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6828 233K 234T C   N    1844 1846   61.35   55.90   45.07    3.56 -124.80 -176.33   10.83\n",
      "    1        234T 234T N   CA   1846 1847   84.24   39.50                  143.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -60.35  -70.70   16.93    1.05  -63.50  169.43   23.71\n",
      "    2        237L 237L N   CA   1868 1869  128.20  141.60                  -41.20\n",
      "    3   6832 237L 238P C   N    1874 1876  -55.35  -58.70    6.61    0.81  -64.50  172.25   12.38\n",
      "    3        238P 238P N   CA   1876 1877  -24.80  -30.50                  147.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -104.99 -118.00   14.28    0.65  -62.90  179.24   25.76\n",
      "    4        239K 239K N   CA   1883 1884  144.98  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -69.23  -68.20    2.99    0.21  -62.50  171.13   28.38\n",
      "    5        240A 240A N   CA   1892 1893  148.10  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    5   23   41   53   84  126  145  183  200  237  321\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqi_MODLL.B99990001.pdb        68.13525\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqj\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26738    25035\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25035     401\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       75\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          95.6631\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.21313       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.943   1.943      6.1584       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      2  54.638  54.638      14.921       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.471   1.471     0.90565       1.000\n",
      " 5 Soft-sphere overlap restraints     :      75       0      0   0.004   0.004     0.11085       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      52       0      0   0.313   0.313      4.5783       1.000\n",
      "10 Distance restraints 2 (N-O)        :      57       0      0   0.364   0.364      4.1569       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.097   2.097     0.41506       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  64.109  64.109     0.97941       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  56.707  56.707      1.7257       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  61.949  61.949      1.5411       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  44.824  44.824      1.9929       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.367   0.367     0.81498E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       6      2  43.526 145.300      57.582       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      32       0      0   0.376   0.376     0.30087       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqj_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1466.6794\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -155.16  -62.90  100.99   14.26  -62.90  100.99   14.26\n",
      "    1        233K 233K N   CA   1837 1838    0.26  -40.80                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846   57.41   55.90   51.55    3.72 -124.80 -174.63   10.83\n",
      "    2        234T 234T N   CA   1846 1847   91.02   39.50                  143.50\n",
      "    3   6830 235R 236G C   N    1862 1864 -175.91 -167.20   16.73    0.67   82.20 -177.17   13.30\n",
      "    3        236G 236G N   CA   1864 1865  160.32  174.60                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868  -71.43  -70.70    0.74    0.06  -63.50  177.30   25.08\n",
      "    4        237L 237L N   CA   1868 1869  141.68  141.60                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883 -108.96 -118.00   34.12    1.75  -62.90  154.23   22.77\n",
      "    5        239K 239K N   CA   1883 1884  172.01  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892 -118.75 -134.00   17.74    0.42  -62.50 -172.53   28.54\n",
      "    6        240A 240A N   CA   1892 1893  137.93  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1060 1070   2.171\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    6    6   24   42   64   84  122  166  194  194  252  289\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqj_MODLL.B99990001.pdb        95.66308\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqk\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26698    24995\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       93.3249\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24995     414\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       89\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          93.7637\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.19253       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.219   2.219      7.7444       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      2  53.215  53.215      14.282       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.483   1.483     0.95185       1.000\n",
      " 5 Soft-sphere overlap restraints     :      89       0      0   0.001   0.001     0.15636E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      53       0      0   0.228   0.228      2.8652       1.000\n",
      "10 Distance restraints 2 (N-O)        :      59       0      0   0.345   0.345      4.4847       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.759   2.759     0.71805       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  84.958  84.958     0.97465       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  52.468  52.468     0.46800       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  73.618  73.618      2.5128       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  50.001  50.001      1.4783       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.306   0.306     0.55441E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       7      2  42.780 159.741      56.496       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      40       0      0   0.431   0.431     0.52330       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqk_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1637.2405\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -150.76  -62.90  101.75   13.71  -62.90  101.75   13.71\n",
      "    1        233K 233K N   CA   1837 1838   10.52  -40.80                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846   53.01   55.90   55.50    3.67 -124.80 -175.68    8.42\n",
      "    2        234T 234T N   CA   1846 1847   94.92   39.50                  143.50\n",
      "    3   6830 235R 236G C   N    1862 1864 -168.08 -167.20    3.24    0.13   82.20 -163.53   14.30\n",
      "    3        236G 236G N   CA   1864 1865  171.48  174.60                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868  -75.08  -70.70   11.16    0.71  -63.50  167.34   23.92\n",
      "    4        237L 237L N   CA   1868 1869  151.86  141.60                  -41.20\n",
      "    5   6832 237L 238P C   N    1874 1876  -64.54  -58.70    7.33    0.59  -64.50  173.27   12.93\n",
      "    5        238P 238P N   CA   1876 1877  -26.07  -30.50                  147.20\n",
      "    6   6833 238P 239K C   N    1881 1883  -77.60  -70.20    9.03    0.55  -62.90  174.25   23.46\n",
      "    6        239K 239K N   CA   1883 1884  145.57  140.40                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892 -107.15 -134.00   27.00    0.79  -62.50 -179.42   31.95\n",
      "    7        240A 240A N   CA   1892 1893  144.13  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   506  511   2.211\n",
      "  serious non-bonded atom clash:  1267 1286   2.289\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    3    5   19   39   57   78  119  144  194  208  242  311\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqk_MODLL.B99990001.pdb        93.76369\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iql\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26425    24722\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24722     378\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       85\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          77.1951\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.14309       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.958   1.958      6.2469       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  61.127  61.127      15.584       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.412   1.412     0.80647       1.000\n",
      " 5 Soft-sphere overlap restraints     :      85       0      0   0.001   0.001     0.11141E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      49       0      0   0.234   0.234      1.8206       1.000\n",
      "10 Distance restraints 2 (N-O)        :      54       0      0   0.377   0.377      4.0054       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   3.301   3.301      1.0278       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  46.974  46.974    -0.11603E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  84.600  84.600      2.0305       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  87.918  87.918      3.9334       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  81.445  81.445      2.1385       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      30       0      0   0.487   0.487     0.13439       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       6      3  40.007 140.053      39.167       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      11       0      0   0.494   0.494     0.15724       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iql_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1490.8479\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -121.03  -62.90   77.46    9.63  -62.90   77.46    9.63\n",
      "    1        233K 233K N   CA   1837 1838   10.39  -40.80                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846   61.05   55.90   36.74    2.96 -124.80 -173.18   11.11\n",
      "    2        234T 234T N   CA   1846 1847   75.88   39.50                  143.50\n",
      "    3   6830 235R 236G C   N    1862 1864  -64.46  -62.40    2.66    0.39   82.20  154.33   11.70\n",
      "    3        236G 236G N   CA   1864 1865  -39.52  -41.20                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868  -58.56  -70.70   17.79    1.13  -63.50  169.87   23.87\n",
      "    4        237L 237L N   CA   1868 1869  128.59  141.60                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883  -79.55  -70.20   17.01    1.03  -62.90  165.44   22.43\n",
      "    5        239K 239K N   CA   1883 1884  154.60  140.40                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -61.02  -68.20    7.72    0.55  -62.50  176.62   28.87\n",
      "    6        240A 240A N   CA   1892 1893  142.48  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    4    2   15   33   54   83  120  138  165  224  267  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iql_MODLL.B99990001.pdb        77.19506\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqm\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26656    24953\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24953     402\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       76\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          65.9085\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.17589       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.290   2.290      8.0577       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  54.206  54.206      14.092       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.416   1.416     0.88283       1.000\n",
      " 5 Soft-sphere overlap restraints     :      76       0      0   0.009   0.009     0.69859       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      49       0      0   0.123   0.123     0.76771       1.000\n",
      "10 Distance restraints 2 (N-O)        :      54       0      0   0.370   0.370      4.2630       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   4.372   4.372      1.8033       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  67.291  67.291      1.4738       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  77.535  77.535     0.50620       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  70.634  70.634      1.9893       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 102.854 102.854      1.0707       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      34       0      0   0.380   0.380     0.11814       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      2  35.669 135.934      27.733       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      31       0      0   1.224   1.224      2.2766       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqm_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1242.7336\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -105.35  -62.90   61.01    7.38  -62.90   61.01    7.38\n",
      "    1        233K 233K N   CA   1837 1838    3.03  -40.80                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846   53.74   55.90   55.39    3.71 -124.80 -174.95    8.45\n",
      "    2        234T 234T N   CA   1846 1847   94.85   39.50                  143.50\n",
      "    3   6830 235R 236G C   N    1862 1864 -120.28  -80.20   40.46    2.51   82.20 -127.37   17.69\n",
      "    3        236G 236G N   CA   1864 1865  179.69  174.10                    8.50\n",
      "    4   6833 238P 239K C   N    1881 1883  -75.07  -70.20   18.29    1.21  -62.90  161.63   21.68\n",
      "    4        239K 239K N   CA   1883 1884  158.03  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -62.51  -68.20    8.04    0.79  -62.50  168.12   27.55\n",
      "    5        240A 240A N   CA   1892 1893  150.99  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   603  608   2.216\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    1    3   20   23   54   83  124  139  174  206  247  308\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqm_MODLL.B99990001.pdb        65.90852\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1iqn\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26592    24889\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24889     397\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          59.5529\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.19443       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.278   2.278      8.1654       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      2  54.454  54.454      16.061       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.526   1.526      1.0597       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.001   0.001     0.70594E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      49       0      0   0.250   0.250      3.1419       1.000\n",
      "10 Distance restraints 2 (N-O)        :      53       0      0   0.240   0.240      2.1270       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   3.340   3.340      1.0527       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  46.698  46.698     0.46398       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  19.825  19.825      1.8692       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0 110.855 110.855      3.5078       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  70.429  70.429      2.2058       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      32       0      0   0.186   0.186     0.24829E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      2  30.040 151.683      19.142       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      29       0      0   0.606   0.606     0.52996       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1iqn_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1418.6951\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864  -87.37  -80.20   17.51    0.41   82.20 -113.85   18.78\n",
      "    1        236G 236G N   CA   1864 1865 -169.92  174.10                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -75.99  -70.70   19.54    1.30  -63.50  158.88   22.80\n",
      "    2        237L 237L N   CA   1868 1869  160.41  141.60                  -41.20\n",
      "    3   6832 237L 238P C   N    1874 1876  -61.02  -58.70    7.71    0.48  -64.50  170.38   12.53\n",
      "    3        238P 238P N   CA   1876 1877  -23.14  -30.50                  147.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -98.81 -118.00   20.87    0.94  -62.90  175.62   24.91\n",
      "    4        239K 239K N   CA   1883 1884  147.29  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -112.04 -134.00   25.61    0.60  -62.50 -178.40   27.78\n",
      "    5        240A 240A N   CA   1892 1893  133.81  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    4   13   24   47   77  119  146  169  227  232  323\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1iqn_MODLL.B99990001.pdb        59.55293\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1ksn\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26541    24838\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24838     395\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      111\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          37.4737\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.12107       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.642   1.642      5.1242       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.972  46.972      12.235       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.528   1.528      1.4992       1.000\n",
      " 5 Soft-sphere overlap restraints     :     111       0      0   0.000   0.000     0.96364E-06   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.135   0.135     0.75832       1.000\n",
      "10 Distance restraints 2 (N-O)        :      50       0      0   0.252   0.252      1.2252       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.528   3.528      1.3210       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  62.341  62.341      1.1722       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  75.008  75.008      1.4203       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  17.815  17.815      1.1268       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  72.741  72.741      2.9710       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.695   0.695     0.27865       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  23.914 120.883      7.0721       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       5       0      0   1.628   1.628      1.1484       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1ksn_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1249.5664\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -131.20 -125.20    6.75    0.35  -63.00 -168.84   22.16\n",
      "    1        235R 235R N   CA   1853 1854  137.49  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  130.72   78.70   52.59    2.45   82.20 -175.84   11.28\n",
      "    2        236G 236G N   CA   1864 1865 -173.84 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -78.24  -70.70   13.21    0.82  -63.50  167.00   24.06\n",
      "    3        237L 237L N   CA   1868 1869  152.45  141.60                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892  -63.58  -68.20    5.19    0.36  -62.50  176.16   28.92\n",
      "    4        240A 240A N   CA   1892 1893  142.95  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    4   13   22   43   66  114  116  151  205  231  299\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1ksn_MODLL.B99990001.pdb        37.47373\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1lpg\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26584    24881\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24881     384\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      105\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          31.8286\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.85278E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.561   1.561      4.5445       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  53.712  53.712      13.880       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   0.940   0.940     0.43339       1.000\n",
      " 5 Soft-sphere overlap restraints     :     105       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.073   0.073     0.21234       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.195   0.195     0.63853       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.644   2.644     0.74170       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  22.996  22.996     0.35006       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  63.077  63.077      1.4889       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  71.663  71.663      2.8766       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 129.643 129.643      3.6270       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.524   0.524     0.13232       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      0  15.196 138.217      2.8180       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1lpg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1334.9937\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868  -66.03  -70.70    6.90    0.68  -63.50  172.13   24.05\n",
      "    1        237L 237L N   CA   1868 1869  146.68  141.60                  -41.20\n",
      "    2   6832 237L 238P C   N    1874 1876  -66.34  -58.70   12.27    0.78  -64.50  168.10   12.64\n",
      "    2        238P 238P N   CA   1876 1877  -20.89  -30.50                  147.20\n",
      "    3   6833 238P 239K C   N    1881 1883 -110.32 -118.00   11.84    0.43  -62.90  177.34   20.93\n",
      "    3        239K 239K N   CA   1883 1884  130.08  139.10                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892 -119.21 -134.00   15.81    0.36  -62.50 -173.48   33.59\n",
      "    4        240A 240A N   CA   1892 1893  141.41  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    0    5   16   43   85   91  119  148  203  243  317\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1lpg_MODLL.B99990001.pdb        31.82858\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1lpk\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26633    24930\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.4768\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24930     387\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      160\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         121.2400\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.50672       1.000\n",
      " 2 Bond angle potential               :      94       0      0   2.529   2.529      11.043       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.179  49.179      15.566       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.942   1.942      1.1989       1.000\n",
      " 5 Soft-sphere overlap restraints     :     160       3      5   0.063   0.063      75.836       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.255   0.255      2.4147       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.326   0.326      1.7111       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.453   2.453     0.63839       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  84.981  84.981      1.8975       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  93.978  93.978      2.7276       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  77.917  77.917      1.9957       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  87.609  87.609      2.5506       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      29       0      0   0.598   0.598     0.18137       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      0  25.444 137.535      2.9719       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1lpk_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1810.9120\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -152.50 -125.20   46.60    1.54  -63.00  166.62   28.47\n",
      "    1        235R 235R N   CA   1853 1854  178.36  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -129.05 -167.20   39.22    0.58   82.20 -143.73   16.50\n",
      "    2        236G 236G N   CA   1864 1865  165.49  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -97.32 -108.50   12.73    0.68  -63.50 -177.06   23.68\n",
      "    3        237L 237L N   CA   1868 1869  138.59  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -76.59  -70.20   12.66    0.78  -62.90  168.44   22.65\n",
      "    4        239K 239K N   CA   1883 1884  151.32  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -65.97  -68.20    5.80    0.39  -62.50  179.18   29.54\n",
      "    5        240A 240A N   CA   1892 1893  139.95  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    5   17   64   78   91  118  157  224  251  328\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1lpk_MODLL.B99990001.pdb       121.23995\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1lpz\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26626    24923\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.6103\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.7371\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24923     386\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      176\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          57.7906\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.52768       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.995   1.995      7.1031       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  52.577  52.577      16.112       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   2.439   2.439      1.9468       1.000\n",
      " 5 Soft-sphere overlap restraints     :     176       0      1   0.019   0.019      7.8585       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      48       0      0   0.263   0.263      2.2425       1.000\n",
      "10 Distance restraints 2 (N-O)        :      48       0      0   0.448   0.448      5.0947       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.709   2.709     0.77900       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0 109.035 109.035      4.3482       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  85.709  85.709      2.0538       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  71.522  71.522      2.0478       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  76.389  76.389      2.2543       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.685   0.685     0.20138       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  33.424 127.381      5.2209       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1lpz_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1687.1510\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  163.11 -125.20   73.25    2.56  -63.00 -148.86   37.14\n",
      "    1        235R 235R N   CA   1853 1854  155.64  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  165.65 -167.20   27.18    0.55   82.20 -175.41   12.75\n",
      "    2        236G 236G N   CA   1864 1865  173.15  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -113.94 -108.50    9.50    0.44  -63.50 -174.50   28.86\n",
      "    3        237L 237L N   CA   1868 1869  140.29  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -71.70  -70.20    8.34    0.57  -62.90  170.82   22.67\n",
      "    4        239K 239K N   CA   1883 1884  148.60  140.40                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3    8   16   44   74  112  115  168  202  230  312\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1lpz_MODLL.B99990001.pdb        57.79060\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1lqd\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26511    24808\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1845 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.1719\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24808     388\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      101\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          26.3610\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.59476E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.421   1.421      3.9283       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.018  47.018      11.259       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   0.909   0.909     0.36004       1.000\n",
      " 5 Soft-sphere overlap restraints     :     101       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.115   0.115     0.48272       1.000\n",
      "10 Distance restraints 2 (N-O)        :      49       0      0   0.204   0.204     0.82903       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.045   3.045     0.98409       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  60.110  60.110     0.32786       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  76.075  76.075      1.4055       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  73.763  73.763      2.4619       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  96.632  96.632      3.5428       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.625   0.625     0.18685       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       3      0  15.778 102.545     0.53326       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1lqd_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1175.3000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868  -73.84  -70.70    4.24    0.28  -63.50  174.66   24.86\n",
      "    1        237L 237L N   CA   1868 1869  144.45  141.60                  -41.20\n",
      "    2   6833 238P 239K C   N    1881 1883  -75.23  -70.20   12.56    0.80  -62.90  167.74   22.48\n",
      "    2        239K 239K N   CA   1883 1884  151.91  140.40                  -40.80\n",
      "    3   6834 239K 240A C   N    1890 1892 -106.10 -134.00   29.78    0.69  -62.50 -177.24   28.15\n",
      "    3        240A 240A N   CA   1892 1893  136.58  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    2   17   43   82   93  113  165  203  248  281\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1lqd_MODLL.B99990001.pdb        26.36098\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 1mq5\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26702    24999\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1845 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.9372\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24999     503\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      287\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         431.3616\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       1      4   0.028   0.028      17.755       1.000\n",
      " 2 Bond angle potential               :     102       7      9   9.032   9.032      148.92       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      4  63.198  63.198      23.748       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.743   3.743      9.2779       1.000\n",
      " 5 Soft-sphere overlap restraints     :     287       3     10   0.046   0.046      71.722       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       0      6   0.788   0.788      37.462       1.000\n",
      "10 Distance restraints 2 (N-O)        :      67       1      8   0.927   0.927      37.565       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       2      3  14.761  14.761      28.267       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      1  65.257  65.257      5.1713       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  84.610  84.610      4.3601       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      1 117.893 117.893      5.8330       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 123.319 123.319      1.8866       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.335   0.335     0.70950E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      4  36.526 130.791      26.067       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      60       0      0   1.445   1.445      13.261       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1mq5_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3017.9944\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1936 240A 241K C   N    1895 1897    1.47    1.35    0.13    4.51    1.35    0.13    4.51\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4477 232M 232M N   CA   1829 1830  130.39  107.00   23.39    6.73  107.00   23.39    6.73\n",
      "    2   4488 233K 233K CA  CB   1838 1839  134.80  113.50   21.30    5.23  113.50   21.30    5.23\n",
      "    3   4491 233K 233K CB  CG   1839 1840  139.75  113.60   26.15    6.42  113.60   26.15    6.42\n",
      "    4   4563 240A 240A CA  C    1893 1895  135.70  116.50   19.20    5.52  116.50   19.20    5.52\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   66.07   56.60   19.20    0.98  -62.90  143.40   25.05\n",
      "    1        233K 233K N   CA   1837 1838   21.90   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -113.17 -124.80   19.68    0.61  -63.20  176.93   19.81\n",
      "    2        234T 234T N   CA   1846 1847  127.62  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -150.83 -125.20   51.60    1.83  -63.00  159.82   27.42\n",
      "    3        235R 235R N   CA   1853 1854 -174.62  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -61.73  -62.40    1.06    0.15   82.20  152.54   11.62\n",
      "    4        236G 236G N   CA   1864 1865  -42.02  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -96.95 -108.50   35.75    2.02  -63.50  156.09   23.71\n",
      "    5        237L 237L N   CA   1868 1869  166.33  132.50                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883 -163.01 -118.00   63.49    2.24  -62.90  168.31   27.74\n",
      "    6        239K 239K N   CA   1883 1884 -176.11  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892 -122.61 -134.00   16.81    0.99  -62.50  170.68   31.19\n",
      "    7        240A 240A N   CA   1892 1893  159.35  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    6   27   58   87   99  142  169  235  248  328\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1mq5_MODLL.B99990001.pdb       431.36157\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 1mq6\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26314    24611\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.3987\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.8482\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24611     502\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      307\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         481.9940\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      4   0.034   0.034      23.998       1.000\n",
      " 2 Bond angle potential               :     102       6     13   8.193   8.193      122.38       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      6  67.699  67.699      24.976       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   3.534   3.534      7.7034       1.000\n",
      " 5 Soft-sphere overlap restraints     :     307       4     13   0.050   0.050      91.927       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      64       3     11   1.140   1.140      78.754       1.000\n",
      "10 Distance restraints 2 (N-O)        :      66       1     12   1.175   1.175      65.020       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       2      2  11.057  11.057      15.860       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      1  78.038  78.038      4.1707       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      1  75.772  75.772      7.4977       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      1 105.163 105.163      5.0086       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      1 119.942 119.942      4.2325       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      30       0      0   0.726   0.726     0.44568       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      2  25.821 123.258      17.214       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      57       0      0   1.444   1.444      12.811       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1mq6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2870.8557\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4466 230R 231S C   N    1821 1823  140.60  120.00   20.60    4.68  120.00   20.60    4.68\n",
      "    2   4474 231S 232M C   N    1827 1829  147.21  120.00   27.21    6.18  120.00   27.21    6.18\n",
      "    3   4477 232M 232M N   CA   1829 1830  125.47  107.00   18.47    5.31  107.00   18.47    5.31\n",
      "    4   4563 240A 240A CA  C    1893 1895  134.47  116.50   17.97    5.17  116.50   17.97    5.17\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   66.16   56.60   17.34    0.92  -62.90  144.47   25.24\n",
      "    1        233K 233K N   CA   1837 1838   24.13   38.60                  -40.80\n",
      "    2   6829 234T 235R C   N    1851 1853  -72.87  -72.10    6.03    0.45  -63.00  171.31   23.81\n",
      "    2        235R 235R N   CA   1853 1854  147.88  141.90                  -41.10\n",
      "    3   6830 235R 236G C   N    1862 1864   97.28   78.70   36.11    0.55   82.20  155.17    8.29\n",
      "    3        236G 236G N   CA   1864 1865  162.93 -166.10                    8.50\n",
      "    4   6832 237L 238P C   N    1874 1876  -64.30  -58.70    5.70    0.67  -64.50  176.64   13.17\n",
      "    4        238P 238P N   CA   1876 1877  -29.44  -30.50                  147.20\n",
      "    5   6833 238P 239K C   N    1881 1883   59.81   56.60    6.55    0.33  -62.90  143.14   24.98\n",
      "    5        239K 239K N   CA   1883 1884   32.89   38.60                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  169.13 -134.00   66.61    3.52  -62.50 -160.11   30.40\n",
      "    6        240A 240A N   CA   1892 1893  112.32  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    4    8   21   49   86  100  131  173  190  259  315\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1mq6_MODLL.B99990001.pdb       481.99402\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1nfu\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26530    24827\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24827     390\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      104\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          42.8684\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.80923E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.456   1.456      4.0299       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.002  49.002      12.990       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.634   2.634      4.9280       1.000\n",
      " 5 Soft-sphere overlap restraints     :     104       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.174   0.174      1.0127       1.000\n",
      "10 Distance restraints 2 (N-O)        :      49       0      0   0.244   0.244      1.5398       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.870   3.870      1.5893       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  60.224  60.224     0.34024       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  67.943  67.943     0.28595E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  58.650  58.650      1.5383       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  88.469  88.469      1.3386       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      29       0      0   0.518   0.518     0.16563       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  24.834 139.303      13.201       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       2       0      0   0.601   0.601     0.85574E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1nfu_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1639.5033\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -120.10 -125.20   29.19    1.51  -63.00  160.09   25.56\n",
      "    1        235R 235R N   CA   1853 1854  169.34  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -86.90  -80.20   26.91    0.75   82.20 -121.40   18.31\n",
      "    2        236G 236G N   CA   1864 1865 -159.83  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -109.51 -108.50   34.17    1.84  -63.50  158.94   24.91\n",
      "    3        237L 237L N   CA   1868 1869  166.66  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -69.90  -70.20   11.37    0.82  -62.90  169.98   21.70\n",
      "    4        239K 239K N   CA   1883 1884  129.03  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -141.51 -134.00   11.56    0.38  -62.50 -178.57   33.96\n",
      "    5        240A 240A N   CA   1892 1893  155.78  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   11   22   39   66   82  124  141  183  253  322\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1nfu_MODLL.B99990001.pdb        42.86836\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1nfw\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26492    24789\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24789     392\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      105\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          61.2910\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.18635       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.760   1.760      5.5248       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  49.050  49.050      13.306       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       1      1   3.243   3.243      7.6339       1.000\n",
      " 5 Soft-sphere overlap restraints     :     105       0      0   0.001   0.001     0.42958E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.251   0.251      2.3234       1.000\n",
      "10 Distance restraints 2 (N-O)        :      50       0      0   0.382   0.382      3.8758       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   5.280   5.280      2.9584       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  66.619  66.619     0.38549       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  80.116  80.116      1.2324       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  54.456  54.456      2.1786       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  54.177  54.177      1.7077       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.692   0.692     0.31250       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      4  36.660 141.817      19.364       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       2       0      0   1.567   1.567     0.29717       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1nfw_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1621.0062\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6574 232M 232M C   CA   1835 1830   13.84    0.00   13.84    4.87    0.00   13.84    4.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -124.13 -125.20    5.69    0.30  -63.00 -176.79   28.97\n",
      "    1        235R 235R N   CA   1853 1854  146.19  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -94.89  -80.20   53.10    2.82   82.20 -149.08    8.52\n",
      "    2        236G 236G N   CA   1864 1865  123.07  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -132.03 -108.50   24.14    1.22  -63.50 -178.28   22.42\n",
      "    3        237L 237L N   CA   1868 1869  127.10  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -83.74  -70.20   13.60    1.02  -62.90 -178.94   22.45\n",
      "    4        239K 239K N   CA   1883 1884  139.05  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -72.61  -68.20    6.40    0.40  -62.50  169.47   28.28\n",
      "    5        240A 240A N   CA   1892 1893  149.94  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    5    5   24   42   58   85  118  162  187  243  290\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1nfw_MODLL.B99990001.pdb        61.29100\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1nfx\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26167    24464\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24464     393\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       84\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          52.4074\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.25638       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.868   1.868      6.8762       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.299  47.299      12.702       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.625   2.625      4.7792       1.000\n",
      " 5 Soft-sphere overlap restraints     :      84       0      0   0.001   0.001     0.13943E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.247   0.247      2.4521       1.000\n",
      "10 Distance restraints 2 (N-O)        :      49       0      0   0.342   0.342      2.2769       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.687   4.687      2.3312       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  90.103  90.103      2.2324       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  52.370  52.370    -0.14108E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  24.671  24.671      1.4451       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  58.170  58.170      2.5638       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      31       0      0   0.996   0.996     0.79112       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  23.429 138.243      13.676       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   1.129   1.129     0.24588E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1nfx_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1620.7310\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -114.40 -125.20   11.91    0.60  -63.00 -179.24   27.97\n",
      "    1        235R 235R N   CA   1853 1854  145.61  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -69.99  -80.20   26.87    0.64   82.20 -152.71    6.78\n",
      "    2        236G 236G N   CA   1864 1865  149.24  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -116.81 -108.50   12.38    0.56  -63.50 -175.03   28.97\n",
      "    3        237L 237L N   CA   1868 1869  141.68  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -57.57  -70.20   12.68    0.90  -62.90  179.94   23.08\n",
      "    4        239K 239K N   CA   1883 1884  139.33  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -72.95  -68.20   13.23    0.90  -62.50  161.79   27.04\n",
      "    5        240A 240A N   CA   1892 1893  157.65  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    6   20   39   65   80  111  151  188  234  272\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1nfx_MODLL.B99990001.pdb        52.40738\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1nfy\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26530    24827\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1863 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24827     400\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      104\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          50.2359\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.12191       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.772   1.772      6.0092       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  45.426  45.426      10.781       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.419   2.419      4.1493       1.000\n",
      " 5 Soft-sphere overlap restraints     :     104       0      0   0.000   0.000     0.22995E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.345   0.345      5.0029       1.000\n",
      "10 Distance restraints 2 (N-O)        :      51       0      0   0.315   0.315      1.9914       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.399   3.399      1.2259       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  58.239  58.239     0.92534       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  43.300  43.300     0.73272       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  41.755  41.755      3.0449       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 118.059 118.059      2.8043       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      31       0      0   0.843   0.843     0.52514       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  33.655 115.035      10.632       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       6       0      0   2.294   2.294      2.2876       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1nfy_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1367.7842\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864  134.28 -167.20   58.52    1.11   82.20  174.11   10.94\n",
      "    1        236G 236G N   CA   1864 1865  174.65  174.60                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868 -103.95 -108.50   25.09    1.40  -63.50  166.61   25.61\n",
      "    2        237L 237L N   CA   1868 1869  157.17  132.50                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -56.27  -70.20   20.81    1.25  -62.90  165.87   21.91\n",
      "    3        239K 239K N   CA   1883 1884  124.94  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -78.71  -68.20   12.82    0.84  -62.50  167.24   28.23\n",
      "    4        240A 240A N   CA   1892 1893  152.65  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2   13   17   39   67   94  109  162  176  263  279\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1nfy_MODLL.B99990001.pdb        50.23592\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1p0s\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26587    24884\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1775 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24884     384\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       72\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          26.3378\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.16259       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.545   1.545      3.9841       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      0  44.132  44.132      9.4765       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   0.933   0.933     0.32894       1.000\n",
      " 5 Soft-sphere overlap restraints     :      72       0      0   0.001   0.001     0.11148E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      37       0      0   0.059   0.059     0.12826       1.000\n",
      "10 Distance restraints 2 (N-O)        :      21       0      0   0.078   0.078     0.71245E-01   1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.206   2.206     0.45895       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  52.354  52.354    -0.57942       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  82.700  82.700      1.2646       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  88.037  88.037      3.7654       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 104.466 104.466      2.9660       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      51       0      0   0.335   0.335     0.11440       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       3      0  21.645 107.041      3.8055       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      41       0      0   0.381   0.381     0.37953       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1p0s_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1156.1558\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868  -73.43  -70.70    9.43    0.62  -63.50  168.46   23.97\n",
      "    1        237L 237L N   CA   1868 1869  150.63  141.60                  -41.20\n",
      "    2   6833 238P 239K C   N    1881 1883  -67.73  -70.20    2.53    0.20  -62.90  178.30   23.42\n",
      "    2        239K 239K N   CA   1883 1884  140.96  140.40                  -40.80\n",
      "    3   6834 239K 240A C   N    1890 1892  -76.80  -68.20   11.21    0.72  -62.50  167.22   28.13\n",
      "    3        240A 240A N   CA   1892 1893  152.49  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   165  182   2.295\n",
      "  serious non-bonded atom clash:  1507 1531   2.290\n",
      "  serious non-bonded atom clash:  1772 1808   2.197\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    2    1    3   11   28   52   72  116  132  171  204  234  304\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1p0s_MODLL.B99990001.pdb        26.33780\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 1v3x\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26183    24480\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1777 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24480     500\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      280\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         200.5417\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      2   0.021   0.021      9.1353       1.000\n",
      " 2 Bond angle potential               :     102       4      5   5.911   5.911      64.416       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      1  49.005  49.005      14.918       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   3.245   3.245      6.4265       1.000\n",
      " 5 Soft-sphere overlap restraints     :     280       0      3   0.022   0.022      16.639       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      67       0      6   0.745   0.745      34.991       1.000\n",
      "10 Distance restraints 2 (N-O)        :      67       0      2   0.746   0.746      20.638       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1   8.186   8.186      8.6918       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  81.881  81.881      3.4026       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  69.746  69.746      2.6370       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  93.207  93.207      4.4834       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  76.636  76.636      1.7312       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.545   0.545     0.13193       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      3  27.204 127.395      9.9359       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      57       0      0   0.969   0.969      2.3648       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1v3x_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2177.9089\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  146.99  120.00   26.99    6.13  120.00   26.99    6.13\n",
      "    2   4477 232M 232M N   CA   1829 1830  126.37  107.00   19.38    5.57  107.00   19.38    5.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7074 240A 240A CA  C    1893 1895 -155.58 -180.00   24.42    4.88 -180.00   24.42    4.88\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -112.85 -118.00   13.96    0.59  -62.90  174.24   20.46\n",
      "    1        233K 233K N   CA   1837 1838  126.12  139.10                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -111.16 -124.80   58.12    2.89  -63.20  127.28   19.48\n",
      "    2        234T 234T N   CA   1846 1847 -160.00  143.50                  -42.10\n",
      "    3   6830 235R 236G C   N    1862 1864  -85.81  -80.20   19.30    1.04   82.20 -136.67    7.54\n",
      "    3        236G 236G N   CA   1864 1865  155.64  174.10                    8.50\n",
      "    4   6832 237L 238P C   N    1874 1876  -61.52  -58.70    3.66    0.28  -64.50  175.38   12.93\n",
      "    4        238P 238P N   CA   1876 1877  -28.16  -30.50                  147.20\n",
      "    5   6833 238P 239K C   N    1881 1883 -101.53 -118.00   29.86    1.54  -62.90  159.93   23.05\n",
      "    5        239K 239K N   CA   1883 1884  164.01  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892 -132.66 -134.00   35.11    2.02  -62.50  153.93   28.98\n",
      "    6        240A 240A N   CA   1892 1893 -177.91  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1387 1403   1.837\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    5   16   21   50   68   97  123  137  213  278  325\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1v3x_MODLL.B99990001.pdb       200.54173\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 1wu1\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26077    24374\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24374     504\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      285\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         194.2204\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.020   0.020      8.3611       1.000\n",
      " 2 Bond angle potential               :     102       3      6   5.913   5.913      63.237       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  56.470  56.470      18.835       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.524   3.524      7.0798       1.000\n",
      " 5 Soft-sphere overlap restraints     :     285       0      3   0.025   0.025      21.826       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      65       0      0   0.449   0.449      12.001       1.000\n",
      "10 Distance restraints 2 (N-O)        :      67       0      0   0.563   0.563      10.494       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      2   9.823   9.823      12.518       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  87.164  87.164      3.0517       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      1  63.212  63.212      3.6787       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  81.549  81.549      2.7829       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0   5.192   5.192      1.0762       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.366   0.366     0.82778E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      3  38.361 112.062      25.472       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      59       0      0   0.735   0.735      3.7235       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1wu1_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2288.8660\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4477 232M 232M N   CA   1829 1830  123.62  107.00   16.62    4.78  107.00   16.62    4.78\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   57.90   56.60    4.03    0.39  -62.90  146.69   25.52\n",
      "    1        233K 233K N   CA   1837 1838   42.42   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -68.71  -78.10   14.23    0.50  -63.20  178.89   23.30\n",
      "    2        234T 234T N   CA   1846 1847  139.10  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -148.64 -125.20   37.50    1.21  -63.00  171.89   28.98\n",
      "    3        235R 235R N   CA   1853 1854  169.87  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -62.18  -62.40    7.22    1.07   82.20  155.20   11.91\n",
      "    4        236G 236G N   CA   1864 1865  -48.42  -41.20                    8.50\n",
      "    5   6833 238P 239K C   N    1881 1883  -99.11 -118.00   92.36    4.69  -62.90   96.72   14.73\n",
      "    5        239K 239K N   CA   1883 1884 -130.49  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -94.05  -68.20   50.00    3.22  -62.50  134.75   23.75\n",
      "    6        240A 240A N   CA   1892 1893 -171.90  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1398 1403   2.296\n",
      "  serious non-bonded atom clash:  1403 1649   2.226\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    6    5   15   26   62   78   97  116  165  213  243  313\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1wu1_MODLL.B99990001.pdb       194.22035\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1xka\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    27117    25414\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1863 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25414     405\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      102\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          60.0762\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.004   0.004     0.21727       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.978   1.978      6.3100       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  58.379  58.379      16.344       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.181   1.181     0.68985       1.000\n",
      " 5 Soft-sphere overlap restraints     :     102       0      0   0.002   0.002     0.27493E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      50       0      0   0.135   0.135     0.85486       1.000\n",
      "10 Distance restraints 2 (N-O)        :      58       0      0   0.236   0.236      1.7679       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   2.325   2.325     0.51017       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  49.063  49.063     0.50009       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  97.163  97.163      1.7703       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  49.420  49.420      1.3519       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  86.672  86.672      2.1263       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.437   0.437     0.82911E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       6      1  28.664 160.524      24.758       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      38       0      0   0.738   0.738      2.7650       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1xka_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1486.0913\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -110.92  -62.90   60.38    7.72  -62.90   60.38    7.72\n",
      "    1        233K 233K N   CA   1837 1838   -4.20  -40.80                  -40.80\n",
      "    2   6829 234T 235R C   N    1851 1853   69.58   57.30   26.36    1.30 -125.20 -152.27   11.15\n",
      "    2        235R 235R N   CA   1853 1854   14.67   38.00                  140.60\n",
      "    3   6830 235R 236G C   N    1862 1864  -91.89  -80.20   22.87    0.54   82.20 -113.33   18.91\n",
      "    3        236G 236G N   CA   1864 1865 -166.25  174.10                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868  -67.84  -70.70    3.75    0.25  -63.50  179.68   25.20\n",
      "    4        237L 237L N   CA   1868 1869  139.17  141.60                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883  -66.80  -70.20    4.63    0.39  -62.90  175.69   23.03\n",
      "    5        239K 239K N   CA   1883 1884  143.55  140.40                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -72.08  -68.20    4.86    0.48  -62.50  176.98   29.48\n",
      "    6        240A 240A N   CA   1892 1893  142.38  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   505  808   2.294\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    3    5    9   32   44   73   97  158  166  217  277  295\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1xka_MODLL.B99990001.pdb        60.07623\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 1xkb\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26950    25247\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25247     415\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      120\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          75.8847\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.22208       1.000\n",
      " 2 Bond angle potential               :      83       0      0   1.964   1.964      5.9984       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      1  45.142  45.142      9.2286       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      1   2.460   2.460      3.8492       1.000\n",
      " 5 Soft-sphere overlap restraints     :     120       0      0   0.003   0.003     0.10669       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.194   0.194      1.3097       1.000\n",
      "10 Distance restraints 2 (N-O)        :      51       0      0   0.246   0.246      2.0733       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   1.800   1.800     0.30555       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  95.626  95.626      2.8028       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  65.102  65.102      1.6853       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0 119.634 119.634      3.2493       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 119.330 119.330      1.7964       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      37       0      0   0.359   0.359     0.87027E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       5      3  42.277 133.042      40.226       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      48       0      0   0.671   0.671      2.9448       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1xkb_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1504.6995\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -106.94  -62.90   85.00    9.79  -62.90   85.00    9.79\n",
      "    1        233K 233K N   CA   1837 1838   31.90  -40.80                  -40.80\n",
      "    2   6830 235R 236G C   N    1862 1864 -133.26 -167.20   44.98    0.92   82.20 -161.15   15.32\n",
      "    2        236G 236G N   CA   1864 1865  145.08  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -74.98  -70.70    9.99    0.63  -63.50  168.56   24.08\n",
      "    3        237L 237L N   CA   1868 1869  150.63  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -69.41  -70.20    5.74    0.40  -62.90  175.64   22.46\n",
      "    4        239K 239K N   CA   1883 1884  134.72  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -86.43  -68.20   28.05    1.76  -62.50  154.35   26.53\n",
      "    5        240A 240A N   CA   1892 1893  166.62  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    5   16   32   45   64  100  156  180  229  261  287\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1xkb_MODLL.B99990001.pdb        75.88474\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 1z6e\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        1\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26633    24930\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24930     409\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      100\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          99.6096\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.43342       1.000\n",
      " 2 Bond angle potential               :      94       0      0   2.359   2.359      10.649       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  50.904  50.904      14.338       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       1      1   3.707   3.707      9.5855       1.000\n",
      " 5 Soft-sphere overlap restraints     :     100       0      0   0.003   0.003     0.10701       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      50       0      0   0.348   0.348      5.1880       1.000\n",
      "10 Distance restraints 2 (N-O)        :      57       0      0   0.389   0.389      5.3402       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      2   6.964   6.964      5.1477       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  65.228  65.228      2.1419       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  97.853  97.853      2.2454       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  69.831  69.831      1.5244       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  55.851  55.851      1.4414       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      31       0      0   0.781   0.781     0.43005       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       6      4  59.344 136.785      35.285       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       6       0      0   2.676   2.676      5.7520       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 1z6e_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1784.0891\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6574 232M 232M C   CA   1835 1830   14.92    0.00   14.92    5.25    0.00   14.92    5.25\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6826 231S 232M C   N    1827 1829  -75.48  -63.40   53.06    7.07  -63.40   53.06    7.07\n",
      "    1        232M 232M N   CA   1829 1830   11.17  -40.50                  -40.50\n",
      "    2   6829 234T 235R C   N    1851 1853  107.71 -125.20  130.21    4.52  -63.00 -132.78   27.52\n",
      "    2        235R 235R N   CA   1853 1854  168.95  140.60                  -41.10\n",
      "    3   6830 235R 236G C   N    1862 1864  103.41   78.70   46.06    0.73   82.20  148.05    8.24\n",
      "    3        236G 236G N   CA   1864 1865  155.02 -166.10                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868 -111.59 -108.50   25.50    1.34  -63.50  168.01   26.29\n",
      "    4        237L 237L N   CA   1868 1869  157.81  132.50                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883  -96.02 -118.00   28.22    1.39  -62.90  165.74   23.46\n",
      "    5        239K 239K N   CA   1883 1884  156.80  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -68.68  -68.20    6.56    0.51  -62.50  167.38   27.74\n",
      "    6        240A 240A N   CA   1892 1893  151.84  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    9   18   45   70   98  125  159  209  249  327\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "1z6e_MODLL.B99990001.pdb        99.60955\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2bmg\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26390    24687\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24687     383\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      101\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          35.5514\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.20162       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.767   1.767      5.5005       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.353  47.353      11.257       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.530   1.530      1.2782       1.000\n",
      " 5 Soft-sphere overlap restraints     :     101       0      0   0.002   0.002     0.50844E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.156   0.156     0.73020       1.000\n",
      "10 Distance restraints 2 (N-O)        :      41       0      0   0.440   0.440      2.4475       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.110   2.110     0.47251       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  64.259  64.259     0.82324       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  64.068  64.068     0.98498       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  66.322  66.322      2.5277       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  74.234  74.234      2.8516       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.354   0.354     0.84907E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       3      1  15.182  98.312      6.0247       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       7       0      0   0.945   0.945     0.31547       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2bmg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1150.7339\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864 -168.04 -167.20   18.75    0.74   82.20 -176.25   13.59\n",
      "    1        236G 236G N   CA   1864 1865  155.87  174.60                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -77.29  -70.70   19.10    1.23  -63.50  159.87   23.01\n",
      "    2        237L 237L N   CA   1868 1869  159.53  141.60                  -41.20\n",
      "    3   6834 239K 240A C   N    1890 1892  -82.40  -68.20   18.45    1.18  -62.50  163.25   27.77\n",
      "    3        240A 240A N   CA   1892 1893  157.07  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    9   18   38   73   94  125  161  196  255  302\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2bmg_MODLL.B99990001.pdb        35.55145\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2boh\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26437    24734\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1756 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24734     379\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      103\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          34.2928\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.18395       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.483   1.483      4.0564       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  51.387  51.387      13.693       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.050   1.050     0.45407       1.000\n",
      " 5 Soft-sphere overlap restraints     :     103       0      0   0.001   0.001     0.19714E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.096   0.096     0.37945       1.000\n",
      "10 Distance restraints 2 (N-O)        :      45       0      0   0.199   0.199     0.90180       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.614   2.614     0.72500       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  76.335  76.335     0.78903       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  86.810  86.810     0.96611       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  69.545  69.545      2.7143       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  52.500  52.500      2.9571       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.416   0.416     0.75937E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      0  25.357 122.276      6.3768       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2boh_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1509.6748\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -106.19 -125.20   35.95    1.24  -63.00  157.24   18.73\n",
      "    1        235R 235R N   CA   1853 1854  110.09  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   54.45   78.70   50.02    0.75   82.20  133.77    7.85\n",
      "    2        236G 236G N   CA   1864 1865 -122.35 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -79.55  -70.70   11.39    1.14  -63.50  176.38   23.62\n",
      "    3        237L 237L N   CA   1868 1869  134.44  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -128.02 -118.00   17.08    0.64  -62.90  178.56   27.08\n",
      "    4        239K 239K N   CA   1883 1884  152.94  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -76.62  -68.20   16.44    1.06  -62.50  160.31   26.99\n",
      "    5        240A 240A N   CA   1892 1893  159.42  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   494  497   2.140\n",
      "  serious non-bonded atom clash:   497  502   2.267\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    1    0    1    3   13   54   56  103   99  154  215  252  286\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2boh_MODLL.B99990001.pdb        34.29281\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2bok\n",
      "{'num_gaps': 2, 'gap_lengths': [11, 8], 'gap_list': [[56, 66], [233, 240]], 'gap_window': [[54, 68], [231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    24997    23294\n",
      "184 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.6626\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     184\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    23294    1341\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      430\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         220.6557\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     187       0      1   0.010   0.010      6.4725       1.000\n",
      " 2 Bond angle potential               :     253       2      6   3.179   3.179      47.609       1.000\n",
      " 3 Stereochemical cosine torsion poten:      91       0      3  53.233  53.233      43.514       1.000\n",
      " 4 Stereochemical improper torsion pot:      64       0      0   1.878   1.878      5.4014       1.000\n",
      " 5 Soft-sphere overlap restraints     :     430       1      3   0.020   0.020      20.718       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     232       0      0   0.218   0.218      9.3096       1.000\n",
      "10 Distance restraints 2 (N-O)        :     216       0      0   0.348   0.348      15.026       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      27       1      1   6.926   6.926      14.662       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  60.363  60.363      3.4113       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  64.465  64.465      5.9505       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      11       0      0 111.701 111.701      9.4762       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  31.000  31.000      2.2569       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      95       0      0   0.235   0.235     0.12769       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      28      14      5  32.868 125.098      34.556       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     100       0      0   0.439   0.439      2.1631       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2bok_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4306.8862\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6650  55G  56D C   N     426  428  -68.83  -96.50   27.73    1.13  -63.30  152.52   18.50\n",
      "    1         56D  56D N   CA    428  429  112.43  114.20                  -40.00\n",
      "    2   6651  56D  57R C   N     434  436 -155.11 -125.20   30.18    1.11  -63.00 -162.93   32.86\n",
      "    2         57R  57R N   CA    436  437  144.68  140.60                  -41.10\n",
      "    3   6652  57R  58N C   N     445  447  -84.65  -71.20   15.38    1.24  -63.20  177.73   21.11\n",
      "    3         58N  58N N   CA    447  448  135.34  142.80                  -41.10\n",
      "    4   6653  58N  59T C   N     453  455 -154.25 -124.80   63.28    2.00  -63.20  149.36   24.77\n",
      "    4         59T  59T N   CA    455  456 -160.50  143.50                  -42.10\n",
      "    5   6654  59T  60E C   N     460  462  -67.90  -69.30    1.52    0.09  -63.60  177.84   23.95\n",
      "    5         60E  60E N   CA    462  463  141.91  142.50                  -40.30\n",
      "    6   6657  62E  63E C   N     487  489 -118.47 -117.80   12.50    0.62  -63.60  179.03   26.97\n",
      "    6         63E  63E N   CA    489  490  149.28  136.80                  -40.30\n",
      "    7   6658  63E  64G C   N     496  498  106.49   78.70   41.78    0.85   82.20  156.10    8.78\n",
      "    7         64G  64G N   CA    498  499  162.70 -166.10                    8.50\n",
      "    8   6659  64G  65G C   N     500  502 -151.70 -167.20   41.07    1.33   82.20  179.72   13.76\n",
      "    8         65G  65G N   CA    502  503  136.56  174.60                    8.50\n",
      "    9   6660  65G  66E C   N     504  506  -89.20  -69.30   20.67    1.36  -63.60  173.48   24.57\n",
      "    9         66E  66E N   CA    506  507  148.11  142.50                  -40.30\n",
      "   10   6661  66E  67A C   N     513  515 -112.41 -134.00   24.84    0.58  -62.50 -177.43   27.93\n",
      "   10         67A  67A N   CA    515  516  134.71  147.00                  -40.90\n",
      "   11   6827 232M 233K C   N    1835 1837   71.15   56.60  111.12    9.07  -62.90 -143.17   25.47\n",
      "   11        233K 233K N   CA   1837 1838  148.76   38.60                  -40.80\n",
      "   12   6829 234T 235R C   N    1851 1853 -147.99 -125.20   30.43    0.91  -63.00  179.54   30.00\n",
      "   12        235R 235R N   CA   1853 1854  160.76  140.60                  -41.10\n",
      "   13   6830 235R 236G C   N    1862 1864  160.20 -167.20   33.51    0.84   82.20  176.50   12.13\n",
      "   13        236G 236G N   CA   1864 1865  166.83  174.60                    8.50\n",
      "   14   6834 239K 240A C   N    1890 1892  -79.13  -68.20   24.58    1.62  -62.50  152.70   25.87\n",
      "   14        240A 240A N   CA   1892 1893  167.32  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    61  824   2.174\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    0    2    2   12   43   60  101  100  140  196  246  298\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2bok_MODLL.B99990001.pdb       220.65570\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2bq6\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26677    24974\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24974     379\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          30.8256\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.50592E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.485   1.485      4.0264       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.108  46.108      10.342       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.194   1.194     0.71766       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.144   0.144     0.75342       1.000\n",
      "10 Distance restraints 2 (N-O)        :      39       0      0   0.334   0.334      1.2040       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.607   2.607     0.72120       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  85.442  85.442      1.8418       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  90.871  90.871      2.5312       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  85.851  85.851      3.4507       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0   2.217   2.217      1.2426       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      30       0      0   0.317   0.317     0.57300E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      0  13.505 120.208      3.6471       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       3       0      0   1.006   1.006     0.24011       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2bq6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1398.3517\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864 -158.92 -167.20   21.40    0.69   82.20 -171.44   14.12\n",
      "    1        236G 236G N   CA   1864 1865  154.87  174.60                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -75.47  -70.70    5.47    0.55  -63.50 -179.73   25.73\n",
      "    2        237L 237L N   CA   1868 1869  138.93  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -71.70  -70.20    9.63    0.66  -62.90  169.52   22.51\n",
      "    3        239K 239K N   CA   1883 1884  149.91  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -61.80  -68.20    6.47    0.58  -62.50  172.84   28.29\n",
      "    4        240A 240A N   CA   1892 1893  146.26  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    3    9   10   27   62   94  117  138  162  205  235  311\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2bq6_MODLL.B99990001.pdb        30.82565\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2bq7\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26462    24759\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24759     376\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      100\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          28.8431\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.53744E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.354   1.354      3.3871       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  45.795  45.795      11.069       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   0.960   0.960     0.41378       1.000\n",
      " 5 Soft-sphere overlap restraints     :     100       0      0   0.001   0.001     0.79785E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.058   0.058     0.10875       1.000\n",
      "10 Distance restraints 2 (N-O)        :      41       0      0   0.282   0.282      1.5897       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   1.881   1.881     0.37556       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  88.968  88.968     0.86609       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  85.016  85.016      1.1307       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  56.750  56.750      1.5940       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  76.642  76.642      2.1775       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.430   0.430     0.85734E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  14.026 131.923      5.9837       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2bq7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1555.9263\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -112.01 -125.20   13.20    0.52  -63.00 -174.78   28.41\n",
      "    1        235R 235R N   CA   1853 1854  140.28  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   67.48   78.70   11.29    0.54   82.20  173.93    9.18\n",
      "    2        236G 236G N   CA   1864 1865 -164.80 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -73.58  -70.70    7.36    0.67  -63.50  176.32   23.92\n",
      "    3        237L 237L N   CA   1868 1869  134.83  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -61.38  -70.20   10.61    0.88  -62.90  172.90   22.37\n",
      "    4        239K 239K N   CA   1883 1884  146.31  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -66.06  -68.20    7.36    0.66  -62.50  166.80   27.51\n",
      "    5        240A 240A N   CA   1892 1893  152.34  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    1    7   18   42   67  101  133  158  205  253  300\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2bq7_MODLL.B99990001.pdb        28.84308\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2bqw\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26457    24754\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1787 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24754     372\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      145\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          39.7952\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.24319       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.934   1.934      6.7019       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  51.454  51.454      14.750       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.229   1.229     0.83448       1.000\n",
      " 5 Soft-sphere overlap restraints     :     145       0      0   0.004   0.004     0.28823       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.156   0.156     0.72575       1.000\n",
      "10 Distance restraints 2 (N-O)        :      39       0      0   0.451   0.451      2.2275       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.193   3.193      1.0822       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  44.785  44.785     0.89737       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  92.215  92.215      3.7534       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  24.752  24.752      1.5098       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  79.185  79.185      2.4413       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.256   0.256     0.28077E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      0  23.373 137.133      4.3125       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2bqw_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1665.8259\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -153.53 -125.20   34.07    1.01  -63.00 -176.72   30.85\n",
      "    1        235R 235R N   CA   1853 1854  159.53  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -140.90 -167.20   28.88    0.43   82.20 -153.83   15.62\n",
      "    2        236G 236G N   CA   1864 1865  162.67  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -91.75  -70.70   31.92    2.01  -63.50  155.79   23.34\n",
      "    3        237L 237L N   CA   1868 1869  165.59  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -136.78 -118.00   19.40    0.66  -62.90 -169.84   29.12\n",
      "    4        239K 239K N   CA   1883 1884  143.97  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -65.14  -68.20    9.83    0.69  -62.50  176.88   28.86\n",
      "    5        240A 240A N   CA   1892 1893  135.96  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   13   21   46   80   92  136  185  224  256  280\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2bqw_MODLL.B99990001.pdb        39.79518\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2cji\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25669    23966\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1770 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    23966     363\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      103\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          34.6357\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.18637       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.699   1.699      5.2773       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  48.697  48.697      13.145       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.577   1.577      1.5450       1.000\n",
      " 5 Soft-sphere overlap restraints     :     103       0      0   0.001   0.001     0.56307E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.072   0.072     0.19408       1.000\n",
      "10 Distance restraints 2 (N-O)        :      45       0      0   0.181   0.181     0.66191       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.857   2.857     0.86637       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  88.518  88.518      1.3912       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  43.034  43.034     0.41489       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  11.975  11.975     0.97254       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 108.383 108.383      2.5898       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      10       0      0   0.197   0.197     0.73569E-02   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      0  22.564 125.234      7.3787       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2cji_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1455.5223\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -118.39 -125.20   13.28    0.47  -63.00  179.07   21.07\n",
      "    1        235R 235R N   CA   1853 1854  129.19  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   92.32   78.70   35.30    0.54   82.20  153.16    7.94\n",
      "    2        236G 236G N   CA   1864 1865  161.33 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -107.68 -108.50   20.48    1.12  -63.50  171.61   26.54\n",
      "    3        237L 237L N   CA   1868 1869  152.97  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -119.32 -118.00   21.89    1.04  -62.90  168.00   25.18\n",
      "    4        239K 239K N   CA   1883 1884  160.96  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -82.58  -68.20   21.88    1.38  -62.50  158.59   27.02\n",
      "    5        240A 240A N   CA   1892 1893  161.79  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    3   14   25   56   92   88  123  180  253  264\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2cji_MODLL.B99990001.pdb        34.63573\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2d1j\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25922    24219\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1771 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.5352\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.7220\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24219     499\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      232\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         206.2226\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.016   0.016      5.6516       1.000\n",
      " 2 Bond angle potential               :     102       2      4   4.783   4.783      42.983       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      2  51.453  51.453      17.647       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.646   3.646      8.9834       1.000\n",
      " 5 Soft-sphere overlap restraints     :     232       0      3   0.023   0.023      14.303       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      64       0      0   0.557   0.557      18.163       1.000\n",
      "10 Distance restraints 2 (N-O)        :      65       0      1   0.619   0.619      12.571       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      2  16.208  16.208      34.078       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0 102.982 102.982      2.6822       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  59.069  59.069      3.6710       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  89.405  89.405      3.5056       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  32.146  32.146      1.5284       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.599   0.599     0.18187       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      4  53.746 133.691      36.653       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      58       0      0   0.740   0.740      3.6203       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2d1j_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2505.6362\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4477 232M 232M N   CA   1829 1830  123.30  107.00   16.30    4.69  107.00   16.30    4.69\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   79.33   56.60  128.70   10.96  -62.90 -150.42   25.15\n",
      "    1        233K 233K N   CA   1837 1838  165.27   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -77.45  -78.10   31.93    1.39  -63.20  136.92   18.51\n",
      "    2        234T 234T N   CA   1846 1847 -178.28  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853  -76.86  -72.10   14.51    1.01  -63.00  163.89   23.08\n",
      "    3        235R 235R N   CA   1853 1854  155.60  141.90                  -41.10\n",
      "    4   6831 236G 237L C   N    1866 1868  -64.25  -70.70    8.88    0.57  -63.50  176.70   24.49\n",
      "    4        237L 237L N   CA   1868 1869  135.49  141.60                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883 -160.81 -118.00   52.82    1.78  -62.90  178.42   28.97\n",
      "    5        239K 239K N   CA   1883 1884  170.04  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -93.14  -68.20   75.02    5.19  -62.50  107.50   19.25\n",
      "    6        240A 240A N   CA   1892 1893 -143.94  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   174  176   2.089\n",
      "  serious non-bonded atom clash:   174  414   2.222\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    1    1    3   18   25   43   72   79  125  170  207  247  280\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2d1j_MODLL.B99990001.pdb       206.22256\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2ei6\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25931    24228\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1779 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.1444\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.5152\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24228     500\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      231\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         210.9202\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      2   0.021   0.021      9.5961       1.000\n",
      " 2 Bond angle potential               :     102       4      7   6.023   6.023      69.263       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  53.835  53.835      19.662       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   2.787   2.787      4.9318       1.000\n",
      " 5 Soft-sphere overlap restraints     :     231       0      3   0.025   0.025      17.655       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       0      4   0.674   0.674      27.807       1.000\n",
      "10 Distance restraints 2 (N-O)        :      67       0      1   0.711   0.711      17.678       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       0      0   5.337   5.337      3.6948       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  88.679  88.679      3.0643       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      1  45.211  45.211      1.9518       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 102.240 102.240      4.5855       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  59.802  59.802      1.8614       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      23       0      0   0.580   0.580     0.14606       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      5  46.300 140.327      25.126       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      59       0      0   0.760   0.760      3.8969       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2ei6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2532.4065\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  147.97  120.00   27.97    6.36  120.00   27.97    6.36\n",
      "    2   4477 232M 232M N   CA   1829 1830  127.73  107.00   20.73    5.96  107.00   20.73    5.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   75.59   56.60  108.89    9.25  -62.90 -138.10   26.11\n",
      "    1        233K 233K N   CA   1837 1838  145.82   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -76.04  -78.10   34.06    1.52  -63.20  134.71   18.13\n",
      "    2        234T 234T N   CA   1846 1847 -176.20  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853  -64.76  -72.10   10.09    0.88  -63.00  170.08   23.09\n",
      "    3        235R 235R N   CA   1853 1854  148.83  141.90                  -41.10\n",
      "    4   6831 236G 237L C   N    1866 1868  -66.19  -70.70    4.93    0.49  -63.50  175.22   24.48\n",
      "    4        237L 237L N   CA   1868 1869  143.60  141.60                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883 -172.39 -118.00   63.49    2.11  -62.90 -176.42   30.28\n",
      "    5        239K 239K N   CA   1883 1884  171.85  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892 -130.52 -134.00   39.49    2.31  -62.50  149.18   28.09\n",
      "    6        240A 240A N   CA   1892 1893 -173.66  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   341  590   1.795\n",
      "  serious non-bonded atom clash:   342  589   2.238\n",
      "  serious non-bonded atom clash:   342  590   0.838\n",
      "  serious non-bonded atom clash:  1067 1658   2.283\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    2    1    2   15   18   67   65   91  115  154  204  235  297\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2ei6_MODLL.B99990001.pdb       210.92018\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2ei7\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25983    24280\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1788 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       94.5136\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24280     494\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      263\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         186.8354\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.011   0.011      2.6003       1.000\n",
      " 2 Bond angle potential               :     102       1      3   4.278   4.278      35.487       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  60.120  60.120      21.975       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.443   3.443      7.7761       1.000\n",
      " 5 Soft-sphere overlap restraints     :     263       0      1   0.015   0.015      7.1854       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      63       0      1   0.526   0.526      15.808       1.000\n",
      "10 Distance restraints 2 (N-O)        :      64       0      1   0.742   0.742      20.580       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  18.270  18.270      43.298       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  37.909  37.909      1.5387       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  66.909  66.909      1.4549       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  16.328  16.328      1.4494       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 121.620 121.620      1.8818       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.579   0.579     0.16680       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      3  38.164 134.576      22.290       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      57       0      0   0.719   0.719      3.3436       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2ei7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2348.3359\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4477 232M 232M N   CA   1829 1830  123.92  107.00   16.92    4.87  107.00   16.92    4.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837  -80.08  -70.20   13.29    1.11  -62.90  173.16   21.60\n",
      "    1        233K 233K N   CA   1837 1838  131.51  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -128.94 -124.80    8.91    0.28  -63.20  179.02   27.29\n",
      "    2        234T 234T N   CA   1846 1847  151.39  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -104.23 -125.20   40.54    1.41  -63.00  152.68   18.22\n",
      "    3        235R 235R N   CA   1853 1854  105.91  140.60                  -41.10\n",
      "    4   6831 236G 237L C   N    1866 1868 -128.56 -108.50   36.92    1.74  -63.50  168.38   27.41\n",
      "    4        237L 237L N   CA   1868 1869  163.50  132.50                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883  -66.77  -70.20    3.75    0.30  -62.90  177.34   23.24\n",
      "    5        239K 239K N   CA   1883 1884  141.90  140.40                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  129.91 -134.00   98.30    2.55  -62.50 -134.18   45.38\n",
      "    6        240A 240A N   CA   1892 1893  167.75  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   145  156   2.240\n",
      "  serious non-bonded atom clash:   151  156   1.953\n",
      "  serious non-bonded atom clash:   341  590   1.925\n",
      "  serious non-bonded atom clash:   342  589   1.973\n",
      "  serious non-bonded atom clash:   342  590   0.642\n",
      "  serious non-bonded atom clash:   505  808   2.254\n",
      "  serious non-bonded atom clash:  1226 1247   2.054\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    0    2    2    9   14   22   39   64   98  113  186  202  251  297\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2ei7_MODLL.B99990001.pdb       186.83536\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2ei8\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26209    24506\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24506     506\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      251\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         206.8483\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.013   0.013      3.5117       1.000\n",
      " 2 Bond angle potential               :     102       2      4   4.457   4.457      38.555       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      1  48.893  48.893      15.905       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.364   3.364      7.4897       1.000\n",
      " 5 Soft-sphere overlap restraints     :     251       0      2   0.021   0.021      12.773       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       0      2   0.612   0.612      23.268       1.000\n",
      "10 Distance restraints 2 (N-O)        :      66       0      1   0.705   0.705      19.584       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      2  18.192  18.192      42.930       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  80.565  80.565      3.6459       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  68.804  68.804      2.3613       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  79.213  79.213      3.9806       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  92.437  92.437      2.1514       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.558   0.558     0.16830       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      3  44.701 123.328      28.091       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      61       0      0   0.574   0.574      2.4339       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2ei8_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2465.0503\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  139.96  120.00   19.96    4.54  120.00   19.96    4.54\n",
      "    2   4477 232M 232M N   CA   1829 1830  127.91  107.00   20.91    6.01  107.00   20.91    6.01\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837  -98.21 -118.00   22.99    1.09  -62.90  172.06   24.41\n",
      "    1        233K 233K N   CA   1837 1838  150.80  139.10                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -160.90 -124.80   57.73    1.80  -63.20  162.11   26.83\n",
      "    2        234T 234T N   CA   1846 1847 -171.46  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -111.89 -125.20   27.17    1.50  -63.00  162.16   25.28\n",
      "    3        235R 235R N   CA   1853 1854  164.28  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864   76.06   78.70   30.92    0.77   82.20  143.92    7.30\n",
      "    4        236G 236G N   CA   1864 1865 -135.29 -166.10                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -63.15  -70.70    7.99    0.79  -63.50  174.60   24.22\n",
      "    5        237L 237L N   CA   1868 1869  144.20  141.60                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883 -150.83 -118.00   96.10    4.12  -62.90  125.66   21.34\n",
      "    6        239K 239K N   CA   1883 1884 -130.58  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892 -135.08 -134.00   69.37    3.92  -62.50  125.80   24.44\n",
      "    7        240A 240A N   CA   1892 1893 -143.64  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1067 1658   2.262\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    1    1   17   25   51   73   90  114  167  219  248  300\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2ei8_MODLL.B99990001.pdb       206.84830\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2fzz\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26333    24630\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.6577\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24630     403\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       91\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          49.9114\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.37218       1.000\n",
      " 2 Bond angle potential               :      94       0      0   2.288   2.288      8.9666       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  48.803  48.803      13.634       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.488   1.488      1.2250       1.000\n",
      " 5 Soft-sphere overlap restraints     :      91       0      0   0.002   0.002     0.26368E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      48       0      0   0.120   0.120     0.61882       1.000\n",
      "10 Distance restraints 2 (N-O)        :      51       0      0   0.210   0.210      1.6677       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   5.479   5.479      3.1858       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  83.080  83.080      2.0339       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  49.479  49.479    -0.27001       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  80.431  80.431      2.4634       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 143.643 143.643      4.4522       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.470   0.470     0.16269       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      3  28.575 115.701      11.239       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      12       0      0   0.353   0.353     0.13374       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2fzz_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1410.7120\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864  158.70 -167.20   41.14    1.39   82.20  162.24   11.33\n",
      "    1        236G 236G N   CA   1864 1865  151.57  174.60                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -79.70  -70.70   12.88    1.28  -63.50  174.35   23.33\n",
      "    2        237L 237L N   CA   1868 1869  132.40  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -71.11  -70.20   11.68    0.87  -62.90  169.76   21.61\n",
      "    3        239K 239K N   CA   1883 1884  128.76  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -73.23  -68.20    9.10    0.58  -62.50  166.56   27.83\n",
      "    4        240A 240A N   CA   1892 1893  152.89  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    3    6   40   65   82  124  162  212  259  295\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2fzz_MODLL.B99990001.pdb        49.91135\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2g00\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26788    25085\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1840 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25085     414\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       83\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          49.5949\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.15312       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.879   1.879      6.3427       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  48.425  48.425      14.330       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.544   1.544      1.4455       1.000\n",
      " 5 Soft-sphere overlap restraints     :      83       0      0   0.002   0.002     0.23132E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      51       0      0   0.123   0.123     0.57858       1.000\n",
      "10 Distance restraints 2 (N-O)        :      44       0      1   0.457   0.457      4.3785       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.046   4.046      1.7371       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  38.737  38.737     0.32426       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  89.322  89.322      1.7665       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  15.083  15.083     0.93626       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  67.431  67.431      2.3420       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      35       0      0   0.353   0.353     0.12835       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  29.398 139.542      15.074       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      19       0      0   0.130   0.130     0.34445E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2g00_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1590.5107\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -151.47 -125.20   44.08    1.45  -63.00  168.08   28.61\n",
      "    1        235R 235R N   CA   1853 1854  175.99  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -114.31  -80.20   41.51    3.04   82.20 -143.49   16.78\n",
      "    2        236G 236G N   CA   1864 1865  150.45  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -121.71 -108.50   14.96    0.66  -63.50 -171.51   29.77\n",
      "    3        237L 237L N   CA   1868 1869  139.52  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -75.38  -70.20   13.70    0.87  -62.90  166.59   22.34\n",
      "    4        239K 239K N   CA   1883 1884  153.08  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -68.41  -68.20    8.07    0.64  -62.50  165.84   27.47\n",
      "    5        240A 240A N   CA   1892 1893  153.37  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    9   24   44   68  101  133  171  213  233  326\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2g00_MODLL.B99990001.pdb        49.59486\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDAGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTRGLPK\n",
      "Modelando proteína 2gd4\n",
      "{'num_gaps': 1, 'gap_lengths': [2], 'gap_list': [[239, 240]], 'gap_window': [[237, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26418    24715\n",
      "29 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1846 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      29\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24715     269\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      121\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         202.1493\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      31       0      0   0.019   0.019      3.9568       1.000\n",
      " 2 Bond angle potential               :      44       2      8   8.827   8.827      60.759       1.000\n",
      " 3 Stereochemical cosine torsion poten:      21       0      3  71.325  71.325      15.088       1.000\n",
      " 4 Stereochemical improper torsion pot:      10       1      1   7.604   7.604      19.203       1.000\n",
      " 5 Soft-sphere overlap restraints     :     121       0      2   0.029   0.029      11.850       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      49       0      0   0.526   0.526      10.234       1.000\n",
      "10 Distance restraints 2 (N-O)        :      53       0      0   0.613   0.613      11.278       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       5       3      3  20.793  20.793      25.493       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       3       0      0  40.143  40.143      2.1304       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       3       0      0  79.582  79.582      2.9191       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       2       0      0  19.355  19.355      1.6189       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   5.427   5.427     0.24923       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      30       0      0   0.918   0.918     0.50930       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       5       4      3  59.990 127.565      36.212       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      12       0      0   0.722   0.722     0.64911       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2gd4_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1367.3574\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6584 236G 236G C   CA   1866 1865   21.25    0.00   21.25    7.48    0.00   21.25    7.48\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7071 237L 237L CA  C    1869 1874 -150.45 -180.00   29.55    5.91 -180.00   29.55    5.91\n",
      "    2   7072 238P 238P CA  C    1877 1881 -156.49 -180.00   23.51    4.70 -180.00   23.51    4.70\n",
      "    3   7073 239K 239K CA  C    1884 1890 -154.01 -180.00   25.99    5.20 -180.00   25.99    5.20\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868 -113.18  -63.50   50.07    8.32  -63.50   50.07    8.32\n",
      "    1        237L 237L N   CA   1868 1869  -34.99  -41.20                  -41.20\n",
      "    2   6832 237L 238P C   N    1874 1876  -57.73  -58.70   14.26    1.10  -64.50  168.21   12.91\n",
      "    2        238P 238P N   CA   1876 1877  -44.72  -30.50                  147.20\n",
      "    3   6833 238P 239K C   N    1881 1883   48.01   56.60   12.54    0.77  -62.90  141.91   24.53\n",
      "    3        239K 239K N   CA   1883 1884   47.74   38.60                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892   26.30  -68.20  106.55    7.29  -62.50  163.24   31.46\n",
      "    4        240A 240A N   CA   1892 1893   96.08  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    61  823   2.070\n",
      "  serious non-bonded atom clash:   304  700   2.294\n",
      "  serious non-bonded atom clash:   316 1458   2.115\n",
      "  serious non-bonded atom clash:   501  506   2.219\n",
      "  serious non-bonded atom clash:  1168 1370   2.300\n",
      "  serious non-bonded atom clash:  1845 1850   1.950\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    1    3    3    5   25   47   79  109  129  162  164  211  226  284\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2gd4_MODLL.B99990001.pdb       202.14926\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2h9e\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26464    24761\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1819 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.0158\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       94.1364\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24761     504\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      262\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         192.0686\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.017   0.017      5.8015       1.000\n",
      " 2 Bond angle potential               :     102       3      4   6.028   6.028      62.538       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      1  55.918  55.918      16.795       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   2.056   2.056      2.9198       1.000\n",
      " 5 Soft-sphere overlap restraints     :     262       0      0   0.016   0.016      8.4093       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      69       0      4   0.692   0.692      30.465       1.000\n",
      "10 Distance restraints 2 (N-O)        :      68       0      3   0.762   0.762      25.109       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       0      0   3.341   3.341      1.4481       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      1  89.506  89.506      4.8684       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  96.122  96.122      4.1291       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  62.918  62.918      3.2805       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  60.722  60.722      1.9579       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      36       0      0   0.849   0.849     0.90483       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       4      3  46.713 115.553      21.273       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      46       0      0   0.790   0.790      2.1687       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2h9e_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1996.2513\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  143.83  120.00   23.83    5.42  120.00   23.83    5.42\n",
      "    2   4477 232M 232M N   CA   1829 1830  126.70  107.00   19.70    5.66  107.00   19.70    5.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837  -59.43  -70.20   17.55    1.06  -62.90  167.37   21.93\n",
      "    1        233K 233K N   CA   1837 1838  126.54  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -66.12  -78.10   30.79    1.02  -63.20  163.56   20.78\n",
      "    2        234T 234T N   CA   1846 1847  121.43  149.80                  -42.10\n",
      "    3   6833 238P 239K C   N    1881 1883  -96.87 -118.00   21.28    0.85  -62.90 -179.17   25.46\n",
      "    3        239K 239K N   CA   1883 1884  141.59  139.10                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892   51.33  -68.20  130.66   12.53  -62.50  166.14   25.53\n",
      "    4        240A 240A N   CA   1892 1893 -161.93  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    65  823   2.263\n",
      "  serious non-bonded atom clash:   157  373   2.181\n",
      "  serious non-bonded atom clash:  1218 1309   2.166\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    2    1    3    6   12   27   36   67   80  146  170  211  228  303\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2h9e_MODLL.B99990001.pdb       192.06857\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2j2u\n",
      "{'num_gaps': 2, 'gap_lengths': [4, 8], 'gap_list': [[62, 65], [233, 240]], 'gap_window': [[60, 67], [231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     66     1   76    81      E     A    8.731\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25889    24186\n",
      "132 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     132\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24186     989\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      345\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         304.3713\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     135       0      0   0.012   0.012      5.2272       1.000\n",
      " 2 Bond angle potential               :     182       1      4   4.491   4.491      62.084       1.000\n",
      " 3 Stereochemical cosine torsion poten:      63       0      8  65.962  65.962      40.996       1.000\n",
      " 4 Stereochemical improper torsion pot:      44       0      1   2.601   2.601      6.2482       1.000\n",
      " 5 Soft-sphere overlap restraints     :     345       0      0   0.011   0.011      5.1423       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     159       1      6   0.945   0.945      60.977       1.000\n",
      "10 Distance restraints 2 (N-O)        :     155       2     10   1.184   1.184      64.802       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       0      2   7.386   7.386      12.868       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      13       0      0  74.212  74.212      3.2838       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  32.450  32.450     0.90796       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  98.048  98.048      9.0545       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  73.701  73.701      1.3706       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      81       0      0   0.349   0.349     0.25008       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      21       9      3  34.777 114.530      26.332       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      92       0      0   0.732   0.732      4.8265       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2j2u_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3458.1616\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14365  66E 141T N   O     506 1115   12.28    9.23    3.06    5.05    9.23    3.06    5.05\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6656  61Q  62E C   N     478  480  -72.53  -69.30   15.85    1.32  -63.60  167.52   21.86\n",
      "    1         62E  62E N   CA    480  481  126.98  142.50                  -40.30\n",
      "    2   6657  62E  63E C   N     487  489 -105.71 -117.80   25.73    1.00  -63.60  160.02   19.34\n",
      "    2         63E  63E N   CA    489  490  114.08  136.80                  -40.30\n",
      "    3   6659  64G  65G C   N     500  502   93.43   78.70   55.17    1.00   82.20  132.70    7.00\n",
      "    3         65G  65G N   CA    502  503  140.73 -166.10                    8.50\n",
      "    4   6827 232M 233K C   N    1835 1837   -8.72  -70.20   76.00    4.61  -62.90  146.88   22.31\n",
      "    4        233K 233K N   CA   1837 1838   95.72  140.40                  -40.80\n",
      "    5   6828 233K 234T C   N    1844 1846 -173.45 -124.80   76.06    2.38  -63.20  160.00   27.06\n",
      "    5        234T 234T N   CA   1846 1847 -158.04  143.50                  -42.10\n",
      "    6   6829 234T 235R C   N    1851 1853 -108.23 -125.20   18.47    0.58  -63.00 -179.82   21.67\n",
      "    6        235R 235R N   CA   1853 1854  133.31  140.60                  -41.10\n",
      "    7   6831 236G 237L C   N    1866 1868  -72.74  -70.70   24.54    1.78  -63.50  153.02   21.79\n",
      "    7        237L 237L N   CA   1868 1869  166.05  141.60                  -41.20\n",
      "    8   6833 238P 239K C   N    1881 1883   70.80   56.60   23.94    1.31  -62.90  146.59   25.59\n",
      "    8        239K 239K N   CA   1883 1884   19.32   38.60                  -40.80\n",
      "    9   6834 239K 240A C   N    1890 1892 -160.52 -134.00   30.22    1.55  -62.50 -160.79   29.84\n",
      "    9        240A 240A N   CA   1892 1893  132.52  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    8   23   40   69   73  120  154  177  259  295\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2j2u_MODLL.B99990001.pdb       304.37131\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2j34\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25504    23801\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.3568\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    23801     368\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      106\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          50.5542\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.30206       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.874   1.874      6.4435       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.451  49.451      13.765       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.771   1.771      1.9990       1.000\n",
      " 5 Soft-sphere overlap restraints     :     106       0      0   0.000   0.000     0.30494E-03   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.178   0.178      1.1086       1.000\n",
      "10 Distance restraints 2 (N-O)        :      41       0      0   0.145   0.145     0.62464       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.630   4.630      2.2756       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  95.433  95.433      1.9390       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  81.520  81.520      2.4886       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 118.576 118.576      5.1375       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 113.038 113.038      3.6883       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      17       0      0   0.416   0.416     0.84674E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  22.942 130.888      10.163       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   2.594   2.594     0.53419       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2j34_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1902.2212\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -69.91  -72.10    4.97    0.42  -63.00  172.68   23.79\n",
      "    1        235R 235R N   CA   1853 1854  146.36  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -61.65  -62.40    4.36    0.69   82.20  150.85   11.40\n",
      "    2        236G 236G N   CA   1864 1865  -36.91  -41.20                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -59.01  -70.70   13.70    0.97  -63.50  175.71   24.66\n",
      "    3        237L 237L N   CA   1868 1869  134.45  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -71.64  -70.20    1.72    0.14  -62.90  179.95   23.86\n",
      "    4        239K 239K N   CA   1883 1884  139.46  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -158.51 -134.00   27.62    0.63  -62.50 -173.95   35.60\n",
      "    5        240A 240A N   CA   1892 1893  159.74  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    1    4   28   52   68   99  118  153  238  272\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2j34_MODLL.B99990001.pdb        50.55418\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2j38\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25833    24130\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1837 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.8371\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.0315\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24130     376\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       90\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          34.2524\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.11733       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.734   1.734      5.6722       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.028  46.028      12.430       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.564   1.564      1.4406       1.000\n",
      " 5 Soft-sphere overlap restraints     :      90       0      0   0.001   0.001     0.13579E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.152   0.152     0.77836       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.199   0.199     0.85913       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.615   2.615     0.72586       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  91.220  91.220      1.7592       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  73.826  73.826      1.2795       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  12.267  12.267      1.3266       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  88.632  88.632      1.3530       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      19       0      0   0.672   0.672     0.24593       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      0  20.956 130.621      6.2509       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2j38_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1536.6097\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -120.21 -125.20    8.16    0.45  -63.00 -178.89   28.42\n",
      "    1        235R 235R N   CA   1853 1854  147.06  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  161.74 -167.20   31.21    0.67   82.20 -178.59   12.44\n",
      "    2        236G 236G N   CA   1864 1865  171.55  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -106.89 -108.50    5.72    0.29  -63.50  173.72   22.04\n",
      "    3        237L 237L N   CA   1868 1869  127.01  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -109.14 -118.00   13.37    0.68  -62.90  176.26   25.63\n",
      "    4        239K 239K N   CA   1883 1884  149.11  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -80.49  -68.20   22.33    1.43  -62.50  156.20   26.52\n",
      "    5        240A 240A N   CA   1892 1893  163.95  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    4   16   37   51   79   94  133  188  218  287\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2j38_MODLL.B99990001.pdb        34.25243\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2j4i\n",
      "{'num_gaps': 2, 'gap_lengths': [2, 7], 'gap_list': [[63, 64], [235, 241]], 'gap_window': [[61, 66], [233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26273    24570\n",
      "114 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.8463\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.6288\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     114\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24570     818\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      170\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          80.9300\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     116       0      0   0.004   0.004     0.56478       1.000\n",
      " 2 Bond angle potential               :     155       0      0   1.628   1.628      7.7265       1.000\n",
      " 3 Stereochemical cosine torsion poten:      56       0      1  48.428  48.428      18.767       1.000\n",
      " 4 Stereochemical improper torsion pot:      37       0      0   0.902   0.902     0.70612       1.000\n",
      " 5 Soft-sphere overlap restraints     :     170       0      0   0.001   0.001     0.36368E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     140       0      0   0.209   0.209      3.2097       1.000\n",
      "10 Distance restraints 2 (N-O)        :     140       0      0   0.355   0.355      6.7156       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      16       0      0   3.059   3.059     0.90681       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  75.459  75.459      1.9482       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      10       0      0  35.507  35.507      1.3020       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0 100.723 100.723      8.2442       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  68.849  68.849      1.7947       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      70       0      0   1.061   1.061      1.5033       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17       8      4  38.301 121.862      26.167       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      37       0      0   0.692   0.692      1.3370       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2j4i_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2221.7188\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6657  62E  63E C   N     487  489  -65.82  -69.30    3.65    0.31  -63.60  176.11   23.60\n",
      "    1         63E  63E N   CA    489  490  143.60  142.50                  -40.30\n",
      "    2   6658  63E  64G C   N     496  498  -66.53  -80.20   28.41    0.66   82.20 -155.27    6.61\n",
      "    2         64G  64G N   CA    498  499  149.19  174.10                    8.50\n",
      "    3   6827 232M 233K C   N    1835 1837  -64.14  -70.20  114.03    8.40 -118.00  127.13    6.59\n",
      "    3        233K 233K N   CA   1837 1838 -105.74  140.40                  139.10\n",
      "    4   6829 234T 235R C   N    1851 1853  -98.76 -125.20   26.50    1.02  -63.00 -176.61   22.58\n",
      "    4        235R 235R N   CA   1853 1854  138.77  140.60                  -41.10\n",
      "    5   6830 235R 236G C   N    1862 1864   66.26   78.70   12.60    0.58   82.20  173.37    9.21\n",
      "    5        236G 236G N   CA   1864 1865 -164.13 -166.10                    8.50\n",
      "    6   6831 236G 237L C   N    1866 1868  -65.63  -70.70    5.33    0.53  -63.50  175.55   24.50\n",
      "    6        237L 237L N   CA   1868 1869  143.26  141.60                  -41.20\n",
      "    7   6833 238P 239K C   N    1881 1883  -72.12  -70.20    3.93    0.24  -62.90  175.62   23.32\n",
      "    7        239K 239K N   CA   1883 1884  143.82  140.40                  -40.80\n",
      "    8   6834 239K 240A C   N    1890 1892  -74.13  -68.20   13.01    0.85  -62.50  162.64   27.24\n",
      "    8        240A 240A N   CA   1892 1893  156.88  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    5    9   45   69   81  108  154  207  266  276\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2j4i_MODLL.B99990001.pdb        80.93004\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2j94\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26390    24687\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.0890\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24687     375\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      180\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          89.8169\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.006   0.006     0.98784       1.000\n",
      " 2 Bond angle potential               :      94       0      2   2.676   2.676      11.830       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  54.959  54.959      15.935       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   2.854   2.854      2.3341       1.000\n",
      " 5 Soft-sphere overlap restraints     :     180       1      5   0.039   0.039      31.843       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.090   0.090     0.30010       1.000\n",
      "10 Distance restraints 2 (N-O)        :      45       0      0   0.147   0.147     0.68898       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.772   3.772      1.5105       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  81.685  81.685      2.4923       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  78.698  78.698      1.6668       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      1  90.515  90.515      3.7005       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  46.709  46.709      1.6351       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      18       0      0   0.339   0.339     0.58384E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      3  29.599 129.960      13.606       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       4       0      0   1.909   1.909      1.2282       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2j94_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1773.6797\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853   75.57   57.30   28.64    1.68  -63.00  149.85   27.13\n",
      "    1        235R 235R N   CA   1853 1854   15.94   38.00                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -165.13 -167.20   17.40    0.65   82.20 -173.34   13.84\n",
      "    2        236G 236G N   CA   1864 1865  157.32  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -77.66  -70.70    7.00    0.61  -63.50  177.04   25.42\n",
      "    3        237L 237L N   CA   1868 1869  142.33  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -91.37  -70.20   27.36    1.65  -62.90  163.96   22.95\n",
      "    4        239K 239K N   CA   1883 1884  157.73  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -60.20  -68.20    8.07    0.65  -62.50  174.89   28.55\n",
      "    5        240A 240A N   CA   1892 1893  144.23  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3    4   23   40   61   84  108  150  193  247  289\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2j94_MODLL.B99990001.pdb        89.81691\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2j95\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25763    24060\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1843 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24060     364\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      101\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          42.8100\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.10080       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.512   1.512      4.4122       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.641  46.641      11.841       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.539   2.539      4.5811       1.000\n",
      " 5 Soft-sphere overlap restraints     :     101       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      40       0      0   0.253   0.253      2.1934       1.000\n",
      "10 Distance restraints 2 (N-O)        :      41       0      0   0.277   0.277      1.6536       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.207   3.207      1.0915       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  87.867  87.867      2.4595       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  46.913  46.913    -0.80469       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  47.218  47.218      2.2050       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  74.906  74.906      2.8630       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      18       0      0   0.304   0.304     0.51020E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  19.818 128.255      10.163       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2j95_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1536.1230\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -115.32 -125.20   13.02    0.71  -63.00  177.71   27.62\n",
      "    1        235R 235R N   CA   1853 1854  149.07  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   72.19   78.70   29.72    0.59   82.20  145.95    7.59\n",
      "    2        236G 236G N   CA   1864 1865 -137.10 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -62.39  -70.70    8.52    0.83  -63.50  175.32   24.28\n",
      "    3        237L 237L N   CA   1868 1869  143.48  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -95.05 -118.00   23.07    0.91  -62.90 -179.34   25.33\n",
      "    4        239K 239K N   CA   1883 1884  141.43  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -73.17  -68.20    6.83    0.43  -62.50  169.45   28.31\n",
      "    5        240A 240A N   CA   1892 1893  149.99  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    3   11   28   48   71  106  128  187  228  282\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2j95_MODLL.B99990001.pdb        42.80996\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2jkh\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26308    24605\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.0044\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.1600\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.2117\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24605     510\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      270\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         217.5189\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.019   0.019      7.6161       1.000\n",
      " 2 Bond angle potential               :     102       2      5   4.937   4.937      46.715       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      2  51.693  51.693      17.567       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.439   3.439      8.4768       1.000\n",
      " 5 Soft-sphere overlap restraints     :     270       1      3   0.026   0.026      21.011       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      68       0      0   0.527   0.527      18.395       1.000\n",
      "10 Distance restraints 2 (N-O)        :      69       0      2   0.630   0.630      16.456       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  15.003  15.003      29.198       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  80.114  80.114      4.1677       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      1  44.838  44.838      1.8117       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  85.021  85.021      4.9185       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 105.388 105.388      3.9979       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.504   0.504     0.13644       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      4  43.470 110.986      35.463       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      60       0      0   0.469   0.469      1.5887       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2jkh_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2188.2695\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4477 232M 232M N   CA   1829 1830  124.69  107.00   17.69    5.09  107.00   17.69    5.09\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6826 231S 232M C   N    1827 1829  -99.97  -63.40   36.88    6.51  -63.40   36.88    6.51\n",
      "    1        232M 232M N   CA   1829 1830  -35.72  -40.50                  -40.50\n",
      "    2   6827 232M 233K C   N    1835 1837   63.36   56.60   84.22    6.54  -62.90 -153.54   34.21\n",
      "    2        233K 233K N   CA   1837 1838  122.55   38.60                  -40.80\n",
      "    3   6829 234T 235R C   N    1851 1853  -94.06  -72.10   30.55    1.98  -63.00  158.83   23.60\n",
      "    3        235R 235R N   CA   1853 1854  163.14  141.90                  -41.10\n",
      "    4   6832 237L 238P C   N    1874 1876  -63.21  -58.70   19.02    1.25  -64.50  159.22   11.81\n",
      "    4        238P 238P N   CA   1876 1877  -12.02  -30.50                  147.20\n",
      "    5   6833 238P 239K C   N    1881 1883 -125.30 -118.00   27.78    1.23  -62.90  165.51   25.23\n",
      "    5        239K 239K N   CA   1883 1884  165.91  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -90.35  -68.20   81.38    5.78  -62.50   99.47   17.78\n",
      "    6        240A 240A N   CA   1892 1893 -136.39  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   10   15   40   84   90  105  156  197  283  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2jkh_MODLL.B99990001.pdb       217.51886\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2p16\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26360    24657\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1864 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.9970\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24657     387\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       92\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          37.1014\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.99272E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.601   1.601      4.8292       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.830  46.830      11.313       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.830   1.830      2.1204       1.000\n",
      " 5 Soft-sphere overlap restraints     :      92       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.121   0.121     0.55454       1.000\n",
      "10 Distance restraints 2 (N-O)        :      44       0      0   0.320   0.320      1.7134       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.272   3.272      1.1364       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  74.326  74.326     0.28539       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  62.389  62.389      1.2785       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  59.630  59.630      1.7505       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  98.037  98.037      3.0147       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.337   0.337     0.64239E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  17.794 115.645      8.8344       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      10       0      0   0.352   0.352     0.10715       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2p16_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1405.6993\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -139.14 -125.20   21.82    0.70  -63.00  178.57   29.32\n",
      "    1        235R 235R N   CA   1853 1854  157.38  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   82.66   78.70   16.55    0.32   82.20  169.33    8.24\n",
      "    2        236G 236G N   CA   1864 1865  177.83 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -100.93 -108.50   22.70    1.28  -63.50  169.09   25.76\n",
      "    3        237L 237L N   CA   1868 1869  153.90  132.50                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892  -73.24  -68.20    7.74    0.49  -62.50  168.26   28.11\n",
      "    4        240A 240A N   CA   1892 1893  151.18  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    8   16   35   57  100  137  159  193  248  266\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2p16_MODLL.B99990001.pdb        37.10142\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTR\n",
      "Modelando proteína 2p3f\n",
      "{'num_gaps': 1, 'gap_lengths': [6], 'gap_list': [[236, 241]], 'gap_window': [[234, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    27113    25410\n",
      "61 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1832 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      61\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    25410     349\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       54\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          52.2199\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      62       0      0   0.003   0.003     0.18422       1.000\n",
      " 2 Bond angle potential               :      83       0      0   2.144   2.144      7.4106       1.000\n",
      " 3 Stereochemical cosine torsion poten:      36       0      3  55.009  55.009      15.307       1.000\n",
      " 4 Stereochemical improper torsion pot:      19       0      0   1.037   1.037     0.39614       1.000\n",
      " 5 Soft-sphere overlap restraints     :      54       0      0   0.003   0.003     0.46858E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      35       0      0   0.196   0.196     0.88829       1.000\n",
      "10 Distance restraints 2 (N-O)        :      31       0      0   0.333   0.333     0.93468       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       8       0      0   3.229   3.229     0.98351       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  40.312  40.312     0.64787       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  60.501  60.501     0.52763       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  54.944  54.944      1.2203       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  94.175  94.175      3.1001       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      38       0      0   0.492   0.492     0.22731       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       8       4      2  32.981 130.965      19.869       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      11       0      0   0.567   0.567     0.47616       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2p3f_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1149.5583\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864  -82.12  -80.20   13.90    0.45   82.20 -116.74    7.27\n",
      "    1        236G 236G N   CA   1864 1865 -172.13  174.10                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -82.42  -70.70   23.41    1.45  -63.50  158.07   23.08\n",
      "    2        237L 237L N   CA   1868 1869  161.87  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883 -122.35 -118.00   56.50    2.68  -62.90  137.31   21.39\n",
      "    3        239K 239K N   CA   1883 1884 -164.57  139.10                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892 -105.12 -134.00   29.08    1.09  -62.50  174.01   30.77\n",
      "    4        240A 240A N   CA   1892 1893  150.40  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   147  177   2.259\n",
      "  serious non-bonded atom clash:   639  658   2.296\n",
      "  serious non-bonded atom clash:   788  807   2.276\n",
      "  serious non-bonded atom clash:  1318 1732   2.268\n",
      "  serious non-bonded atom clash:  1745 1748   2.243\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    5    3   12   23   45   70  101  127  150  218  231  258  299\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2p3f_MODLL.B99990001.pdb        52.21988\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2p3t\n",
      "{'num_gaps': 2, 'gap_lengths': [2, 8], 'gap_list': [[63, 64], [233, 240]], 'gap_window': [[61, 66], [231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26136    24433\n",
      "118 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1845 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.5316\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     118\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24433     957\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      351\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         252.9209\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     121       0      0   0.011   0.011      4.2160       1.000\n",
      " 2 Bond angle potential               :     163       1      4   4.793   4.793      62.660       1.000\n",
      " 3 Stereochemical cosine torsion poten:      57       0      6  62.856  62.856      31.555       1.000\n",
      " 4 Stereochemical improper torsion pot:      39       0      1   2.644   2.644      5.8277       1.000\n",
      " 5 Soft-sphere overlap restraints     :     351       0      1   0.015   0.015      9.4678       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     164       0      0   0.406   0.406      20.538       1.000\n",
      "10 Distance restraints 2 (N-O)        :     165       0      7   0.646   0.646      37.996       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      18       1      2   8.678   8.678      15.987       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  94.484  94.484      3.5320       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      10       0      0  71.311  71.311      3.3046       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  84.990  84.990      7.3297       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 100.047 100.047      2.3592       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      79       0      0   1.047   1.047      1.5017       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      10      9  44.070 122.157      40.212       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      98       0      0   1.391   1.391      6.4357       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2p3t_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3467.8467\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7074 240A 240A CA  C    1893 1895 -153.13 -180.00   26.87    5.37 -180.00   26.87    5.37\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6656  61Q  62E C   N     478  480  -72.42  -69.30   33.85    2.47  -63.60  143.76   19.66\n",
      "    1         62E  62E N   CA    480  481  176.21  142.50                  -40.30\n",
      "    2   6657  62E  63E C   N     487  489  -71.74  -69.30   17.22    1.23  -63.60  160.36   21.83\n",
      "    2         63E  63E N   CA    489  490  159.54  142.50                  -40.30\n",
      "    3   6659  64G  65G C   N     500  502  -60.64  -62.40   11.22    1.77   82.20  147.97   11.06\n",
      "    3         65G  65G N   CA    502  503  -30.12  -41.20                    8.50\n",
      "    4   6827 232M 233K C   N    1835 1837  -17.53  -70.20   70.78    4.24  -62.90  141.40   21.07\n",
      "    4        233K 233K N   CA   1837 1838   93.12  140.40                  -40.80\n",
      "    5   6828 233K 234T C   N    1844 1846 -150.66 -124.80   72.84    2.43  -63.20  137.66   22.99\n",
      "    5        234T 234T N   CA   1846 1847 -148.41  143.50                  -42.10\n",
      "    6   6830 235R 236G C   N    1862 1864  147.50 -167.20   57.99    2.06   82.20  145.38   10.03\n",
      "    6        236G 236G N   CA   1864 1865  138.39  174.60                    8.50\n",
      "    7   6831 236G 237L C   N    1866 1868 -134.24 -108.50   40.80    1.87  -63.50  170.05   27.99\n",
      "    7        237L 237L N   CA   1868 1869  164.16  132.50                  -41.20\n",
      "    8   6832 237L 238P C   N    1874 1876  -64.55  -58.70    6.26    0.88  -64.50  179.92   13.43\n",
      "    8        238P 238P N   CA   1876 1877  -32.72  -30.50                  147.20\n",
      "    9   6833 238P 239K C   N    1881 1883 -140.94 -118.00   55.82    2.89  -62.90  150.78   17.37\n",
      "    9        239K 239K N   CA   1883 1884   88.21  139.10                  -40.80\n",
      "   10   6834 239K 240A C   N    1890 1892  166.13 -134.00   61.98    2.68  -62.50 -143.67   32.70\n",
      "   10        240A 240A N   CA   1892 1893  130.98  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    5   19   58   73   92  127  170  205  239  316\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2p3t_MODLL.B99990001.pdb       252.92088\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2p3u\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26379    24676\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.1162\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.8783\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24676     498\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      313\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         471.7623\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      2   0.032   0.032      21.877       1.000\n",
      " 2 Bond angle potential               :     102       7     14   8.258   8.258      125.28       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  60.432  60.432      22.934       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      3   3.951   3.951      8.6493       1.000\n",
      " 5 Soft-sphere overlap restraints     :     313       3      5   0.038   0.038      54.037       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       7     14   1.182   1.182      94.132       1.000\n",
      "10 Distance restraints 2 (N-O)        :      67       1     12   1.154   1.154      65.311       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       2      2  15.874  15.874      32.689       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      1  86.288  86.288      3.5073       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      1  73.324  73.324      7.3441       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      2  70.830  70.830      5.2182       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  68.085  68.085      1.8386       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.788   0.788     0.31246       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      5  35.716 132.856      23.328       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      56       0      0   0.998   0.998      5.3073       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2p3u_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2981.5298\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4466 230R 231S C   N    1821 1823  140.54  120.00   20.54    4.67  120.00   20.54    4.67\n",
      "    2   4469 231S 231S N   CA   1823 1824  126.90  107.00   19.90    5.72  107.00   19.90    5.72\n",
      "    3   4488 233K 233K CA  CB   1838 1839  137.38  113.50   23.89    5.86  113.50   23.89    5.86\n",
      "    4   4494 233K 233K C   CA   1844 1838  130.58  108.00   22.58    5.23  108.00   22.58    5.23\n",
      "    5   4563 240A 240A CA  C    1893 1895  132.54  116.50   16.04    4.61  116.50   16.04    4.61\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   84.87   56.60   55.14    2.83  -62.90  151.21   25.90\n",
      "    1        233K 233K N   CA   1837 1838   -8.74   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -79.34  -78.10   23.70    0.97  -63.20  145.33   19.71\n",
      "    2        234T 234T N   CA   1846 1847  173.47  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -141.90 -125.20   52.42    2.12  -63.00  150.89   25.66\n",
      "    3        235R 235R N   CA   1853 1854 -169.71  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -59.83  -62.40    3.64    0.51   82.20  151.35   11.56\n",
      "    4        236G 236G N   CA   1864 1865  -43.78  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -83.91  -70.70   14.78    1.09  -63.50  171.81   25.07\n",
      "    5        237L 237L N   CA   1868 1869  148.21  141.60                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883 -150.08 -118.00   49.91    1.83  -62.90  166.51   26.81\n",
      "    6        239K 239K N   CA   1883 1884  177.33  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  174.10 -134.00   53.96    2.36  -62.50 -147.38   32.03\n",
      "    7        240A 240A N   CA   1892 1893  132.24  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    1    6   21   55   69  110  129  154  199  270  347\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2p3u_MODLL.B99990001.pdb       471.76233\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2p93\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26289    24586\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24586     371\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      110\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          40.4352\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.74718E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.541   1.541      4.3070       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.447  46.447      10.457       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.045   1.045     0.56791       1.000\n",
      " 5 Soft-sphere overlap restraints     :     110       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.073   0.073     0.20973       1.000\n",
      "10 Distance restraints 2 (N-O)        :      42       0      0   0.458   0.458      4.8342       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.832   2.832     0.85110       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  89.147  89.147      1.3685       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  91.263  91.263      1.7515       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  76.995  76.995      3.2713       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  77.244  77.244      2.7531       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      21       0      0   0.630   0.630     0.15233       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       3      1  17.945 101.809      9.8371       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2p93_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1398.9121\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868  -72.04  -70.70   10.31    0.73  -63.50  167.19   23.71\n",
      "    1        237L 237L N   CA   1868 1869  151.83  141.60                  -41.20\n",
      "    2   6833 238P 239K C   N    1881 1883  -85.50  -70.20   15.32    1.13  -62.90 -179.14   24.78\n",
      "    2        239K 239K N   CA   1883 1884  139.76  140.40                  -40.80\n",
      "    3   6834 239K 240A C   N    1890 1892  -84.82  -68.20   17.40    1.29  -62.50  170.14   29.03\n",
      "    3        240A 240A N   CA   1892 1893  150.44  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    5   16   36   73   84  139  172  217  221  299\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2p93_MODLL.B99990001.pdb        40.43519\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2p94\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26301    24598\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24598     388\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       93\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          41.1915\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.23556       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.738   1.738      5.1031       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.535  47.535      11.963       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.675   1.675      1.8603       1.000\n",
      " 5 Soft-sphere overlap restraints     :      93       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.214   0.214      1.7025       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.347   0.347      2.4606       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.864   2.864     0.87061       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  81.471  81.471      1.6954       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  80.707  80.707      1.8735       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  53.101  53.101      3.8379       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  97.886  97.886      2.7584       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.303   0.303     0.60098E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  30.303 116.597      6.3998       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       5       0      0   0.970   0.970     0.37045       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2p94_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1473.1831\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864 -125.17 -167.20   69.70    1.81   82.20 -171.57   14.71\n",
      "    1        236G 236G N   CA   1864 1865  119.00  174.60                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -77.60  -70.70   19.97    1.29  -63.50  159.08   22.92\n",
      "    2        237L 237L N   CA   1868 1869  160.34  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883 -145.98 -118.00   37.87    1.32  -62.90  175.49   27.75\n",
      "    3        239K 239K N   CA   1883 1884  164.62  139.10                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -67.50  -68.20    3.13    0.28  -62.50  170.83   28.24\n",
      "    4        240A 240A N   CA   1892 1893  148.35  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   10   17   35   72  101  117  172  213  233  302\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2p94_MODLL.B99990001.pdb        41.19148\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2p95\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26202    24499\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24499     390\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      164\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         294.6277\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       1      1   0.018   0.018      10.334       1.000\n",
      " 2 Bond angle potential               :      94       1      3   3.720   3.720      22.560       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.846  49.846      18.224       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       1      2   4.788   4.788      10.418       1.000\n",
      " 5 Soft-sphere overlap restraints     :     164      10     12   0.102   0.102      203.26       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.303   0.303      3.4435       1.000\n",
      "10 Distance restraints 2 (N-O)        :      46       0      0   0.382   0.382      3.3884       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.090   4.090      1.7754       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  62.774  62.774     0.59411       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  76.338  76.338     0.38876       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  46.842  46.842      2.2807       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  95.296  95.296      3.0405       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      30       0      0   0.465   0.465     0.18883       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       6      2  20.401 144.288      14.621       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       4       0      0   0.570   0.570     0.11451       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2p95_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2026.9500\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6574 232M 232M C   CA   1835 1830   13.48    0.00   13.48    4.74    0.00   13.48    4.74\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -122.09 -125.20   21.26    1.09  -63.00  168.00   26.76\n",
      "    1        235R 235R N   CA   1853 1854  161.63  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -154.90 -167.20   14.50    0.25   82.20 -159.50   14.92\n",
      "    2        236G 236G N   CA   1864 1865  166.92  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -97.60 -108.50   12.74    0.56  -63.50  170.56   21.97\n",
      "    3        237L 237L N   CA   1868 1869  125.91  132.50                  -41.20\n",
      "    4   6832 237L 238P C   N    1874 1876  -68.60  -58.70   12.71    1.00  -64.50  169.79   12.89\n",
      "    4        238P 238P N   CA   1876 1877  -22.54  -30.50                  147.20\n",
      "    5   6833 238P 239K C   N    1881 1883  -70.30  -70.20    6.75    0.49  -62.90  174.61   22.28\n",
      "    5        239K 239K N   CA   1883 1884  133.65  140.40                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -79.54  -68.20   13.49    0.89  -62.50  167.35   28.29\n",
      "    6        240A 240A N   CA   1892 1893  152.62  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   558 1859   2.165\n",
      "  serious non-bonded atom clash:   562 1861   2.206\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    1    2    1   11   19   38   84   92  113  158  187  225  262\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2p95_MODLL.B99990001.pdb       294.62772\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2phb\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26487    24784\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24784     383\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       92\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          45.3687\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.30966       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.757   1.757      5.4980       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.371  46.371      12.213       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.551   1.551      1.1671       1.000\n",
      " 5 Soft-sphere overlap restraints     :      92       0      0   0.002   0.002     0.61257E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.139   0.139     0.71712       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.368   0.368      2.4832       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.178   3.178      1.0716       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  86.199  86.199      2.3422       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  74.646  74.646      1.3888       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  95.860  95.860      4.7029       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  91.117  91.117      2.7345       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.350   0.350     0.81131E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  26.405 137.641      10.599       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2phb_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1654.1399\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -85.84  -72.10   24.68    1.63  -63.00  158.16   22.94\n",
      "    1        235R 235R N   CA   1853 1854  162.40  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -68.13  -80.20   27.32    0.64   82.20 -153.83    6.69\n",
      "    2        236G 236G N   CA   1864 1865  149.59  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -95.62 -108.50   14.31    0.76  -63.50 -177.22   23.73\n",
      "    3        237L 237L N   CA   1868 1869  138.74  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -59.62  -70.20   10.65    0.75  -62.90 -179.98   23.56\n",
      "    4        239K 239K N   CA   1883 1884  139.19  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -106.40 -134.00   28.08    0.75  -62.50 -177.36   32.25\n",
      "    5        240A 240A N   CA   1892 1893  141.82  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    6   28   47   63   98  123  186  205  248  310\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2phb_MODLL.B99990001.pdb        45.36870\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2pr3\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26364    24661\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24661     383\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       91\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          39.5728\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.13266       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.507   1.507      4.2322       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.016  47.016      12.471       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   1.878   1.878      2.4731       1.000\n",
      " 5 Soft-sphere overlap restraints     :      91       0      0   0.001   0.001     0.55405E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.172   0.172      1.1049       1.000\n",
      "10 Distance restraints 2 (N-O)        :      46       0      0   0.401   0.401      2.5712       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.047   3.047     0.98541       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  45.921  45.921    -0.13456       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  76.230  76.230     0.58002       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  72.201  72.201      2.7428       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  81.165  81.165      2.4625       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.398   0.398     0.96452E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  27.900 135.287      9.8128       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   1.161   1.161     0.36684E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2pr3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1528.8892\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -72.17  -72.10   15.81    1.21  -63.00  161.45   22.43\n",
      "    1        235R 235R N   CA   1853 1854  157.71  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -73.29  -80.20   31.21    0.90   82.20 -153.98    6.99\n",
      "    2        236G 236G N   CA   1864 1865  143.66  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -117.37 -108.50   24.43    1.21  -63.50  172.18   27.23\n",
      "    3        237L 237L N   CA   1868 1869  155.26  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -141.20 -118.00   31.57    1.10  -62.90  176.96   27.66\n",
      "    4        239K 239K N   CA   1883 1884  160.51  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -66.36  -68.20    3.05    0.19  -62.50  176.27   29.08\n",
      "    5        240A 240A N   CA   1892 1893  142.87  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    2   19   42   70  107  114  152  229  236  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2pr3_MODLL.B99990001.pdb        39.57281\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2q1j\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26371    24668\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24668     378\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       98\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          44.1431\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.18547       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.531   1.531      4.1770       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  48.934  48.934      12.944       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.732   1.732      2.0491       1.000\n",
      " 5 Soft-sphere overlap restraints     :      98       0      0   0.000   0.000     0.35136E-03   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.084   0.084     0.25376       1.000\n",
      "10 Distance restraints 2 (N-O)        :      43       0      0   0.428   0.428      2.2889       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.579   3.579      1.3595       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  76.793  76.793     0.64538       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  85.566  85.566      2.0706       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  50.331  50.331      2.5640       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 101.027 101.027      2.2980       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.664   0.664     0.30577       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  29.693 122.102      12.969       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   1.083   1.083     0.31617E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2q1j_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1480.6234\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864  175.80 -167.20   26.38    1.02   82.20  173.36   12.51\n",
      "    1        236G 236G N   CA   1864 1865  154.42  174.60                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -71.09  -70.70    5.44    0.40  -63.50  171.94   24.32\n",
      "    2        237L 237L N   CA   1868 1869  147.03  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883 -134.31 -118.00   17.19    0.75  -62.90 -171.48   21.77\n",
      "    3        239K 239K N   CA   1883 1884  133.66  139.10                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892 -113.51 -134.00   20.51    0.65  -62.50 -179.63   32.27\n",
      "    4        240A 240A N   CA   1892 1893  146.10  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    4   15   36   60  107  131  151  194  255  312\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2q1j_MODLL.B99990001.pdb        44.14310\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2ra0\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26414    24711\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24711     390\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       80\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          67.4800\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.27323       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.853   1.853      6.2102       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  56.831  56.831      16.999       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.802   1.802      2.0498       1.000\n",
      " 5 Soft-sphere overlap restraints     :      80       0      0   0.003   0.003     0.96075E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      40       0      0   0.564   0.564      8.0457       1.000\n",
      "10 Distance restraints 2 (N-O)        :      37       0      0   0.565   0.565      5.3074       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.574   2.574     0.70300       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  62.440  62.440      1.2342       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  80.049  80.049      3.0785       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  88.144  88.144      2.7624       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 106.068 106.068      2.0744       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      35       0      0   0.662   0.662     0.33646       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  21.531 137.782      17.821       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      13       0      0   0.537   0.537     0.48873       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2ra0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1614.2070\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -79.24  -72.10   10.25    0.66  -63.00  170.42   24.13\n",
      "    1        235R 235R N   CA   1853 1854  149.26  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -80.82  -80.20    4.50    0.21   82.20 -130.78    7.22\n",
      "    2        236G 236G N   CA   1864 1865  169.64  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -66.48  -70.70    8.99    0.56  -63.50  174.89   24.11\n",
      "    3        237L 237L N   CA   1868 1869  133.66  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -76.89  -70.20    7.37    0.47  -62.90  176.26   23.68\n",
      "    4        239K 239K N   CA   1883 1884  143.49  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -83.55  -68.20   24.66    1.55  -62.50  155.92   26.64\n",
      "    5        240A 240A N   CA   1892 1893  164.61  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   454  495   2.224\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    1    6    6   23   48   81  100  133  183  190  254  284\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2ra0_MODLL.B99990001.pdb        67.48003\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2uwl\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26338    24635\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1837 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_484W> Dihedral still outside +-90:       90.2759\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24635     368\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      174\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          64.4654\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.35447       1.000\n",
      " 2 Bond angle potential               :      94       0      1   2.413   2.413      9.9241       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.490  46.490      15.076       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   3.193   3.193      3.4728       1.000\n",
      " 5 Soft-sphere overlap restraints     :     174       1      3   0.027   0.027      14.881       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.108   0.108     0.42704       1.000\n",
      "10 Distance restraints 2 (N-O)        :      44       0      0   0.278   0.278      1.4830       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.619   3.619      1.3905       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  81.522  81.522      1.0408       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  74.409  74.409      1.4060       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  75.237  75.237      3.1284       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  55.315  55.315      2.7236       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      15       0      0   0.175   0.175     0.11728E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       3      1  29.521 117.418      9.1460       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2uwl_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1473.7372\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -155.08 -125.20   30.72    1.06  -63.00 -165.65   32.48\n",
      "    1        235R 235R N   CA   1853 1854  147.75  140.60                  -41.10\n",
      "    2   6831 236G 237L C   N    1866 1868  -75.83  -70.70   24.05    1.64  -63.50  154.19   22.14\n",
      "    2        237L 237L N   CA   1868 1869  165.10  141.60                  -41.20\n",
      "    3   6834 239K 240A C   N    1890 1892  -69.76  -68.20   20.98    1.64  -62.50  153.05   25.44\n",
      "    3        240A 240A N   CA   1892 1893  166.22  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    6   10   45   71   84  113  170  183  259  305\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2uwl_MODLL.B99990001.pdb        64.46542\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2uwo\n",
      "{'num_gaps': 2, 'gap_lengths': [2, 7], 'gap_list': [[63, 64], [235, 241]], 'gap_window': [[61, 66], [233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26177    24474\n",
      "114 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.4516\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.6228\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.3886\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     114\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24474     833\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      277\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         366.3648\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     116       0      1   0.014   0.014      9.7174       1.000\n",
      " 2 Bond angle potential               :     155       1      3   3.262   3.262      29.728       1.000\n",
      " 3 Stereochemical cosine torsion poten:      56       0      6  61.690  61.690      34.129       1.000\n",
      " 4 Stereochemical improper torsion pot:      37       0      1   2.909   2.909      5.9790       1.000\n",
      " 5 Soft-sphere overlap restraints     :     277       9     13   0.082   0.082      218.84       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     142       0      0   0.151   0.151      1.7125       1.000\n",
      "10 Distance restraints 2 (N-O)        :     140       0      0   0.484   0.484      9.3326       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      16       1      0  44.868  44.868      2.8870       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  90.078  90.078      4.5740       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      10       0      0  61.665  61.665      4.6917       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      1  92.872  92.872      8.4232       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  60.632  60.632      2.4517       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      70       0      0   1.210   1.210      1.9536       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17       8      4  44.948 132.781      29.506       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      50       0      0   0.923   0.923      2.4419       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2uwo_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3177.0337\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4514 235R 235R CD  NE   1857 1858  141.49  120.00   21.48    5.45  120.00   21.48    5.45\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6657  62E  63E C   N     487  489   86.11  -69.30  156.67   11.15  -63.60 -138.72   37.28\n",
      "    1         63E  63E N   CA    489  490  122.65  142.50                  -40.30\n",
      "    2   6658  63E  64G C   N     496  498  -78.26  -80.20    3.90    0.09   82.20 -131.83    7.09\n",
      "    2         64G  64G N   CA    498  499  170.72  174.10                    8.50\n",
      "    3   6829 234T 235R C   N    1851 1853   60.72   57.30    5.88    0.64  -63.00  149.48   27.23\n",
      "    3        235R 235R N   CA   1853 1854   42.78   38.00                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  175.37 -167.20   17.49    0.30   82.20 -168.27   13.48\n",
      "    4        236G 236G N   CA   1864 1865  176.07  174.60                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868 -113.10 -108.50    6.85    0.31  -63.50 -174.46   23.42\n",
      "    5        237L 237L N   CA   1868 1869  137.58  132.50                  -41.20\n",
      "    6   6832 237L 238P C   N    1874 1876  -56.14  -58.70    3.65    0.25  -64.50  179.89   13.87\n",
      "    6        238P 238P N   CA   1876 1877  -33.10  -30.50                  147.20\n",
      "    7   6833 238P 239K C   N    1881 1883  -71.93  -70.20    1.79    0.14  -62.90  179.49   23.81\n",
      "    7        239K 239K N   CA   1883 1884  139.93  140.40                  -40.80\n",
      "    8   6834 239K 240A C   N    1890 1892  -70.14  -68.20    1.95    0.17  -62.50  174.16   28.92\n",
      "    8        240A 240A N   CA   1892 1893  145.11  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   562 1861   2.241\n",
      "  serious non-bonded atom clash:   566 1858   2.293\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    2    4   10   14   39   76   82  114  142  222  248  270\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2uwo_MODLL.B99990001.pdb       366.36478\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2uwp\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26334    24631\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.5759\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.0381\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24631     372\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      180\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          86.9476\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.32936       1.000\n",
      " 2 Bond angle potential               :      94       0      0   2.062   2.062      7.0536       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  53.189  53.189      15.713       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   3.276   3.276      3.8529       1.000\n",
      " 5 Soft-sphere overlap restraints     :     180       1      4   0.038   0.038      30.775       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.097   0.097     0.33776       1.000\n",
      "10 Distance restraints 2 (N-O)        :      43       0      0   0.400   0.400      2.7619       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.439   3.439      1.2554       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  83.157  83.157      1.1739       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0 110.604 110.604      2.7504       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  86.948  86.948      3.7006       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  55.721  55.721      1.4906       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      16       0      0   0.371   0.371     0.62657E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      3  44.228 135.201      15.653       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       6       0      0   0.268   0.268     0.37612E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2uwp_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1953.0542\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  143.96 -125.20   93.58    3.20  -63.00 -141.60   39.05\n",
      "    1        235R 235R N   CA   1853 1854  163.09  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   94.05   78.70   45.69    0.74   82.20  142.85    7.53\n",
      "    2        236G 236G N   CA   1864 1865  150.86 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -70.90  -70.70    7.81    0.60  -63.50  175.16   23.91\n",
      "    3        237L 237L N   CA   1868 1869  133.80  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -118.97 -118.00   19.63    0.94  -62.90  170.01   25.42\n",
      "    4        239K 239K N   CA   1883 1884  158.71  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -64.58  -68.20    3.73    0.28  -62.50  174.72   28.74\n",
      "    5        240A 240A N   CA   1892 1893  144.39  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    5   17   48   64   79  112  157  191  267  288\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2uwp_MODLL.B99990001.pdb        86.94758\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2vh0\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26247    24544\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1837 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.0177\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24544     368\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      102\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          36.7022\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.90594E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.644   1.644      5.0029       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.196  49.196      13.236       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.275   1.275      1.0259       1.000\n",
      " 5 Soft-sphere overlap restraints     :     102       0      0   0.000   0.000     0.82137E-03   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.112   0.112     0.45792       1.000\n",
      "10 Distance restraints 2 (N-O)        :      45       0      0   0.211   0.211     0.88284       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.140   3.140      1.0464       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  46.975  46.975     0.20661       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  77.226  77.226     0.80415       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  74.487  74.487      1.4741       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  64.511  64.511      2.3318       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      16       0      0   0.351   0.351     0.53767E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      3  28.070 125.370      10.088       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vh0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1471.9512\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -87.92  -72.10   37.16    2.52  -63.00  145.52   21.38\n",
      "    1        235R 235R N   CA   1853 1854  175.53  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -58.17  -62.40    4.81    0.74   82.20  149.68   11.44\n",
      "    2        236G 236G N   CA   1864 1865  -43.48  -41.20                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -106.41 -108.50   19.19    1.01  -63.50  160.47   20.26\n",
      "    3        237L 237L N   CA   1868 1869  113.43  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -118.45 -118.00    9.31    0.45  -62.90  179.60   26.63\n",
      "    4        239K 239K N   CA   1883 1884  148.41  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -116.78 -134.00   18.55    0.43  -62.50 -172.96   33.54\n",
      "    5        240A 240A N   CA   1892 1893  140.11  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    3   13   37   64   72  108  156  183  261  275\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vh0_MODLL.B99990001.pdb        36.70219\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2vh6\n",
      "{'num_gaps': 2, 'gap_lengths': [2, 7], 'gap_list': [[63, 64], [235, 241]], 'gap_window': [[61, 66], [233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26282    24579\n",
      "114 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1843 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.9934\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     114\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24579     825\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      195\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          88.3952\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     116       0      0   0.004   0.004     0.67840       1.000\n",
      " 2 Bond angle potential               :     155       0      0   2.214   2.214      14.170       1.000\n",
      " 3 Stereochemical cosine torsion poten:      56       0      3  53.526  53.526      24.991       1.000\n",
      " 4 Stereochemical improper torsion pot:      37       0      0   1.631   1.631      2.6722       1.000\n",
      " 5 Soft-sphere overlap restraints     :     195       0      0   0.001   0.001     0.14334E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     146       0      0   0.239   0.239      4.1603       1.000\n",
      "10 Distance restraints 2 (N-O)        :     142       0      0   0.251   0.251      4.0790       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      16       0      0   4.265   4.265      2.7133       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  54.838  54.838      2.5451       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      10       0      0  65.344  65.344      2.8875       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  78.373  78.373      7.3347       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  92.220  92.220      2.5808       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      71       0      0   1.037   1.037      1.3413       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17       8      2  28.937 119.519      17.726       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      35       0      0   0.510   0.510     0.50196       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vh6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2347.5115\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6657  62E  63E C   N     487  489  -76.32  -69.30    7.22    0.49  -63.60  175.95   24.16\n",
      "    1         63E  63E N   CA    489  490  144.21  142.50                  -40.30\n",
      "    2   6658  63E  64G C   N     496  498  -65.35  -80.20   29.08    0.69   82.20 -156.19    6.55\n",
      "    2         64G  64G N   CA    498  499  149.10  174.10                    8.50\n",
      "    3   6829 234T 235R C   N    1851 1853   57.70   57.30    5.85    0.36  -63.00  141.20   25.74\n",
      "    3        235R 235R N   CA   1853 1854   32.17   38.00                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864 -136.74 -167.20   32.60    0.47   82.20 -150.80   15.90\n",
      "    4        236G 236G N   CA   1864 1865  162.99  174.60                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -90.72 -108.50   19.61    0.87  -63.50  167.66   21.87\n",
      "    5        237L 237L N   CA   1868 1869  124.23  132.50                  -41.20\n",
      "    6   6832 237L 238P C   N    1874 1876  -71.59  -58.70   26.50    1.55  -64.50  154.71   11.92\n",
      "    6        238P 238P N   CA   1876 1877   -7.35  -30.50                  147.20\n",
      "    7   6833 238P 239K C   N    1881 1883 -100.56 -118.00   71.96    3.67  -62.90  116.53   17.38\n",
      "    7        239K 239K N   CA   1883 1884 -151.08  139.10                  -40.80\n",
      "    8   6834 239K 240A C   N    1890 1892  -64.62  -68.20    6.08    0.38  -62.50  178.73   29.39\n",
      "    8        240A 240A N   CA   1892 1893  140.39  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1508 1535   2.216\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    2    8   13   36   67   84  116  146  195  242  285\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vh6_MODLL.B99990001.pdb        88.39517\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2vvc\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26453    24750\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1843 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24750     506\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      259\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         184.3645\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.009   0.009      2.0280       1.000\n",
      " 2 Bond angle potential               :     102       1      5   6.299   6.299      69.219       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      1  55.094  55.094      18.427       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   3.455   3.455      5.9557       1.000\n",
      " 5 Soft-sphere overlap restraints     :     259       0      1   0.013   0.013      5.0227       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      68       0      0   0.347   0.347      7.9684       1.000\n",
      "10 Distance restraints 2 (N-O)        :      69       0      0   0.649   0.649      20.468       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      2  12.527  12.527      20.355       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  87.773  87.773      3.6818       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  59.038  59.038      1.3042       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 101.860 101.860      4.5046       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 114.028 114.028      2.6522       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.320   0.320     0.51675E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       5      3  33.308 136.778      20.471       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      58       0      0   0.572   0.572      2.2546       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vvc_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2215.7441\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837  -74.02  -70.20   14.90    1.16  -62.90  167.16   21.12\n",
      "    1        233K 233K N   CA   1837 1838  125.99  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -127.77 -124.80   16.12    0.61  -63.20  171.20   26.21\n",
      "    2        234T 234T N   CA   1846 1847  159.34  143.50                  -42.10\n",
      "    3   6830 235R 236G C   N    1862 1864  -86.21  -80.20    7.71    0.29   82.20 -120.40    7.45\n",
      "    3        236G 236G N   CA   1864 1865  178.93  174.10                    8.50\n",
      "    4   6833 238P 239K C   N    1881 1883  -96.26 -118.00   22.38    0.94  -62.90  177.92   25.05\n",
      "    4        239K 239K N   CA   1883 1884  144.43  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  141.45 -134.00   85.28    3.26  -62.50 -124.24   36.01\n",
      "    5        240A 240A N   CA   1892 1893  135.83  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    9   12   25   57   70   96  141  142  202  290  285\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vvc_MODLL.B99990001.pdb       184.36447\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2vvu\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26273    24570\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1843 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.2666\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24570     505\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      255\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         276.1753\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.017   0.017      5.5698       1.000\n",
      " 2 Bond angle potential               :     102       4      6   5.355   5.355      53.511       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      2  57.346  57.346      16.898       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       1      2   5.014   5.014      16.658       1.000\n",
      " 5 Soft-sphere overlap restraints     :     255       0      4   0.023   0.023      15.860       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      68       0      7   0.719   0.719      33.795       1.000\n",
      "10 Distance restraints 2 (N-O)        :      68       1      3   0.813   0.813      28.493       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  22.439  22.439      65.314       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  85.162  85.162      3.6430       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  79.889  79.889      2.5684       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  90.218  90.218      3.9762       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 115.917 115.917      1.8905       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.571   0.571     0.18020       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      5  45.059 115.966      25.362       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      58       0      0   0.600   0.600      2.4569       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vvu_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2284.1064\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  144.55  120.00   24.55    5.58  120.00   24.55    5.58\n",
      "    2   4477 232M 232M N   CA   1829 1830  127.64  107.00   20.64    5.93  107.00   20.64    5.93\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7074 240A 240A CA  C    1893 1895 -106.40 -180.00   73.60   14.71 -180.00   73.60   14.71\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   31.13  -70.20  117.72    9.71  -62.90  151.57   17.82\n",
      "    1        233K 233K N   CA   1837 1838 -159.68  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -86.49  -78.10   11.83    0.43  -63.20  161.45   22.26\n",
      "    2        234T 234T N   CA   1846 1847  158.14  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853  -84.76  -72.10   28.18    1.90  -63.00  153.38   22.22\n",
      "    3        235R 235R N   CA   1853 1854  167.07  141.90                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864   98.85   78.70   26.76    1.46   82.20  157.87    6.82\n",
      "    4        236G 236G N   CA   1864 1865 -148.49 -166.10                    8.50\n",
      "    5   6833 238P 239K C   N    1881 1883 -103.09 -118.00   61.64    3.15  -62.90  126.83   18.87\n",
      "    5        239K 239K N   CA   1883 1884 -161.09  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892 -133.64 -134.00   24.04    1.36  -62.50  178.64   26.86\n",
      "    6        240A 240A N   CA   1892 1893  122.96  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    9   27   54   85   97  132  164  219  252  284\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vvu_MODLL.B99990001.pdb       276.17532\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2vvv\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26281    24578\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.4135\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24578     507\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      259\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         219.4795\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      2   0.021   0.021      9.8041       1.000\n",
      " 2 Bond angle potential               :     102       6      8   6.377   6.377      79.224       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      2  54.918  54.918      20.032       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   2.517   2.517      4.4062       1.000\n",
      " 5 Soft-sphere overlap restraints     :     259       2      7   0.033   0.033      33.113       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      68       0      0   0.512   0.512      16.564       1.000\n",
      "10 Distance restraints 2 (N-O)        :      68       0      0   0.509   0.509      9.0760       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       0      2   8.293   8.293      8.9226       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  76.161  76.161      3.1813       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  49.542  49.542      1.7507       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  82.604  82.604      3.0754       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  74.264  74.264      1.3931       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.581   0.581     0.16525       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      5  49.087 137.546      26.423       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      61       0      0   0.575   0.575      2.3477       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vvv_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2499.5278\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  140.68  120.00   20.68    4.70  120.00   20.68    4.70\n",
      "    2   4477 232M 232M N   CA   1829 1830  125.51  107.00   18.51    5.32  107.00   18.51    5.32\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   60.55   56.60  128.74    9.50  -62.90 -164.24   23.08\n",
      "    1        233K 233K N   CA   1837 1838  167.28   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -136.89 -124.80   13.34    0.80  -63.20 -165.55   21.05\n",
      "    2        234T 234T N   CA   1846 1847  137.85  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -151.29 -125.20   58.57    2.16  -63.00  153.74   26.58\n",
      "    3        235R 235R N   CA   1853 1854 -166.96  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -58.91  -62.40    5.26    0.73   82.20  150.96   11.56\n",
      "    4        236G 236G N   CA   1864 1865  -45.14  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868 -101.11 -108.50   11.61    0.66  -63.50 -178.71   27.46\n",
      "    5        237L 237L N   CA   1868 1869  141.45  132.50                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883 -144.80 -118.00   52.28    2.06  -62.90  158.08   25.40\n",
      "    6        239K 239K N   CA   1883 1884 -176.01  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892 -120.56 -134.00   28.22    1.74  -62.50  158.32   29.05\n",
      "    7        240A 240A N   CA   1892 1893  171.82  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    9   15   50   78   87  112  157  219  258  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vvv_MODLL.B99990001.pdb       219.47951\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2vwl\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26488    24785\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1834 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.7291\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24785     381\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       83\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          35.5740\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.63446E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.532   1.532      4.3096       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.823  46.823      11.111       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.711   1.711      1.8694       1.000\n",
      " 5 Soft-sphere overlap restraints     :      83       0      0   0.000   0.000     0.36439E-05   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.136   0.136     0.66997       1.000\n",
      "10 Distance restraints 2 (N-O)        :      39       0      0   0.259   0.259      1.4399       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.528   3.528      1.3213       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  48.612  48.612      1.1148       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  76.564  76.564      1.6400       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  95.317  95.317      3.8629       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  99.288  99.288      3.4158       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.723   0.723     0.26917       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      0  20.099 121.302      3.5575       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      10       0      0   1.024   1.024     0.92920       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vwl_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1331.1041\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -120.72 -125.20    6.71    0.21  -63.00 -174.10   21.86\n",
      "    1        235R 235R N   CA   1853 1854  135.61  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -66.37  -80.20   47.92    1.26   82.20 -169.20    6.78\n",
      "    2        236G 236G N   CA   1864 1865  128.22  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -76.43  -70.70   10.70    0.66  -63.50  168.66   24.18\n",
      "    3        237L 237L N   CA   1868 1869  150.63  141.60                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892  -74.12  -68.20    8.44    0.53  -62.50  168.19   28.15\n",
      "    4        240A 240A N   CA   1892 1893  151.31  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2   10   18   42   72   81  120  147  210  252  302\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vwl_MODLL.B99990001.pdb        35.57400\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2vwm\n",
      "{'num_gaps': 2, 'gap_lengths': [1, 8], 'gap_list': [[63, 63], [233, 240]], 'gap_window': [[61, 65], [231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26524    24821\n",
      "109 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     109\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24821     868\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      350\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         488.6549\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     112       1      2   0.042   0.042      45.606       1.000\n",
      " 2 Bond angle potential               :     151       6      7   6.591   6.591      112.62       1.000\n",
      " 3 Stereochemical cosine torsion poten:      53       0      4  58.986  58.986      25.863       1.000\n",
      " 4 Stereochemical improper torsion pot:      36       0      1   2.405   2.405      6.2596       1.000\n",
      " 5 Soft-sphere overlap restraints     :     350      10     13   0.065   0.065      175.10       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     145       0      0   0.393   0.393      15.137       1.000\n",
      "10 Distance restraints 2 (N-O)        :     152       0      2   0.744   0.744      32.653       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       1      3  43.198  43.198      14.989       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  81.943  81.943      4.2999       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       9       0      0  39.429  39.429     0.83043       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0 103.032 103.032      7.1855       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 105.938 105.938      2.4300       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      56       0      0   0.545   0.545     0.35414       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      18       8      7  37.046 110.122      42.061       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      97       0      0   0.689   0.689      3.2698       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vwm_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3147.9475\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1877 233K 233K CG  CB   1840 1839    1.93    1.53    0.40   10.86    1.53    0.40   10.86\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  145.70  120.00   25.70    5.84  120.00   25.70    5.84\n",
      "    2   4477 232M 232M N   CA   1829 1830  125.38  107.00   18.38    5.28  107.00   18.38    5.28\n",
      "    3   4488 233K 233K CA  CB   1838 1839  138.16  113.50   24.66    6.05  113.50   24.66    6.05\n",
      "    4   4491 233K 233K CB  CG   1839 1840  140.74  113.60   27.14    6.66  113.60   27.14    6.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6896  62E  62E CA  C     481  487   -4.42 -179.18  174.76   40.61 -179.18  174.76   40.61\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6657  62E  63E C   N     487  489  175.64 -117.80   98.79    5.31  -63.60  159.43   19.43\n",
      "    1         63E  63E N   CA    489  490   63.79  136.80                  -40.30\n",
      "    2   6659  64G  65G C   N     500  502  -73.74  -62.40   15.94    2.25   82.20  160.63   11.93\n",
      "    2         65G  65G N   CA    502  503  -30.00  -41.20                    8.50\n",
      "    3   6827 232M 233K C   N    1835 1837   49.69   56.60   31.14    1.76  -62.90  157.24   26.81\n",
      "    3        233K 233K N   CA   1837 1838   68.96   38.60                  -40.80\n",
      "    4   6830 235R 236G C   N    1862 1864  -79.32  -80.20   35.74    1.45   82.20 -133.58   17.39\n",
      "    4        236G 236G N   CA   1864 1865 -150.18  174.10                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -57.01  -70.70   18.56    1.86  -63.50  164.78   22.52\n",
      "    5        237L 237L N   CA   1868 1869  154.14  141.60                  -41.20\n",
      "    6   6832 237L 238P C   N    1874 1876  -58.26  -58.70   10.62    0.91  -64.50  167.20   12.15\n",
      "    6        238P 238P N   CA   1876 1877  -19.89  -30.50                  147.20\n",
      "    7   6833 238P 239K C   N    1881 1883  -75.53  -70.20   52.30    3.66  -62.90  127.41   17.26\n",
      "    7        239K 239K N   CA   1883 1884 -167.58  140.40                  -40.80\n",
      "    8   6834 239K 240A C   N    1890 1892  -92.49  -68.20   62.54    4.22  -62.50  119.97   21.25\n",
      "    8        240A 240A N   CA   1892 1893 -157.06  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    3    6   17   31   61   89  104  125  166  231  277  307\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vwm_MODLL.B99990001.pdb       488.65488\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2vwn\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26305    24602\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.9398\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24602     385\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       96\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          55.4012\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.24641       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.996   1.996      7.1252       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      4  56.647  56.647      17.494       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.352   2.352      3.7744       1.000\n",
      " 5 Soft-sphere overlap restraints     :      96       0      0   0.001   0.001     0.23093E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.270   0.270      2.7406       1.000\n",
      "10 Distance restraints 2 (N-O)        :      39       0      0   0.285   0.285      2.1102       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   5.262   5.262      2.9392       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  75.750  75.750      1.1931       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  26.701  26.701    -0.43405       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  78.804  78.804      4.3306       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  72.563  72.563      2.9321       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.593   0.593     0.28636       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  24.982 126.968      8.5858       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      12       0      0   1.409   1.409      2.0544       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vwn_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1576.6017\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -129.25 -125.20   29.02    1.31  -63.00  163.58   26.63\n",
      "    1        235R 235R N   CA   1853 1854  169.33  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   95.06   78.70   31.04    0.48   82.20  159.54    8.39\n",
      "    2        236G 236G N   CA   1864 1865  167.53 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -130.18 -108.50   24.61    1.32  -63.50  175.23   21.61\n",
      "    3        237L 237L N   CA   1868 1869  120.85  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -98.89 -118.00   28.89    1.05  -62.90  162.28   19.40\n",
      "    4        239K 239K N   CA   1883 1884  117.44  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -82.04  -68.20   13.84    1.17  -62.50  174.83   29.65\n",
      "    5        240A 240A N   CA   1892 1893  145.37  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   342  590   2.296\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    1    7   10   45   69   88  107  128  202  253  282\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vwn_MODLL.B99990001.pdb        55.40117\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2vwo\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26363    24660\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1818 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24660     392\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       84\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          37.8187\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.13551       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.615   1.615      4.9538       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  45.506  45.506      11.224       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.087   2.087      2.7456       1.000\n",
      " 5 Soft-sphere overlap restraints     :      84       0      0   0.001   0.001     0.39731E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.107   0.107     0.30400       1.000\n",
      "10 Distance restraints 2 (N-O)        :      43       0      0   0.247   0.247      1.0519       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.943   3.943      1.6505       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  65.028  65.028     0.25229       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  68.116  68.116     0.68084       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 116.668 116.668      4.3387       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  46.197  46.197      1.5958       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.158   0.158     0.14870E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  17.154 140.543      8.5845       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      14       0      0   0.488   0.488     0.28226       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2vwo_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1534.3719\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -131.59 -125.20   22.69    0.94  -63.00  170.89   27.78\n",
      "    1        235R 235R N   CA   1853 1854  162.37  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -82.43  -80.20    9.59    0.27   82.20 -119.79    7.28\n",
      "    2        236G 236G N   CA   1864 1865 -176.57  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -66.71  -70.70    4.33    0.43  -63.50  175.54   24.56\n",
      "    3        237L 237L N   CA   1868 1869  143.29  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -77.81  -70.20   15.85    0.98  -62.90  165.57   22.35\n",
      "    4        239K 239K N   CA   1883 1884  154.30  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -74.23  -68.20   12.01    0.78  -62.50  163.84   27.44\n",
      "    5        240A 240A N   CA   1892 1893  155.68  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    2   14   44   69   96   96  140  221  257  278\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2vwo_MODLL.B99990001.pdb        37.81868\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2w26\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26317    24614\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24614     494\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      279\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         185.9612\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.011   0.011      3.3965       1.000\n",
      " 2 Bond angle potential               :     102       3      4   5.742   5.742      59.610       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      4  65.114  65.114      23.815       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.626   3.626      7.7953       1.000\n",
      " 5 Soft-sphere overlap restraints     :     279       0      1   0.012   0.012      5.0138       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      65       0      0   0.310   0.310      5.6566       1.000\n",
      "10 Distance restraints 2 (N-O)        :      65       0      0   0.663   0.663      13.749       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       0      2   8.023   8.023      8.3497       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  94.190  94.190      2.8447       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  78.037  78.037     0.53637       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  71.900  71.900      3.2619       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  72.629  72.629      2.5094       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.363   0.363     0.70328E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       8      5  49.453 140.150      46.234       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      55       0      0   0.691   0.691      3.1183       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2w26_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2420.5215\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4563 240A 240A CA  C    1893 1895  133.40  116.50   16.90    4.86  116.50   16.90    4.86\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6826 231S 232M C   N    1827 1829  -99.49  -63.40   42.96    8.74  -63.40   42.96    8.74\n",
      "    1        232M 232M N   CA   1829 1830  -63.81  -40.50                  -40.50\n",
      "    2   6827 232M 233K C   N    1835 1837   80.75   56.60  111.74    9.86  -62.90 -136.29   26.48\n",
      "    2        233K 233K N   CA   1837 1838  147.70   38.60                  -40.80\n",
      "    3   6828 233K 234T C   N    1844 1846 -134.08 -124.80   13.12    0.79  -63.20 -169.97   20.60\n",
      "    3        234T 234T N   CA   1846 1847  134.22  143.50                  -42.10\n",
      "    4   6829 234T 235R C   N    1851 1853 -161.90 -125.20   54.25    2.99  -63.00  172.84   19.62\n",
      "    4        235R 235R N   CA   1853 1854  100.65  140.60                  -41.10\n",
      "    5   6830 235R 236G C   N    1862 1864   59.88   78.70   66.01    1.16   82.20  113.55    6.61\n",
      "    5        236G 236G N   CA   1864 1865 -102.83 -166.10                    8.50\n",
      "    6   6831 236G 237L C   N    1866 1868  -63.65  -70.70   20.63    1.83  -63.50  157.81   21.92\n",
      "    6        237L 237L N   CA   1868 1869  160.99  141.60                  -41.20\n",
      "    7   6833 238P 239K C   N    1881 1883 -110.42 -118.00   30.36    1.55  -62.90  158.02   23.35\n",
      "    7        239K 239K N   CA   1883 1884  168.50  139.10                  -40.80\n",
      "    8   6834 239K 240A C   N    1890 1892 -164.38 -134.00   46.22    2.75  -62.50 -176.14   27.62\n",
      "    8        240A 240A N   CA   1892 1893  112.16  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2   10   25   66   75   99  136  154  227  270  325\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2w26_MODLL.B99990001.pdb       185.96124\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2w3i\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26314    24611\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.6802\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24611     371\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      109\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          39.5400\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.94318E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.594   1.594      4.9042       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  50.973  50.973      13.826       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.107   2.107      3.0254       1.000\n",
      " 5 Soft-sphere overlap restraints     :     109       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      41       0      0   0.241   0.241      1.7490       1.000\n",
      "10 Distance restraints 2 (N-O)        :      37       0      0   0.349   0.349      1.6337       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.128   3.128      1.0388       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  91.855  91.855      2.1941       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  71.751  71.751     0.40893       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  66.215  66.215      2.4548       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  52.027  52.027      1.4880       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.341   0.341     0.77751E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  15.617 127.991      6.6038       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       4       0      0   0.613   0.613     0.41458E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2w3i_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1578.4095\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -88.83  -72.10   28.80    1.89  -63.00  155.71   22.81\n",
      "    1        235R 235R N   CA   1853 1854  165.34  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -61.75  -62.40    1.69    0.23   82.20  152.81   11.65\n",
      "    2        236G 236G N   CA   1864 1865  -42.76  -41.20                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -74.45  -70.70    3.75    0.34  -63.50  177.54   25.29\n",
      "    3        237L 237L N   CA   1868 1869  141.60  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -81.79  -70.20   11.60    0.83  -62.90  179.28   24.36\n",
      "    4        239K 239K N   CA   1883 1884  140.91  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -129.53 -134.00    5.13    0.12  -62.50 -172.95   34.24\n",
      "    5        240A 240A N   CA   1892 1893  144.48  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    6   10   30   56   78  110  135  223  259  287\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2w3i_MODLL.B99990001.pdb        39.53997\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2w3k\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26343    24640\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1806 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24640     377\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       88\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          31.5425\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.18585       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.704   1.704      5.3753       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  46.400  46.400      11.789       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.148   1.148     0.58599       1.000\n",
      " 5 Soft-sphere overlap restraints     :      88       0      0   0.004   0.004     0.19186       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.116   0.116     0.45162       1.000\n",
      "10 Distance restraints 2 (N-O)        :      44       0      0   0.245   0.245      1.1098       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.929   2.929     0.91061       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  77.252  77.252     0.90762       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  70.056  70.056     0.93789       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  57.060  57.060      2.0878       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  68.716  68.716      1.8008       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.659   0.659     0.22869       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       3      1  24.373 104.469      4.9799       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2w3k_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1296.7250\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868 -108.62 -108.50   14.95    0.81  -63.50  177.19   27.37\n",
      "    1        237L 237L N   CA   1868 1869  147.45  132.50                  -41.20\n",
      "    2   6833 238P 239K C   N    1881 1883  -71.18  -70.20    4.87    0.37  -62.90  176.62   22.50\n",
      "    2        239K 239K N   CA   1883 1884  135.63  140.40                  -40.80\n",
      "    3   6834 239K 240A C   N    1890 1892  -83.57  -68.20   15.37    1.31  -62.50  175.42   29.83\n",
      "    3        240A 240A N   CA   1892 1893  144.96  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    3   15   36   45   82  102  145  234  234  297\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2w3k_MODLL.B99990001.pdb        31.54253\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2wyg\n",
      "{'num_gaps': 2, 'gap_lengths': [6, 7], 'gap_list': [[60, 65], [235, 241]], 'gap_window': [[58, 67], [233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     66     1   74    81      E     A   13.140\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25858    24155\n",
      "143 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1828 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.3131\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.4811\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.3999\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     143\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24155    1014\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      351\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         568.7205\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     145       0      3   0.020   0.020      16.303       1.000\n",
      " 2 Bond angle potential               :     195       1      5   3.083   3.083      36.567       1.000\n",
      " 3 Stereochemical cosine torsion poten:      71       0      4  54.196  54.196      33.019       1.000\n",
      " 4 Stereochemical improper torsion pot:      49       0      2   3.066   3.066      10.184       1.000\n",
      " 5 Soft-sphere overlap restraints     :     351       0      2   0.014   0.014      8.4626       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     167       8     12   1.198   1.198      175.05       1.000\n",
      "10 Distance restraints 2 (N-O)        :     157       6     16   1.283   1.283      167.45       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       0      1   5.692   5.692      7.6406       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      14       0      0  66.288  66.288      2.7429       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      12       0      1  84.339  84.339      6.8062       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0 105.560 105.560      6.8109       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  82.827  82.827      2.4471       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      73       0      0   0.472   0.472     0.61886       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      21       7      4  39.555 107.751      31.025       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      76       1     14   2.846   2.846      63.597       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2wyg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3522.4475\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9846  66E 139Q CA  CA    507 1095   15.78   12.09    3.69    6.90   12.09    3.69    6.90\n",
      "    2   9847  66E 140S CA  CA    507 1104   14.77   10.38    4.39    9.12   10.38    4.39    9.12\n",
      "    3   9848  66E 141T CA  CA    507 1110   11.77    6.63    5.13   11.78    6.63    5.13   11.78\n",
      "    4   9849  66E 142R CA  CA    507 1117   12.61    8.47    4.14    8.58    8.47    4.14    8.58\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14315  66E 139Q N   O     506 1102   14.05    9.62    4.43    7.70    9.62    4.43    7.70\n",
      "    2  14316  66E 140S N   O     506 1108   11.35    7.54    3.82    7.31    7.54    3.82    7.31\n",
      "    3  14317  66E 141T N   O     506 1115    9.63    4.70    4.92   10.01    4.70    4.92   10.01\n",
      "    4  14318  66E 142R N   O     506 1126   13.07    9.17    3.89    6.69    9.17    3.89    6.69\n",
      "    5  15865 141T  66E N   O    1109  514   15.30   10.34    4.96    8.53   10.34    4.96    8.53\n",
      "    6  15887 142R  66E N   O    1116  514   14.54    9.93    4.61    7.73    9.93    4.61    7.73\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6654  59T  60E C   N     460  462   58.75  -69.30  131.24   10.89  -63.60 -167.64   22.48\n",
      "    1         60E  60E N   CA    462  463  171.27  142.50                  -40.30\n",
      "    2   6656  61Q  62E C   N     478  480  -51.50  -69.30   25.78    1.44  -63.60  164.59   22.62\n",
      "    2         62E  62E N   CA    480  481  123.85  142.50                  -40.30\n",
      "    3   6657  62E  63E C   N     487  489  -91.43  -69.30   35.78    2.02  -63.60  151.67   21.79\n",
      "    3         63E  63E N   CA    489  490  170.61  142.50                  -40.30\n",
      "    4   6829 234T 235R C   N    1851 1853 -158.01 -125.20   38.23    1.14  -63.00 -175.06   31.35\n",
      "    4        235R 235R N   CA   1853 1854  160.23  140.60                  -41.10\n",
      "    5   6830 235R 236G C   N    1862 1864 -169.32 -167.20    5.23    0.21   82.20 -165.60   14.15\n",
      "    5        236G 236G N   CA   1864 1865  169.82  174.60                    8.50\n",
      "    6   6831 236G 237L C   N    1866 1868  -82.12  -70.70   16.28    1.04  -63.50  166.64   24.25\n",
      "    6        237L 237L N   CA   1868 1869  153.20  141.60                  -41.20\n",
      "    7   6834 239K 240A C   N    1890 1892  -68.62  -68.20    4.00    0.34  -62.50  177.89   29.46\n",
      "    7        240A 240A N   CA   1892 1893  141.32  145.30                  -40.90\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  22714  66E 138R OE2 NH2   512 1091   10.44    3.46    6.97    4.50    3.46    6.97    4.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    7   20   51   85   82  116  143  200  263  272\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2wyg_MODLL.B99990001.pdb       568.72046\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2wyj\n",
      "{'num_gaps': 2, 'gap_lengths': [3, 8], 'gap_list': [[63, 65], [233, 240]], 'gap_window': [[61, 67], [231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25935    24232\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "123 (of 1906 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     123\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24232    1011\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      341\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         230.8474\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     126       0      0   0.009   0.009      3.1140       1.000\n",
      " 2 Bond angle potential               :     170       1      4   4.327   4.327      54.390       1.000\n",
      " 3 Stereochemical cosine torsion poten:      59       0      8  68.199  68.199      41.391       1.000\n",
      " 4 Stereochemical improper torsion pot:      41       0      1   2.529   2.529      5.6643       1.000\n",
      " 5 Soft-sphere overlap restraints     :     341       0      0   0.013   0.013      6.4835       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     177       0      0   0.345   0.345      16.475       1.000\n",
      "10 Distance restraints 2 (N-O)        :     178       0      0   0.383   0.383      15.887       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      19       1      2   8.054   8.054      13.828       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  64.278  64.278      1.5660       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      10       0      0  71.063  71.063      3.6698       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  82.109  82.109      6.4766       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  11.151  11.151      1.3990       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      74       0      0   0.402   0.402     0.25798       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      20      13      7  43.731 127.161      54.041       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     113       0      0   0.807   0.807      6.2029       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2wyj_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3402.4363\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7074 240A 240A CA  C    1893 1895 -151.51 -180.00   28.49    5.70 -180.00   28.49    5.70\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6654  59T  60E C   N     460  462 -108.80  -63.60   46.25    6.84  -63.60   46.25    6.84\n",
      "    1         60E  60E N   CA    462  463  -30.50  -40.30                  -40.30\n",
      "    2   6658  63E  64G C   N     496  498  -62.44  -62.40    6.42    0.96   82.20  155.15   11.89\n",
      "    2         64G  64G N   CA    498  499  -47.63  -41.20                    8.50\n",
      "    3   6659  64G  65G C   N     500  502  140.55 -167.20   53.72    1.35   82.20  164.33   10.72\n",
      "    3         65G  65G N   CA    502  503  162.12  174.60                    8.50\n",
      "    4   6660  65G  66E C   N     504  506  177.41 -117.80   67.10    2.09  -63.60 -156.23   33.59\n",
      "    4         66E  66E N   CA    506  507  154.28  136.80                  -40.30\n",
      "    5   6826 231S 232M C   N    1827 1829  -73.84  -63.40   51.68    6.97  -63.40   51.68    6.97\n",
      "    5        232M 232M N   CA   1829 1830   10.11  -40.50                  -40.50\n",
      "    6   6827 232M 233K C   N    1835 1837  -35.35  -70.20   49.33    2.95  -62.90  148.85   20.94\n",
      "    6        233K 233K N   CA   1837 1838  105.48  140.40                  -40.80\n",
      "    7   6828 233K 234T C   N    1844 1846 -156.96 -124.80   49.18    1.55  -63.20  166.17   27.20\n",
      "    7        234T 234T N   CA   1846 1847 -179.29  143.50                  -42.10\n",
      "    8   6829 234T 235R C   N    1851 1853  -73.12  -72.10    6.40    0.47  -63.00  170.99   23.78\n",
      "    8        235R 235R N   CA   1853 1854  148.21  141.90                  -41.10\n",
      "    9   6830 235R 236G C   N    1862 1864   76.38   78.70   17.56    0.60   82.20  168.09    7.87\n",
      "    9        236G 236G N   CA   1864 1865  176.49 -166.10                    8.50\n",
      "   10   6831 236G 237L C   N    1866 1868 -118.87 -108.50   35.33    1.99  -63.50  150.49   18.59\n",
      "   10        237L 237L N   CA   1868 1869   98.73  132.50                  -41.20\n",
      "   11   6832 237L 238P C   N    1874 1876  -75.26  -58.70   32.92    1.93  -64.50  149.64   11.74\n",
      "   11        238P 238P N   CA   1876 1877   -2.05  -30.50                  147.20\n",
      "   12   6833 238P 239K C   N    1881 1883   75.11   56.60   30.33    1.70  -62.90  148.70   25.91\n",
      "   12        239K 239K N   CA   1883 1884   14.57   38.60                  -40.80\n",
      "   13   6834 239K 240A C   N    1890 1892 -137.28 -134.00   18.55    1.11  -62.50 -174.61   27.86\n",
      "   13        240A 240A N   CA   1892 1893  128.74  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    9   18   50   78  106  105  160  192  229  293\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2wyj_MODLL.B99990001.pdb       230.84744\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2xbv\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26282    24579\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.5232\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24579     378\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       82\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          39.7769\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.20792       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.734   1.734      5.5759       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.575  49.575      13.053       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.955   1.955      2.4028       1.000\n",
      " 5 Soft-sphere overlap restraints     :      82       0      0   0.003   0.003     0.89240E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      41       0      0   0.167   0.167     0.96655       1.000\n",
      "10 Distance restraints 2 (N-O)        :      37       0      0   0.139   0.139     0.41992       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.715   2.715     0.78245       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  73.409  73.409      2.2506       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  40.872  40.872     0.46766       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  81.531  81.531      2.3602       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 107.400 107.400      2.2428       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.658   0.658     0.20436       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  26.806 132.188      7.9381       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      11       0      0   1.157   1.157     0.81554       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2xbv_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1539.0531\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -129.51 -125.20    6.64    0.21  -63.00 -174.43   29.65\n",
      "    1        235R 235R N   CA   1853 1854  145.65  140.60                  -41.10\n",
      "    2   6831 236G 237L C   N    1866 1868 -104.39 -108.50   24.75    1.38  -63.50  166.97   25.69\n",
      "    2        237L 237L N   CA   1868 1869  156.91  132.50                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -95.90 -118.00   22.33    0.90  -62.90  179.93   25.29\n",
      "    3        239K 239K N   CA   1883 1884  142.32  139.10                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -77.25  -68.20   15.06    0.95  -62.50  162.44   27.37\n",
      "    4        240A 240A N   CA   1892 1893  157.34  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    5   15   40   56   75   85  147  177  265  306\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2xbv_MODLL.B99990001.pdb        39.77690\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2xbw\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26368    24665\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.8781\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24665     379\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      106\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          52.1176\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.33179       1.000\n",
      " 2 Bond angle potential               :      94       0      0   2.278   2.278      8.9705       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  51.972  51.972      15.154       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.648   1.648      1.6936       1.000\n",
      " 5 Soft-sphere overlap restraints     :     106       0      0   0.003   0.003     0.14342       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.120   0.120     0.52177       1.000\n",
      "10 Distance restraints 2 (N-O)        :      33       0      0   0.607   0.607      2.0153       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.514   3.514      1.3106       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  42.817  42.817     0.42887       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  61.659  61.659     0.37048       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  66.739  66.739      2.8022       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 116.807 116.807      2.9171       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.473   0.473     0.11651       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  30.160 129.447      13.165       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       9       0      0   1.537   1.537      2.1760       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2xbw_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1582.6667\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -149.85 -125.20   46.83    1.62  -63.00  163.46   27.87\n",
      "    1        235R 235R N   CA   1853 1854 -179.58  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  176.74 -167.20   25.60    1.00   82.20  174.07   12.58\n",
      "    2        236G 236G N   CA   1864 1865  154.66  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -110.83 -108.50    9.73    0.55  -63.50  170.94   21.53\n",
      "    3        237L 237L N   CA   1868 1869  123.06  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -99.36 -118.00   19.12    0.80  -62.90  179.58   25.46\n",
      "    4        239K 239K N   CA   1883 1884  143.36  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -60.60  -68.20   10.89    1.07  -62.50  166.01   27.11\n",
      "    5        240A 240A N   CA   1892 1893  153.10  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3    7   10   45   74   87  113  149  219  232  273\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2xbw_MODLL.B99990001.pdb        52.11756\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2xbx\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26412    24709\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.0917\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24709     385\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       82\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          56.5105\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.31974       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.965   1.965      7.1612       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  46.717  46.717      11.820       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.823   1.823      1.5833       1.000\n",
      " 5 Soft-sphere overlap restraints     :      82       0      0   0.003   0.003     0.10089       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.117   0.117     0.42114       1.000\n",
      "10 Distance restraints 2 (N-O)        :      37       0      0   0.275   0.275      1.1802       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.099   4.099      1.7830       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  75.221  75.221      1.0904       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  87.486  87.486      3.0777       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  86.334  86.334      3.8626       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  81.419  81.419      2.1771       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      30       0      0   0.535   0.535     0.16207       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       6      2  53.338 137.836      21.661       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       6       0      0   0.389   0.389     0.10968       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2xbx_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1695.9598\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6828 233K 234T C   N    1844 1846   60.97  -78.10  139.35    8.64 -124.80  174.88    8.87\n",
      "    1        234T 234T N   CA   1846 1847  158.65  149.80                  143.50\n",
      "    2   6829 234T 235R C   N    1851 1853  -83.38  -72.10   18.23    1.19  -63.00  163.96   23.55\n",
      "    2        235R 235R N   CA   1853 1854  156.21  141.90                  -41.10\n",
      "    3   6830 235R 236G C   N    1862 1864  129.76   78.70   52.02    2.35   82.20 -178.24   11.12\n",
      "    3        236G 236G N   CA   1864 1865 -176.07 -166.10                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868  -79.00  -70.70   10.14    0.69  -63.50  172.07   24.81\n",
      "    4        237L 237L N   CA   1868 1869  147.43  141.60                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883  -94.00  -70.20   45.83    2.81  -62.90  143.05   20.40\n",
      "    5        239K 239K N   CA   1883 1884  179.57  140.40                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -66.75  -68.20    2.61    0.25  -62.50  171.68   28.35\n",
      "    6        240A 240A N   CA   1892 1893  147.47  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   10   11   43   76   84  105  130  236  237  305\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2xbx_MODLL.B99990001.pdb        56.51052\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2xby\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25769    24066\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.1032\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24066     388\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          50.3402\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.14583       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.658   1.658      5.0312       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  54.127  54.127      16.383       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.322   1.322     0.95700       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.001   0.001     0.19509E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.134   0.134     0.71192       1.000\n",
      "10 Distance restraints 2 (N-O)        :      33       0      0   0.684   0.684      2.2781       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.676   2.676     0.75990       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  90.161  90.161      3.5996       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  82.720  82.720      1.3369       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  79.960  79.960      4.2681       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  86.238  86.238      2.8352       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      29       0      0   0.579   0.579     0.20345       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  30.541 140.660      9.8639       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      16       0      0   1.021   1.021      1.9470       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2xby_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1595.0464\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -111.06 -125.20   21.40    1.18  -63.00  169.21   26.18\n",
      "    1        235R 235R N   CA   1853 1854  156.66  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -82.77  -80.20    9.88    0.52   82.20 -132.91    7.33\n",
      "    2        236G 236G N   CA   1864 1865  164.56  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -127.35 -108.50   21.05    1.12  -63.50  176.29   21.80\n",
      "    3        237L 237L N   CA   1868 1869  123.12  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -131.15 -118.00   16.61    0.81  -62.90 -177.04   21.14\n",
      "    4        239K 239K N   CA   1883 1884  128.95  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -83.06  -68.20   19.18    1.23  -62.50  162.98   27.76\n",
      "    5        240A 240A N   CA   1892 1893  157.43  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    4    9   37   65   86  107  120  178  225  281\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2xby_MODLL.B99990001.pdb        50.34017\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2xc0\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26330    24627\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1845 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24627     380\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      116\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          40.6357\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.14998       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.832   1.832      6.2790       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  50.313  50.313      14.042       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.443   1.443      1.1482       1.000\n",
      " 5 Soft-sphere overlap restraints     :     116       0      0   0.001   0.001     0.20027E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.133   0.133     0.74711       1.000\n",
      "10 Distance restraints 2 (N-O)        :      36       0      0   0.351   0.351      1.2096       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.451   3.451      1.2641       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  77.261  77.261      1.7778       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  64.337  64.337     0.50153       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  91.788  91.788      3.4856       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  15.446  15.446      1.9555       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.450   0.450     0.10855       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  17.370 136.253      7.5298       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       5       0      0   0.842   0.842     0.41648       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2xc0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1499.9230\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853   53.10   57.30    6.96    0.39  -63.00  143.68   26.13\n",
      "    1        235R 235R N   CA   1853 1854   43.55   38.00                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -83.80  -80.20    3.80    0.28   82.20 -126.38    7.35\n",
      "    2        236G 236G N   CA   1864 1865  172.88  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -63.40  -70.70    7.32    0.68  -63.50  176.58   24.51\n",
      "    3        237L 237L N   CA   1868 1869  142.22  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -94.81 -118.00   25.79    1.18  -62.90  171.80   24.17\n",
      "    4        239K 239K N   CA   1883 1884  150.39  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -75.23  -68.20   16.40    1.09  -62.50  159.50   26.78\n",
      "    5        240A 240A N   CA   1892 1893  160.11  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   10    7   48   68   68  107  148  211  248  287\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2xc0_MODLL.B99990001.pdb        40.63573\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2xc4\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26221    24518\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24518     505\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      229\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         179.5756\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.012   0.012      3.3313       1.000\n",
      " 2 Bond angle potential               :     102       2      4   5.892   5.892      60.786       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  55.125  55.125      18.156       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   3.002   3.002      6.1779       1.000\n",
      " 5 Soft-sphere overlap restraints     :     229       0      2   0.019   0.019      10.209       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      67       0      0   0.438   0.438      12.204       1.000\n",
      "10 Distance restraints 2 (N-O)        :      69       0      0   0.501   0.501      9.8053       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      2  11.454  11.454      17.017       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  94.240  94.240      3.1104       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  67.270  67.270     0.79943       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  62.053  62.053      1.6896       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  47.012  47.012      1.4907       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.590   0.590     0.17250       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       9      3  33.537 137.637      30.550       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      59       0      0   0.774   0.774      4.0752       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2xc4_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2438.7212\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4477 232M 232M N   CA   1829 1830  124.96  107.00   17.96    5.16  107.00   17.96    5.16\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6826 231S 232M C   N    1827 1829  -98.97  -63.40   35.74    6.82  -63.40   35.74    6.82\n",
      "    1        232M 232M N   CA   1829 1830  -43.91  -40.50                  -40.50\n",
      "    2   6827 232M 233K C   N    1835 1837   50.27   56.60   42.52    2.59  -62.90  166.00   28.06\n",
      "    2        233K 233K N   CA   1837 1838   80.65   38.60                  -40.80\n",
      "    3   6828 233K 234T C   N    1844 1846  -60.57  -78.10   24.40    0.90  -63.20  174.96   22.61\n",
      "    3        234T 234T N   CA   1846 1847  132.84  149.80                  -42.10\n",
      "    4   6829 234T 235R C   N    1851 1853 -159.53 -125.20   52.16    1.64  -63.00  169.25   29.23\n",
      "    4        235R 235R N   CA   1853 1854  179.88  140.60                  -41.10\n",
      "    5   6830 235R 236G C   N    1862 1864  -59.23  -62.40    3.70    0.56   82.20  150.56   11.50\n",
      "    5        236G 236G N   CA   1864 1865  -43.12  -41.20                    8.50\n",
      "    6   6831 236G 237L C   N    1866 1868  -75.57  -70.70   10.84    1.00  -63.50  173.54   23.43\n",
      "    6        237L 237L N   CA   1868 1869  131.92  141.60                  -41.20\n",
      "    7   6832 237L 238P C   N    1874 1876  -63.14  -58.70   15.02    0.94  -64.50  163.36   12.12\n",
      "    7        238P 238P N   CA   1876 1877  -16.15  -30.50                  147.20\n",
      "    8   6833 238P 239K C   N    1881 1883  -56.63  -70.20   20.75    1.73  -62.90  163.23   20.86\n",
      "    8        239K 239K N   CA   1883 1884  156.09  140.40                  -40.80\n",
      "    9   6834 239K 240A C   N    1890 1892  -77.82  -68.20   62.18    4.67  -62.50  113.41   19.37\n",
      "    9        240A 240A N   CA   1892 1893 -153.27  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    9   16   44   66  107   94  157  202  288  309\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2xc4_MODLL.B99990001.pdb       179.57564\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2xc5\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26208    24505\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 3148 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.7794\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24505     374\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      100\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          47.6608\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.12352       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.809   1.809      5.6756       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  48.832  48.832      14.196       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.556   1.556      1.4398       1.000\n",
      " 5 Soft-sphere overlap restraints     :     100       0      0   0.002   0.002     0.52402E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.137   0.137     0.72517       1.000\n",
      "10 Distance restraints 2 (N-O)        :      37       0      0   0.487   0.487      2.9575       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.729   2.729     0.79040       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  97.294  97.294      1.8177       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  77.753  77.753      1.2364       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  60.756  60.756      3.8671       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  85.210  85.210      3.0587       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.741   0.741     0.31915       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  23.412 131.340      10.667       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   2.490   2.490     0.73409       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2xc5_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1760.0337\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -80.79  -72.10    9.11    0.75  -63.00 -179.39   25.61\n",
      "    1        235R 235R N   CA   1853 1854  139.17  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   62.08   78.70   19.12    0.64   82.20  166.38    9.08\n",
      "    2        236G 236G N   CA   1864 1865 -156.65 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -102.50 -108.50    7.72    0.43  -63.50 -177.23   23.44\n",
      "    3        237L 237L N   CA   1868 1869  137.36  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -104.34 -118.00   19.74    1.00  -62.90  170.95   24.64\n",
      "    4        239K 239K N   CA   1883 1884  153.35  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -137.29 -134.00   20.36    1.07  -62.50  169.41   31.77\n",
      "    5        240A 240A N   CA   1892 1893  167.09  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    5   11   34   63   86  103  121  195  252  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2xc5_MODLL.B99990001.pdb        47.66078\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2y5f\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26350    24647\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 3148 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.3277\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.5687\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.6770\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24647     506\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      254\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         216.2813\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.015   0.015      5.1044       1.000\n",
      " 2 Bond angle potential               :     102       2      7   4.692   4.692      43.986       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      2  53.465  53.465      19.224       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.525   3.525      9.1103       1.000\n",
      " 5 Soft-sphere overlap restraints     :     254       0      3   0.022   0.022      15.074       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      67       0      1   0.584   0.584      22.170       1.000\n",
      "10 Distance restraints 2 (N-O)        :      69       0      3   0.698   0.698      21.372       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  16.396  16.396      34.875       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  81.719  81.719      1.7368       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  64.905  64.905      2.4259       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  89.974  89.974      3.9144       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  11.158  11.158      1.3320       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.481   0.481     0.11733       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      3  42.787 130.268      34.751       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      58       0      0   0.400   0.400      1.0891       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y5f_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2481.8137\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  139.84  120.00   19.84    4.51  120.00   19.84    4.51\n",
      "    2   4477 232M 232M N   CA   1829 1830  127.75  107.00   20.75    5.97  107.00   20.75    5.97\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   -1.48  -70.20   71.03    4.76  -62.90  174.41   26.32\n",
      "    1        233K 233K N   CA   1837 1838  122.44  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -77.79  -78.10    8.92    0.37  -63.20  177.62   23.74\n",
      "    2        234T 234T N   CA   1846 1847  140.88  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -144.59 -125.20   46.29    1.74  -63.00  158.83   26.92\n",
      "    3        235R 235R N   CA   1853 1854 -177.37  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -59.51  -62.40    4.63    0.64   82.20  151.41   11.58\n",
      "    4        236G 236G N   CA   1864 1865  -44.82  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -96.54 -108.50   12.01    0.59  -63.50  177.91   23.02\n",
      "    5        237L 237L N   CA   1868 1869  133.61  132.50                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883 -164.98 -118.00   54.85    1.82  -62.90 -177.08   29.80\n",
      "    6        239K 239K N   CA   1883 1884  167.41  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  -81.19  -68.20   83.43    6.26  -62.50   93.28   16.27\n",
      "    7        240A 240A N   CA   1892 1893 -132.29  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   166  182   2.117\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    1    1    9   16   41   70   95   99  148  212  276  257\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y5f_MODLL.B99990001.pdb       216.28131\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2y5g\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26333    24630\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 3148 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.3426\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.6663\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       95.7896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24630     507\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      265\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         219.2027\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.018   0.018      6.8095       1.000\n",
      " 2 Bond angle potential               :     102       3      6   5.053   5.053      50.368       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      2  56.785  56.785      20.167       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.273   3.273      8.2178       1.000\n",
      " 5 Soft-sphere overlap restraints     :     265       1      3   0.024   0.024      17.693       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      67       0      0   0.541   0.541      18.747       1.000\n",
      "10 Distance restraints 2 (N-O)        :      69       0      2   0.646   0.646      17.927       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  15.633  15.633      31.702       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  90.738  90.738      3.2797       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  75.287  75.287      2.9448       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  73.113  73.113      2.4761       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  92.643  92.643      2.2771       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.504   0.504     0.12943       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      4  48.411 131.481      35.407       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      59       0      0   0.387   0.387      1.0588       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y5g_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2605.0513\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  141.88  120.00   21.88    4.97  120.00   21.88    4.97\n",
      "    2   4477 232M 232M N   CA   1829 1830  127.71  107.00   20.71    5.96  107.00   20.71    5.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837    1.06  -70.20   74.76    4.92  -62.90  170.98   26.03\n",
      "    1        233K 233K N   CA   1837 1838  117.76  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -75.22  -78.10   14.90    0.55  -63.20  177.69   22.01\n",
      "    2        234T 234T N   CA   1846 1847  135.18  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -143.38 -125.20   40.99    1.51  -63.00  162.79   27.40\n",
      "    3        235R 235R N   CA   1853 1854  177.34  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -60.38  -62.40    2.19    0.41   82.20  150.72   11.46\n",
      "    4        236G 236G N   CA   1864 1865  -40.36  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -87.34  -70.70   17.84    1.77  -63.50  177.98   23.45\n",
      "    5        237L 237L N   CA   1868 1869  135.18  141.60                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883  162.48 -118.00   90.78    3.01  -62.90 -168.42   32.56\n",
      "    6        239K 239K N   CA   1883 1884 -177.10  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  -87.49  -68.20   84.01    6.09  -62.50   95.36   16.95\n",
      "    7        240A 240A N   CA   1892 1893 -132.93  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   166  182   2.163\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    0    2    9   17   41   70   92  111  137  216  272  284\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y5g_MODLL.B99990001.pdb       219.20267\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGEQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2y5h\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26266    24563\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "74 (of 1906 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.7227\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24563     506\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      268\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         229.6664\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.018   0.018      7.3835       1.000\n",
      " 2 Bond angle potential               :     102       3      7   5.532   5.532      58.825       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      6  63.162  63.162      23.055       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.132   3.132      7.4536       1.000\n",
      " 5 Soft-sphere overlap restraints     :     268       1      3   0.026   0.026      21.252       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      67       0      0   0.533   0.533      17.985       1.000\n",
      "10 Distance restraints 2 (N-O)        :      68       0      1   0.545   0.545      11.792       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  14.241  14.241      26.309       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  72.773  72.773      1.3624       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  57.137  57.137      1.8770       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  83.036  83.036      3.8222       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  53.601  53.601      2.1235       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.524   0.524     0.14250       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      6  59.345 121.826      44.950       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      59       0      0   0.438   0.438      1.3334       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y5h_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2347.6970\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  144.19  120.00   24.18    5.50  120.00   24.18    5.50\n",
      "    2   4477 232M 232M N   CA   1829 1830  129.05  107.00   22.05    6.34  107.00   22.05    6.34\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837  -14.83  -70.20   62.38    3.90  -62.90  159.87   23.62\n",
      "    1        233K 233K N   CA   1837 1838  111.67  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -115.56 -124.80   27.03    0.91  -63.20  168.53   18.67\n",
      "    2        234T 234T N   CA   1846 1847  118.09  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -120.31 -125.20   42.13    1.92  -63.00  151.14   17.42\n",
      "    3        235R 235R N   CA   1853 1854   98.76  140.60                  -41.10\n",
      "    4   6831 236G 237L C   N    1866 1868   19.08  -70.70  110.10    7.52  -63.50  144.91   25.14\n",
      "    4        237L 237L N   CA   1868 1869   77.87  141.60                  -41.20\n",
      "    5   6832 237L 238P C   N    1874 1876  -69.54  -58.70   15.01    2.04  -64.50  172.00   12.57\n",
      "    5        238P 238P N   CA   1876 1877  -40.88  -30.50                  147.20\n",
      "    6   6833 238P 239K C   N    1881 1883 -179.28 -118.00  105.21    3.98  -62.90  149.97   25.89\n",
      "    6        239K 239K N   CA   1883 1884 -135.38  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  -90.05  -68.20   78.02    5.53  -62.50  102.67   18.29\n",
      "    7        240A 240A N   CA   1892 1893 -139.80  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    4   10   16   30   73   95  112  145  206  269  301\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y5h_MODLL.B99990001.pdb       229.66640\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2y7x\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26517    24814\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.1558\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24814     377\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      150\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          70.3862\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.005   0.005     0.74314       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.859   1.859      6.3437       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  51.389  51.389      14.760       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   2.150   2.150      2.3805       1.000\n",
      " 5 Soft-sphere overlap restraints     :     150       1      3   0.031   0.031      17.439       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      47       0      0   0.235   0.235      2.3863       1.000\n",
      "10 Distance restraints 2 (N-O)        :      48       0      0   0.410   0.410      3.7672       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.881   2.881     0.88105       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  99.759  99.759      2.1827       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  80.793  80.793      1.9209       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  25.596  25.596      1.6084       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  90.087  90.087      2.3858       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      17       0      0   0.610   0.610     0.18365       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  34.485 131.382      13.404       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y7x_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1852.0875\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -162.92 -125.20   48.01    1.42  -63.00  179.06   30.80\n",
      "    1        235R 235R N   CA   1853 1854  170.31  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   93.98   78.70   37.95    0.57   82.20  151.11    7.93\n",
      "    2        236G 236G N   CA   1864 1865  159.16 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -132.70 -108.50   26.86    1.19  -63.50 -172.15   30.37\n",
      "    3        237L 237L N   CA   1868 1869  144.16  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -81.92  -70.20   17.63    1.06  -62.90  166.72   22.74\n",
      "    4        239K 239K N   CA   1883 1884  153.57  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -67.78  -68.20    3.51    0.27  -62.50  177.37   29.33\n",
      "    5        240A 240A N   CA   1892 1893  141.81  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    0    4   17   35   81   75  106  156  194  285  295\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y7x_MODLL.B99990001.pdb        70.38625\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2y7z\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26317    24614\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.5344\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.2403\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24614     371\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       92\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          33.4479\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.12125       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.665   1.665      5.3714       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.764  47.764      12.336       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   0.950   0.950     0.39919       1.000\n",
      " 5 Soft-sphere overlap restraints     :      92       0      0   0.001   0.001     0.23051E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.127   0.127     0.77621       1.000\n",
      "10 Distance restraints 2 (N-O)        :      45       0      0   0.243   0.243     0.97957       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.530   3.530      1.3225       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  67.159  67.159     0.12262       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  77.822  77.822     0.37896       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  96.603  96.603      3.7231       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  44.642  44.642      1.6592       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      16       0      0   0.249   0.249     0.19529E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  17.718 138.804      6.2152       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y7z_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1603.2877\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -96.97  -72.10   24.89    1.92  -63.00 -178.96   26.80\n",
      "    1        235R 235R N   CA   1853 1854  141.08  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -74.18  -80.20   23.39    0.64   82.20 -148.10    6.98\n",
      "    2        236G 236G N   CA   1864 1865  151.50  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -86.80  -70.70   16.21    1.53  -63.50 -179.42   26.47\n",
      "    3        237L 237L N   CA   1868 1869  139.73  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -71.55  -70.20    5.35    0.36  -62.90  173.84   23.06\n",
      "    4        239K 239K N   CA   1883 1884  145.57  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -126.47 -134.00    9.75    0.55  -62.50  177.82   32.56\n",
      "    5        240A 240A N   CA   1892 1893  153.19  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    3   12   31   68   75  105  136  196  258  304\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y7z_MODLL.B99990001.pdb        33.44788\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2y80\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26314    24611\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1845 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24611     373\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      180\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          70.9240\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.57815       1.000\n",
      " 2 Bond angle potential               :      94       0      1   2.121   2.121      7.4411       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.305  49.305      14.264       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   2.701   2.701      3.0538       1.000\n",
      " 5 Soft-sphere overlap restraints     :     180       1      3   0.032   0.032      22.045       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.099   0.099     0.36218       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.399   0.399      2.8602       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.969   2.969     0.93540       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  83.290  83.290      1.5501       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  98.867  98.867      2.4446       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  81.445  81.445      3.6541       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  46.910  46.910      1.5632       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      16       0      0   0.666   0.666     0.20606       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  31.884 136.130      9.9660       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y80_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1818.4985\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -145.73 -125.20   20.55    0.85  -63.00 -162.60   32.32\n",
      "    1        235R 235R N   CA   1853 1854  139.68  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   67.40   78.70   15.54    0.84   82.20  175.35    7.76\n",
      "    2        236G 236G N   CA   1864 1865 -176.77 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -109.80 -108.50   17.22    0.94  -63.50  163.24   20.52\n",
      "    3        237L 237L N   CA   1868 1869  115.33  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -101.01 -118.00   17.46    0.73  -62.90 -179.86   25.63\n",
      "    4        239K 239K N   CA   1883 1884  143.13  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -67.46  -68.20    0.85    0.08  -62.50  173.45   28.67\n",
      "    5        240A 240A N   CA   1892 1893  145.72  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    7   14   34   72   68  109  155  194  259  306\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y80_MODLL.B99990001.pdb        70.92402\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 2y81\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26285    24582\n",
      "74 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24582     501\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      239\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         185.3587\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.015   0.015      3.9774       1.000\n",
      " 2 Bond angle potential               :     102       3      6   6.328   6.328      69.997       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      1  53.031  53.031      17.433       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      0   2.490   2.490      3.4136       1.000\n",
      " 5 Soft-sphere overlap restraints     :     239       0      1   0.016   0.016      7.1886       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       0      6   0.689   0.689      29.674       1.000\n",
      "10 Distance restraints 2 (N-O)        :      66       0      2   0.764   0.764      22.903       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       0      0   4.564   4.564      2.7026       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  68.433  68.433      2.6444       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  66.947  66.947      2.6323       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 107.435 107.435      4.6964       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  73.127  73.127      2.3987       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.504   0.504     0.11757       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      5  32.349 132.501      13.729       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      60       0      0   0.511   0.511      1.8503       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y81_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2295.7742\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  142.64  120.00   22.64    5.14  120.00   22.64    5.14\n",
      "    2   4477 232M 232M N   CA   1829 1830  127.54  107.00   20.54    5.91  107.00   20.54    5.91\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -131.33 -118.00   22.06    0.82  -62.90  176.34   26.99\n",
      "    1        233K 233K N   CA   1837 1838  156.68  139.10                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -122.62 -124.80   22.77    1.04  -63.20  162.96   24.82\n",
      "    2        234T 234T N   CA   1846 1847  166.17  143.50                  -42.10\n",
      "    3   6830 235R 236G C   N    1862 1864  -98.11  -80.20   53.55    1.34   82.20 -129.77    8.21\n",
      "    3        236G 236G N   CA   1864 1865 -135.44  174.10                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868  -55.52  -70.70   23.67    2.32  -63.50  159.23   21.66\n",
      "    4        237L 237L N   CA   1868 1869  159.77  141.60                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883 -167.27 -118.00   60.14    2.02  -62.90  179.15   29.42\n",
      "    5        239K 239K N   CA   1883 1884  173.59  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892 -137.09 -134.00   39.68    2.18  -62.50  152.09   28.91\n",
      "    6        240A 240A N   CA   1892 1893 -173.44  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    5   14   47   75   82  104  154  212  263  285\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y81_MODLL.B99990001.pdb       185.35866\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 2y82\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26311    24608\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.2276\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.7121\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24608     370\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      164\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          76.3452\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.007   0.007      1.3642       1.000\n",
      " 2 Bond angle potential               :      94       0      1   2.355   2.355      9.2477       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  47.995  47.995      13.173       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   2.784   2.784      2.2270       1.000\n",
      " 5 Soft-sphere overlap restraints     :     164       1      5   0.037   0.037      26.024       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.166   0.166      1.3286       1.000\n",
      "10 Distance restraints 2 (N-O)        :      43       0      0   0.395   0.395      2.6782       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.446   2.446     0.63520       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  80.304  80.304      2.3509       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0 107.995 107.995      2.7605       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  53.054  53.054      2.5292       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  75.774  75.774      3.7110       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      15       0      0   0.270   0.270     0.28663E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      0  31.086 135.623      7.8982       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       4       0      0   1.098   1.098     0.38916       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y82_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1780.8600\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -169.73 -125.20   56.92    1.69  -63.00  178.32   31.06\n",
      "    1        235R 235R N   CA   1853 1854  176.05  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -72.72  -80.20   19.23    0.46   82.20 -145.83    6.88\n",
      "    2        236G 236G N   CA   1864 1865  156.39  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -81.13  -70.70   21.07    1.31  -63.50  159.87   23.25\n",
      "    3        237L 237L N   CA   1868 1869  159.91  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -87.86  -70.20   25.16    1.50  -62.90  162.81   22.59\n",
      "    4        239K 239K N   CA   1883 1884  158.32  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892 -101.80 -134.00   32.56    0.91  -62.50 -178.73   31.77\n",
      "    5        240A 240A N   CA   1892 1893  142.15  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3    7   12   42   61   78  100  152  213  241  311\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y82_MODLL.B99990001.pdb        76.34522\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3cen\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        1\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26379    24676\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24676     385\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      127\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          42.6636\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.63639E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.543   1.543      4.5952       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  52.932  52.932      14.451       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.794   2.794      5.7478       1.000\n",
      " 5 Soft-sphere overlap restraints     :     127       0      0   0.001   0.001     0.21868E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.265   0.265      2.5586       1.000\n",
      "10 Distance restraints 2 (N-O)        :      41       0      0   0.250   0.250      1.7175       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.884   2.884     0.88288       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  77.890  77.890     0.64260       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  53.068  53.068    -0.58967       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  58.453  58.453      2.2562       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  62.790  62.790      2.5950       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      30       0      0   0.250   0.250     0.58703E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  17.332 115.048      7.4599       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       6       0      0   0.633   0.633     0.20268       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3cen_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1380.1217\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6828 233K 234T C   N    1844 1846  -60.60  -63.20    3.87    0.42 -124.80 -176.85    9.69\n",
      "    1        234T 234T N   CA   1846 1847  -44.97  -42.10                  143.50\n",
      "    2   6829 234T 235R C   N    1851 1853   58.38   57.30    4.39    0.23  -63.00  142.60   26.00\n",
      "    2        235R 235R N   CA   1853 1854   33.74   38.00                  -41.10\n",
      "    3   6831 236G 237L C   N    1866 1868  -66.66  -70.70    8.60    0.80  -63.50  169.64   23.74\n",
      "    3        237L 237L N   CA   1868 1869  149.19  141.60                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892 -137.32 -134.00    5.11    0.17  -62.50 -175.89   34.18\n",
      "    4        240A 240A N   CA   1892 1893  150.88  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    4   14   36   66   91  120  190  222  222  276\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3cen_MODLL.B99990001.pdb        42.66359\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3cs7\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26341    24638\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1879 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24638     393\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      107\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          65.4839\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.15074       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.940   1.940      7.0421       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.360  47.360      12.151       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       1      1   3.043   3.043      6.7465       1.000\n",
      " 5 Soft-sphere overlap restraints     :     107       0      0   0.002   0.002     0.53195E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      48       0      0   0.266   0.266      3.0092       1.000\n",
      "10 Distance restraints 2 (N-O)        :      50       0      0   0.344   0.344      3.6112       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.846   4.846      2.4925       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  91.420  91.420      3.1658       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  83.969  83.969      1.6745       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  61.262  61.262      1.6312       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 115.075 115.075      2.9033       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.719   0.719     0.26422       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  33.945 118.172      19.381       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       5       0      0   1.659   1.659      1.2079       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3cs7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1612.9247\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6574 232M 232M C   CA   1835 1830   13.06    0.00   13.06    4.59    0.00   13.06    4.59\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -106.78 -125.20   22.67    0.67  -63.00  174.07   20.93\n",
      "    1        235R 235R N   CA   1853 1854  127.38  140.60                  -41.10\n",
      "    2   6831 236G 237L C   N    1866 1868  -89.87  -70.70   29.99    1.87  -63.50  156.38   23.31\n",
      "    2        237L 237L N   CA   1868 1869  164.66  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -66.55  -70.20   25.58    1.94  -62.90  153.52   20.14\n",
      "    3        239K 239K N   CA   1883 1884  165.72  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -57.02  -68.20   11.54    0.88  -62.50  176.76   28.70\n",
      "    4        240A 240A N   CA   1892 1893  142.42  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   11   17   37   66  108  125  158  199  226  316\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3cs7_MODLL.B99990001.pdb        65.48392\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTRGLP\n",
      "Modelando proteína 3ens\n",
      "{'num_gaps': 1, 'gap_lengths': [3], 'gap_list': [[239, 241]], 'gap_window': [[237, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26560    24857\n",
      "39 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      39\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24857     225\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       46\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          15.2745\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      40       0      0   0.003   0.003     0.97876E-01   1.000\n",
      " 2 Bond angle potential               :      54       0      0   2.035   2.035      4.0782       1.000\n",
      " 3 Stereochemical cosine torsion poten:      26       0      0  41.387  41.387      4.9339       1.000\n",
      " 4 Stereochemical improper torsion pot:      12       0      0   1.285   1.285     0.38493       1.000\n",
      " 5 Soft-sphere overlap restraints     :      46       0      0   0.003   0.003     0.41270E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      25       0      0   0.209   0.209     0.50915       1.000\n",
      "10 Distance restraints 2 (N-O)        :      18       0      0   0.230   0.230     0.72705E-01   1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       5       0      0   2.419   2.419     0.34506       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       4       0      0  77.662  77.662    -0.91757E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       4       0      0  87.005  87.005     0.32996       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 101.125 101.125      2.6191       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 104.188 104.188      1.9669       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      22       0      0   0.259   0.259     0.34885E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       5       1      0  14.725  70.869    -0.31472       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       5       0      0   0.633   0.633     0.26703       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3ens_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:     621.0099\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6834 239K 240A C   N    1890 1892  -80.95  -68.20   22.29    1.42  -62.50  156.60   26.61\n",
      "    1        240A 240A N   CA   1892 1893  163.59  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    8   15   41   79   74  120  177  207  256  293\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3ens_MODLL.B99990001.pdb        15.27448\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3ffg\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        2\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26210    24507\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1848 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.0844\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24507     397\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          45.4285\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.96499E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.687   1.687      5.4436       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  48.103  48.103      11.600       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.422   2.422      4.1462       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.001   0.001     0.16500E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.260   0.260      2.6311       1.000\n",
      "10 Distance restraints 2 (N-O)        :      49       0      0   0.261   0.261      2.5109       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.706   4.706      2.3506       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  47.495  47.495     0.50642       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  55.530  55.530    -0.45466E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  54.742  54.742      1.6775       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  69.943  69.943      2.3875       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.328   0.328     0.89180E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  28.809 123.551      11.860       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      10       0      0   0.500   0.500     0.15806       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3ffg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1201.3506\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864  -73.12  -80.20   13.71    0.33   82.20 -141.37    6.87\n",
      "    1        236G 236G N   CA   1864 1865  162.36  174.10                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -80.20  -70.70    9.97    0.80  -63.50  175.00   25.29\n",
      "    2        237L 237L N   CA   1868 1869  144.60  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -93.95  -70.20   32.86    1.97  -62.90  159.16   22.48\n",
      "    3        239K 239K N   CA   1883 1884  163.10  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -66.86  -68.20    8.01    0.69  -62.50  165.96   27.41\n",
      "    4        240A 240A N   CA   1892 1893  153.20  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   14   42   57   96  112  161  195  232  295\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3ffg_MODLL.B99990001.pdb        45.42849\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTRGLP\n",
      "Modelando proteína 3hpt\n",
      "{'num_gaps': 1, 'gap_lengths': [3], 'gap_list': [[239, 241]], 'gap_window': [[237, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26515    24812\n",
      "39 (of 1906 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.5260\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      39\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24812     234\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       38\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          16.3311\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      40       0      0   0.002   0.002     0.48708E-01   1.000\n",
      " 2 Bond angle potential               :      54       0      0   1.975   1.975      4.0008       1.000\n",
      " 3 Stereochemical cosine torsion poten:      26       0      0  47.037  47.037      7.0821       1.000\n",
      " 4 Stereochemical improper torsion pot:      12       0      0   1.281   1.281     0.37639       1.000\n",
      " 5 Soft-sphere overlap restraints     :      38       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      26       0      0   0.112   0.112     0.12697       1.000\n",
      "10 Distance restraints 2 (N-O)        :      18       0      0   0.139   0.139     0.23554E-01   1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       5       0      0   0.807   0.807     0.38374E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       4       0      0  68.817  68.817      1.6609       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       4       0      0  66.483  66.483     0.80088E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  71.594  71.594     0.91234       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  73.598  73.598      1.3438       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      23       0      0   0.779   0.779     0.48513       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       5       1      0  10.194  82.199    -0.82188       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      12       0      0   0.766   0.766     0.97377       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3hpt_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:     560.3028\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6834 239K 240A C   N    1890 1892 -119.07 -134.00   15.06    0.43  -62.50 -176.99   33.01\n",
      "    1        240A 240A N   CA   1892 1893  145.06  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    36 1120   2.213\n",
      "  serious non-bonded atom clash:    37 1120   1.923\n",
      "  serious non-bonded atom clash:    37 1123   1.918\n",
      "  serious non-bonded atom clash:    38 1120   1.775\n",
      "  serious non-bonded atom clash:    38 1121   1.715\n",
      "  serious non-bonded atom clash:    38 1122   1.486\n",
      "  serious non-bonded atom clash:    38 1123   1.147\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1787 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     6    0    1    6    3    3   13   39   50   84  111  144  210  251  291\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3hpt_MODLL.B99990001.pdb        16.33108\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 3iit\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26268    24565\n",
      "74 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1860 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       93.2685\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24565     496\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      269\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         218.6392\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      0   0.012   0.012      3.1199       1.000\n",
      " 2 Bond angle potential               :     102       2      3   4.262   4.262      35.488       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      2  59.723  59.723      22.714       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.523   3.523      7.8569       1.000\n",
      " 5 Soft-sphere overlap restraints     :     269       0      2   0.021   0.021      13.585       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       0      2   0.632   0.632      24.688       1.000\n",
      "10 Distance restraints 2 (N-O)        :      67       0      1   0.686   0.686      17.765       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  19.155  19.155      47.595       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  81.397  81.397      4.4572       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  73.219  73.219      3.1851       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  82.804  82.804      4.0038       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  95.872  95.872      1.4149       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.708   0.708     0.26232       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      6  54.260 135.753      27.645       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      54       0      0   0.997   0.997      4.8606       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3iit_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2594.0061\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  140.93  120.00   20.93    4.76  120.00   20.93    4.76\n",
      "    2   4477 232M 232M N   CA   1829 1830  124.79  107.00   17.79    5.12  107.00   17.79    5.12\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   89.58   56.60  127.90   11.71  -62.90 -141.13   26.48\n",
      "    1        233K 233K N   CA   1837 1838  162.18   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -121.67 -124.80   54.88    2.26  -63.20  143.28   15.39\n",
      "    2        234T 234T N   CA   1846 1847   88.71  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -147.03 -125.20   23.21    0.74  -63.00 -169.99   31.38\n",
      "    3        235R 235R N   CA   1853 1854  148.48  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -61.61  -62.40    0.80    0.14   82.20  152.20   11.58\n",
      "    4        236G 236G N   CA   1864 1865  -41.32  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -91.36  -70.70   22.11    1.71  -63.50  171.62   25.51\n",
      "    5        237L 237L N   CA   1868 1869  149.46  141.60                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883 -148.87 -118.00   69.71    2.84  -62.90  145.67   23.98\n",
      "    6        239K 239K N   CA   1883 1884 -158.39  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892 -161.68 -134.00   68.25    3.01  -62.50  147.90   29.34\n",
      "    7        240A 240A N   CA   1892 1893 -150.62  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   342  590   2.057\n",
      "  serious non-bonded atom clash:   503  807   1.892\n",
      "  serious non-bonded atom clash:   504  806   2.171\n",
      "  serious non-bonded atom clash:   504  807   1.904\n",
      "  serious non-bonded atom clash:   504  808   2.066\n",
      "  serious non-bonded atom clash:   505  806   1.217\n",
      "  serious non-bonded atom clash:   505  807   1.758\n",
      "  serious non-bonded atom clash:   505  808   0.914\n",
      "  serious non-bonded atom clash:  1387 1403   1.956\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     8    1    0    0    3   14   29   47   94  107  104  170  209  285  287\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3iit_MODLL.B99990001.pdb       218.63918\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKTRGLP\n",
      "Modelando proteína 3k9x\n",
      "{'num_gaps': 1, 'gap_lengths': [3], 'gap_list': [[239, 241]], 'gap_window': [[237, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26495    24792\n",
      "39 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1836 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      39\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24792     230\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       34\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          23.9362\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      40       0      0   0.003   0.003     0.94673E-01   1.000\n",
      " 2 Bond angle potential               :      54       0      0   2.132   2.132      4.6936       1.000\n",
      " 3 Stereochemical cosine torsion poten:      26       0      2  57.075  57.075      10.309       1.000\n",
      " 4 Stereochemical improper torsion pot:      12       0      0   1.296   1.296     0.40815       1.000\n",
      " 5 Soft-sphere overlap restraints     :      34       0      0   0.002   0.002     0.21060E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      25       0      0   0.238   0.238     0.37570       1.000\n",
      "10 Distance restraints 2 (N-O)        :      18       0      0   0.182   0.182     0.81414E-01   1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       5       0      0   2.272   2.272     0.30435       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       4       0      0  56.572  56.572     0.65670       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       4       0      0  62.806  62.806     0.76036       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  18.698  18.698     0.43152       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  80.358  80.358      1.1578       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      20       0      0   0.591   0.591     0.21137       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       5       3      0  30.147 128.883      3.5935       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      12       0      0   0.715   0.715     0.83693       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3k9x_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:     856.0449\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6832 237L 238P C   N    1874 1876  -64.05  -58.70    5.61    0.61  -64.50  176.02   13.11\n",
      "    1        238P 238P N   CA   1876 1877  -28.82  -30.50                  147.20\n",
      "    2   6833 238P 239K C   N    1881 1883   52.01   56.60    6.92    0.41  -62.90  142.68   24.76\n",
      "    2        239K 239K N   CA   1883 1884   43.78   38.60                  -40.80\n",
      "    3   6834 239K 240A C   N    1890 1892 -133.20 -134.00   21.75    1.25  -62.50  166.16   31.02\n",
      "    3        240A 240A N   CA   1892 1893  168.73  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   449 1123   2.019\n",
      "  serious non-bonded atom clash:   450 1122   2.293\n",
      "  serious non-bonded atom clash:   450 1123   1.304\n",
      "  serious non-bonded atom clash:   451 1122   1.479\n",
      "  serious non-bonded atom clash:   451 1123   0.782\n",
      "  serious non-bonded atom clash:   451 1124   2.014\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    0    1    1    4    2   18   35   69   79  113  143  211  241  269\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3k9x_MODLL.B99990001.pdb        23.93619\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSM\n",
      "Modelando proteína 3kl6\n",
      "{'num_gaps': 1, 'gap_lengths': [9], 'gap_list': [[233, 241]], 'gap_window': [[231, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26149    24446\n",
      "84 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1832 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      84\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24446     522\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      122\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          36.2714\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      85       0      0   0.003   0.003     0.20775       1.000\n",
      " 2 Bond angle potential               :     112       0      0   1.459   1.459      4.6305       1.000\n",
      " 3 Stereochemical cosine torsion poten:      45       0      0  51.430  51.430      17.937       1.000\n",
      " 4 Stereochemical improper torsion pot:      25       0      0   0.886   0.886     0.40132       1.000\n",
      " 5 Soft-sphere overlap restraints     :     122       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      64       0      0   0.070   0.070     0.28186       1.000\n",
      "10 Distance restraints 2 (N-O)        :      63       0      0   0.154   0.154      1.4326       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       0      0   1.682   1.682     0.36711       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  67.195  67.195     0.13452       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       7       0      0  58.063  58.063     0.75567       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  62.421  62.421      3.4920       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  73.015  73.015      3.0710       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      23       0      0   0.095   0.095     0.42056E-02   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      0  15.925 141.464      3.0594       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      57       0      0   0.269   0.269     0.49667       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3kl6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1946.0150\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837 -132.73 -118.00   15.63    0.69  -62.90 -171.89   21.74\n",
      "    1        233K 233K N   CA   1837 1838  133.87  139.10                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -92.37  -78.10   15.58    0.75  -63.20  164.46   23.04\n",
      "    2        234T 234T N   CA   1846 1847  156.05  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -120.26 -125.20    5.62    0.17  -63.00 -172.05   22.15\n",
      "    3        235R 235R N   CA   1853 1854  137.92  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864   87.28   78.70   32.36    0.59   82.20  154.27    7.75\n",
      "    4        236G 236G N   CA   1864 1865  162.69 -166.10                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868 -132.27 -108.50   30.27    1.34  -63.50 -178.88   29.41\n",
      "    5        237L 237L N   CA   1868 1869  151.24  132.50                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883  -75.66  -70.20    6.18    0.51  -62.90  178.76   22.55\n",
      "    6        239K 239K N   CA   1883 1884  137.50  140.40                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892 -130.27 -134.00    4.18    0.21  -62.50 -176.79   33.65\n",
      "    7        240A 240A N   CA   1892 1893  148.89  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0   13   35   60   83  100  142  200  261  266\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3kl6_MODLL.B99990001.pdb        36.27145\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3kqb\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26262    24559\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1827 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.0092\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24559     385\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      163\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         326.3173\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       2      3   0.037   0.037      30.428       1.000\n",
      " 2 Bond angle potential               :      94       0      3   3.350   3.350      19.076       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  45.133  45.133      11.782       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.238   1.238     0.89677       1.000\n",
      " 5 Soft-sphere overlap restraints     :     163      13     19   0.112   0.112      239.23       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.208   0.208      1.9412       1.000\n",
      "10 Distance restraints 2 (N-O)        :      40       0      0   0.698   0.698      6.2704       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.334   3.334      1.1796       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  65.794  65.794     0.80948       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  54.158  54.158    -0.53383       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  82.014  82.014      4.1554       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 105.330 105.330      3.1182       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      26       0      0   0.383   0.383     0.76836E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  25.422 126.864      6.1411       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      10       0      0   1.593   1.593      1.7506       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3kqb_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1714.3295\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1895 235R 235R NE  CD   1858 1857    1.75    1.49    0.26    7.85    1.49    0.26    7.85\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -132.69 -125.20   14.90    0.53  -63.00  179.49   29.03\n",
      "    1        235R 235R N   CA   1853 1854  153.49  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -83.08  -80.20    5.75    0.36   82.20 -129.53    7.32\n",
      "    2        236G 236G N   CA   1864 1865  169.12  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -114.13 -108.50   29.31    1.52  -63.50  165.47   26.10\n",
      "    3        237L 237L N   CA   1868 1869  161.26  132.50                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892  -79.32  -68.20   13.96    0.90  -62.50  166.22   28.10\n",
      "    4        240A 240A N   CA   1892 1893  153.74  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   504  806   2.296\n",
      "  serious non-bonded atom clash:   504  808   1.862\n",
      "  serious non-bonded atom clash:   505  805   1.949\n",
      "  serious non-bonded atom clash:   505  806   1.542\n",
      "  serious non-bonded atom clash:   505  808   0.943\n",
      "  serious non-bonded atom clash:   558 1858   2.145\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    1    3    4    8   33   37   44   67   94  136  203  274  313\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3kqb_MODLL.B99990001.pdb       326.31729\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3kqc\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26384    24681\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1835 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.9851\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.1086\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24681     388\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      170\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         359.5389\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       2      3   0.038   0.038      31.742       1.000\n",
      " 2 Bond angle potential               :      94       2      4   4.125   4.125      29.322       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  49.485  49.485      14.953       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.298   1.298     0.84165       1.000\n",
      " 5 Soft-sphere overlap restraints     :     170      12     19   0.112   0.112      251.77       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.213   0.213      1.9370       1.000\n",
      "10 Distance restraints 2 (N-O)        :      42       0      0   0.303   0.303      1.7851       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   5.200   5.200      2.8702       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  61.359  61.359      1.3621       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0 105.153 105.153      2.9597       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  62.194  62.194      4.8198       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      1 105.764 105.764      9.1480       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.222   0.222     0.27481E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      0  17.876 121.757      4.1342       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      12       0      0   1.367   1.367      1.8659       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3kqc_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1961.9243\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1895 235R 235R NE  CD   1858 1857    1.77    1.49    0.28    8.23    1.49    0.28    8.23\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4513 235R 235R CG  CD   1856 1857  124.77  107.50   17.27    4.57  107.50   17.27    4.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -125.56 -125.20   12.34    0.59  -63.00  177.37   28.27\n",
      "    1        235R 235R N   CA   1853 1854  152.93  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -61.06  -62.40    2.91    0.51   82.20  150.81   11.43\n",
      "    2        236G 236G N   CA   1864 1865  -38.62  -41.20                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -91.41 -108.50   26.77    1.23  -63.50  155.62   20.18\n",
      "    3        237L 237L N   CA   1868 1869  111.90  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -84.42  -70.20   26.85    1.64  -62.90  157.51   21.69\n",
      "    4        239K 239K N   CA   1883 1884  163.17  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -63.65  -68.20    5.84    0.58  -62.50  170.14   27.94\n",
      "    5        240A 240A N   CA   1892 1893  148.97  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   504  806   2.278\n",
      "  serious non-bonded atom clash:   504  808   2.004\n",
      "  serious non-bonded atom clash:   505  805   1.981\n",
      "  serious non-bonded atom clash:   505  806   1.691\n",
      "  serious non-bonded atom clash:   505  808   1.323\n",
      "  serious non-bonded atom clash:   558 1857   2.279\n",
      "  serious non-bonded atom clash:   558 1858   2.085\n",
      "  serious non-bonded atom clash:   736 1858   2.290\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    0    3    2    2   10   22   51   60   71   90  144  205  278  298\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3kqc_MODLL.B99990001.pdb       359.53888\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3kqd\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26161    24458\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1839 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.4004\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24458     384\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      103\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          36.6685\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.32899       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.593   1.593      4.7122       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  46.770  46.770      11.558       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.137   1.137     0.72197       1.000\n",
      " 5 Soft-sphere overlap restraints     :     103       0      0   0.000   0.000     0.11278E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.093   0.093     0.38849       1.000\n",
      "10 Distance restraints 2 (N-O)        :      44       0      0   0.373   0.373      4.4918       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.304   4.304      1.9660       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  67.296  67.296     0.50051       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  85.716  85.716      1.5275       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  61.014  61.014      1.8290       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  86.553  86.553      3.6932       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.537   0.537     0.13178       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  21.978 112.827      4.0494       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       5       0      0   1.823   1.823     0.76833       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3kqd_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1411.7125\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -134.48 -125.20   32.43    1.34  -63.00  163.66   26.98\n",
      "    1        235R 235R N   CA   1853 1854  171.67  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   82.91   78.70   16.49    0.64   82.20  158.66    7.67\n",
      "    2        236G 236G N   CA   1864 1865 -150.15 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -66.50  -70.70    5.19    0.52  -63.50  174.17   24.36\n",
      "    3        237L 237L N   CA   1868 1869  144.65  141.60                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892  -69.21  -68.20    9.35    0.72  -62.50  164.64   27.32\n",
      "    4        240A 240A N   CA   1892 1893  154.60  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    7    9   29   63   58  105  143  184  245  333\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3kqd_MODLL.B99990001.pdb        36.66848\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3kqe\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26342    24639\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1849 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.6908\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.0605\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.7000\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24639     385\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      165\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         257.1586\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       1      3   0.026   0.026      16.707       1.000\n",
      " 2 Bond angle potential               :      94       1      5   4.123   4.123      29.538       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  45.846  45.846      13.457       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.603   1.603      1.5994       1.000\n",
      " 5 Soft-sphere overlap restraints     :     165      10     19   0.093   0.093      169.91       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.209   0.209      2.1308       1.000\n",
      "10 Distance restraints 2 (N-O)        :      42       0      0   0.428   0.428      2.8407       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      1   6.312   6.312      4.2288       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  71.752  71.752      2.7590       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  48.522  48.522     0.70243       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  17.127  17.127      1.1574       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  99.170  99.170      3.6856       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      27       0      0   0.227   0.227     0.34880E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  21.036 119.599      7.8391       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       6       0      0   1.439   1.439     0.57147       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3kqe_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1659.4313\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1895 235R 235R NE  CD   1858 1857    1.65    1.49    0.16    4.90    1.49    0.16    4.90\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4513 235R 235R CG  CD   1856 1857  126.70  107.50   19.21    5.08  107.50   19.21    5.08\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -138.00 -125.20   29.83    1.11  -63.00  168.92   27.92\n",
      "    1        235R 235R N   CA   1853 1854  167.54  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -172.57 -167.20    7.34    0.27   82.20 -167.58   13.93\n",
      "    2        236G 236G N   CA   1864 1865  169.60  174.60                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -114.66 -108.50    8.28    0.37  -63.50 -173.61   23.49\n",
      "    3        237L 237L N   CA   1868 1869  138.03  132.50                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892  -81.52  -68.20   19.45    1.22  -62.50  160.76   27.32\n",
      "    4        240A 240A N   CA   1892 1893  159.48  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   558 1858   2.267\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    2    3   11   22   42   46   75   85  137  203  282  325\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3kqe_MODLL.B99990001.pdb       257.15860\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3liw\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26437    24734\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24734     385\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       92\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          36.9492\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.17181       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.528   1.528      4.4764       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  48.355  48.355      12.366       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.856   1.856      2.2914       1.000\n",
      " 5 Soft-sphere overlap restraints     :      92       0      0   0.001   0.001     0.68641E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.088   0.088     0.27224       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.110   0.110     0.33891       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.352   4.352      2.0104       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  82.937  82.937      1.9752       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  53.464  53.464    -0.38894       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  73.237  73.237      2.4705       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  50.185  50.185      1.4032       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.495   0.495     0.20065       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  24.216 124.816      9.3319       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   0.418   0.418     0.22328E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3liw_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1488.8865\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -137.76 -125.20   23.04    0.79  -63.00  175.69   28.83\n",
      "    1        235R 235R N   CA   1853 1854  159.92  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   94.94   78.70   37.50    0.56   82.20  152.13    8.02\n",
      "    2        236G 236G N   CA   1864 1865  160.10 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -79.22  -70.70    9.18    0.71  -63.50  174.49   25.16\n",
      "    3        237L 237L N   CA   1868 1869  145.02  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -82.67  -70.20   24.48    2.01  -62.90  161.35   19.95\n",
      "    4        239K 239K N   CA   1883 1884  119.33  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -78.42  -68.20   13.92    0.88  -62.50  165.12   27.87\n",
      "    5        240A 240A N   CA   1892 1893  154.75  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    19 1412   2.265\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    1    1   11   17   40   66   96  109  171  213  256  283\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3liw_MODLL.B99990001.pdb        36.94923\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3m36\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        1\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26505    24802\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.5389\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24802     400\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       81\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          40.9369\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.29108       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.608   1.608      4.8488       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  48.306  48.306      11.366       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.564   1.564      1.5291       1.000\n",
      " 5 Soft-sphere overlap restraints     :      81       0      0   0.001   0.001     0.95962E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      48       0      0   0.161   0.161      1.2597       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.297   0.297      2.7321       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.797   3.797      1.5301       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  49.438  49.438     0.12061       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  85.851  85.851      1.2452       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 106.961 106.961      3.9765       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  80.020  80.020      2.5906       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.321   0.321     0.66485E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  26.458 134.656      8.6278       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      12       0      0   0.776   0.776     0.74277       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3m36_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1581.7699\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -90.51  -72.10   37.03    2.47  -63.00  147.46   21.82\n",
      "    1        235R 235R N   CA   1853 1854  174.03  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -67.68  -80.20   23.93    0.57   82.20 -151.31    6.64\n",
      "    2        236G 236G N   CA   1864 1865  153.70  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -102.93 -108.50   15.49    0.88  -63.50  176.31   26.89\n",
      "    3        237L 237L N   CA   1868 1869  146.95  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -72.45  -70.20    5.94    0.38  -62.90  173.57   23.07\n",
      "    4        239K 239K N   CA   1883 1884  145.90  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -78.65  -68.20   12.12    0.81  -62.50  168.43   28.42\n",
      "    5        240A 240A N   CA   1892 1893  151.45  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    6   19   47   68   88  128  167  215  252  292\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3m36_MODLL.B99990001.pdb        40.93692\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 3m37\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26337    24634\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1769 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24634     388\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       97\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          51.3129\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.22332       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.957   1.957      7.1810       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.602  47.602      13.142       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.942   2.942      6.1386       1.000\n",
      " 5 Soft-sphere overlap restraints     :      97       0      0   0.002   0.002     0.56917E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.324   0.324      3.6477       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.314   0.314      2.6212       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.934   3.934      1.6427       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  66.847  66.847     0.92633       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  68.422  68.422      1.1095       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  70.982  70.982      2.6201       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  82.453  82.453      2.3469       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      29       0      0   0.307   0.307     0.76302E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  15.521 128.092      8.6543       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       4       0      0   1.616   1.616     0.92636       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3m37_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1374.2155\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -71.03  -72.10    1.26    0.08  -63.00  177.85   24.57\n",
      "    1        235R 235R N   CA   1853 1854  141.23  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -79.48  -80.20    7.11    0.24   82.20 -133.57    7.16\n",
      "    2        236G 236G N   CA   1864 1865  167.03  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -91.81  -70.70   21.77    1.79  -63.50  174.18   25.89\n",
      "    3        237L 237L N   CA   1868 1869  146.93  141.60                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892 -131.83 -134.00    5.23    0.32  -62.50 -178.86   33.40\n",
      "    4        240A 240A N   CA   1892 1893  151.76  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    7   17   37   73   98  111  161  209  251  270\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3m37_MODLL.B99990001.pdb        51.31293\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 3q3k\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25914    24211\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1749 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "74 (of 1906 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24211     496\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      251\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         238.8174\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      3   0.027   0.027      14.647       1.000\n",
      " 2 Bond angle potential               :     102       4      6   5.926   5.926      66.090       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  56.837  56.837      21.315       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   3.397   3.397      7.2644       1.000\n",
      " 5 Soft-sphere overlap restraints     :     251       1      5   0.031   0.031      27.647       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      65       0      7   0.745   0.745      34.177       1.000\n",
      "10 Distance restraints 2 (N-O)        :      64       0      2   0.775   0.775      21.215       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       0      0   5.621   5.621      4.0987       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      1  60.812  60.812      4.0819       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  46.698  46.698      1.1157       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  55.801  55.801      2.0928       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 138.508 138.508      2.3920       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      24       0      0   0.511   0.511     0.11909       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       5      5  56.395 108.920      27.806       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      58       0      0   0.832   0.832      4.7550       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3q3k_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2155.6182\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  152.46  120.00   32.46    7.38  120.00   32.46    7.38\n",
      "    2   4477 232M 232M N   CA   1829 1830  126.41  107.00   19.41    5.58  107.00   19.41    5.58\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837   81.69   56.60  149.35   12.62  -62.90 -163.29   24.25\n",
      "    1        233K 233K N   CA   1837 1838 -174.17   38.60                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -134.31 -124.80   31.86    1.10  -63.20  160.60   25.22\n",
      "    2        234T 234T N   CA   1846 1847  173.90  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853   71.12   57.30   69.18    3.69  -63.00  134.60   23.01\n",
      "    3        235R 235R N   CA   1853 1854  -29.78   38.00                  -41.10\n",
      "    4   6833 238P 239K C   N    1881 1883 -107.93 -118.00   42.12    2.15  -62.90  146.30   21.68\n",
      "    4        239K 239K N   CA   1883 1884  180.00  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -63.20  -68.20   25.79    2.25  -62.50  148.50   24.37\n",
      "    5        240A 240A N   CA   1892 1893  170.60  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   505  808   2.029\n",
      "  serious non-bonded atom clash:  1535 1538   2.044\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    0    1    3   14   15   56   76   87  112  151  220  241  289\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3q3k_MODLL.B99990001.pdb       238.81743\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 3sw2\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25760    24057\n",
      "74 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1798 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.5201\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.7412\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24057     492\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      263\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         259.5294\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.019   0.019      7.0782       1.000\n",
      " 2 Bond angle potential               :     102       2      5   6.703   6.703      82.102       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      1  54.774  54.774      17.173       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   3.169   3.169      5.0758       1.000\n",
      " 5 Soft-sphere overlap restraints     :     263       4      5   0.050   0.050      77.852       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      65       0      0   0.429   0.429      10.725       1.000\n",
      "10 Distance restraints 2 (N-O)        :      63       0      1   0.593   0.593      14.258       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      2   9.807   9.807      12.476       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  85.335  85.335      3.2938       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  63.059  63.059      1.3407       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  86.562  86.562      3.4809       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  66.468  66.468      2.1338       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      23       0      0   0.353   0.353     0.59326E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      4  30.671 135.342      20.133       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      56       0      0   0.610   0.610      2.3480       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3sw2_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2563.1792\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4477 232M 232M N   CA   1829 1830  127.52  107.00   20.52    5.90  107.00   20.52    5.90\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837  -72.60  -70.20   32.32    2.28  -62.90  146.89   19.62\n",
      "    1        233K 233K N   CA   1837 1838  172.63  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846  -51.78  -78.10   26.45    1.66  -63.20  165.87   20.54\n",
      "    2        234T 234T N   CA   1846 1847  152.42  149.80                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -150.21 -125.20   46.46    1.60  -63.00  164.22   28.00\n",
      "    3        235R 235R N   CA   1853 1854  179.75  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -61.07  -62.40    1.74    0.33   82.20  151.29   11.49\n",
      "    4        236G 236G N   CA   1864 1865  -40.08  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -74.27  -70.70   21.99    1.54  -63.50  155.87   22.28\n",
      "    5        237L 237L N   CA   1868 1869  163.30  141.60                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883  -50.66  -70.20   24.07    1.46  -62.90  167.58   22.45\n",
      "    6        239K 239K N   CA   1883 1884  126.33  140.40                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  171.50 -134.00   54.90    2.07  -62.50 -141.36   42.48\n",
      "    7        240A 240A N   CA   1892 1893  140.42  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   339  590   2.021\n",
      "  serious non-bonded atom clash:   341  590   2.136\n",
      "  serious non-bonded atom clash:   342  590   1.707\n",
      "  serious non-bonded atom clash:   426  494   1.460\n",
      "  serious non-bonded atom clash:   427  493   1.237\n",
      "  serious non-bonded atom clash:   427  494   0.490\n",
      "  serious non-bonded atom clash:   427  495   1.954\n",
      "  serious non-bonded atom clash:   428  494   2.252\n",
      "  serious non-bonded atom clash:   504  806   2.118\n",
      "  serious non-bonded atom clash:   504  808   1.701\n",
      "  serious non-bonded atom clash:   505  805   2.000\n",
      "  serious non-bonded atom clash:   505  806   1.496\n",
      "  serious non-bonded atom clash:   505  808   0.892\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    10    2    1    1    6   14   24   60   75   96  136  164  205  224  313\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3sw2_MODLL.B99990001.pdb       259.52945\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 3tk5\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26200    24497\n",
      "74 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1796 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.2092\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24497     503\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      250\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         239.2852\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.020   0.020      8.0550       1.000\n",
      " 2 Bond angle potential               :     102       4      8   6.430   6.430      77.516       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  55.791  55.791      18.943       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      2   3.344   3.344      8.1100       1.000\n",
      " 5 Soft-sphere overlap restraints     :     250       0      3   0.026   0.026      20.403       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       0      1   0.579   0.579      20.188       1.000\n",
      "10 Distance restraints 2 (N-O)        :      67       0      2   0.718   0.718      19.009       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1  15.098  15.098      29.572       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  60.812  60.812     0.85028E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  43.924  43.924      1.3076       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  45.793  45.793      3.2139       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0   7.085   7.085      1.1144       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.262   0.262     0.37006E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       6      5  41.290 128.970      27.316       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      60       0      0   0.783   0.783      4.4150       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3tk5_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2211.5830\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  147.88  120.00   27.88    6.34  120.00   27.88    6.34\n",
      "    2   4477 232M 232M N   CA   1829 1830  133.04  107.00   26.04    7.49  107.00   26.04    7.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7074 240A 240A CA  C    1893 1895 -130.60 -180.00   49.40    9.88 -180.00   49.40    9.88\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6827 232M 233K C   N    1835 1837  -26.19  -70.20   44.14    3.14  -62.90 -178.49   25.72\n",
      "    1        233K 233K N   CA   1837 1838  136.96  140.40                  -40.80\n",
      "    2   6828 233K 234T C   N    1844 1846 -118.85 -124.80   16.85    0.56  -63.20  178.72   19.79\n",
      "    2        234T 234T N   CA   1846 1847  127.74  143.50                  -42.10\n",
      "    3   6829 234T 235R C   N    1851 1853 -154.67 -125.20   68.94    2.58  -63.00  147.83   25.91\n",
      "    3        235R 235R N   CA   1853 1854 -157.08  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -93.44  -62.40   64.21   11.24   82.20 -154.90   16.02\n",
      "    4        236G 236G N   CA   1864 1865  -97.40  -41.20                    8.50\n",
      "    5   6833 238P 239K C   N    1881 1883 -128.11 -118.00   58.56    2.69  -62.90  138.70   21.91\n",
      "    5        239K 239K N   CA   1883 1884 -163.22  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892 -101.05  -68.20   34.89    3.30  -62.50  178.66   27.65\n",
      "    6        240A 240A N   CA   1892 1893  133.56  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    1   14   18   44   72   90  109  140  196  272  324\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3tk5_MODLL.B99990001.pdb       239.28517\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 3tk6\n",
      "{'num_gaps': 1, 'gap_lengths': [8], 'gap_list': [[233, 240]], 'gap_window': [[231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26278    24575\n",
      "74 (of 1906 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.2959\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.3422\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      74\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24575     502\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      262\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         205.5081\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      76       0      1   0.013   0.013      3.8969       1.000\n",
      " 2 Bond angle potential               :     102       0      3   3.483   3.483      23.843       1.000\n",
      " 3 Stereochemical cosine torsion poten:      40       0      3  63.825  63.825      18.402       1.000\n",
      " 4 Stereochemical improper torsion pot:      23       0      1   2.837   2.837      5.0812       1.000\n",
      " 5 Soft-sphere overlap restraints     :     262       2      5   0.033   0.033      33.592       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      66       0      0   0.387   0.387      10.006       1.000\n",
      "10 Distance restraints 2 (N-O)        :      66       0      0   0.464   0.464      7.6102       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      2  18.840  18.840      46.045       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  99.026  99.026      2.9339       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  55.745  55.745     0.18994E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 110.410 110.410      4.5716       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  49.682  49.682      1.2571       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      25       0      0   0.457   0.457     0.10356       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      11       7      5  53.998 114.911      44.487       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      60       0      0   0.728   0.728      3.6585       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3tk6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2134.7100\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7074 240A 240A CA  C    1893 1895 -120.47 -180.00   59.53   11.90 -180.00   59.53   11.90\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6826 231S 232M C   N    1827 1829 -103.91  -63.40   40.53    7.58  -63.40   40.53    7.58\n",
      "    1        232M 232M N   CA   1829 1830  -41.61  -40.50                  -40.50\n",
      "    2   6827 232M 233K C   N    1835 1837   71.84   56.60  133.64   10.73  -62.90 -159.97   23.96\n",
      "    2        233K 233K N   CA   1837 1838  171.36   38.60                  -40.80\n",
      "    3   6828 233K 234T C   N    1844 1846  -83.22  -78.10   43.98    1.72  -63.20  126.01   17.49\n",
      "    3        234T 234T N   CA   1846 1847 -166.52  149.80                  -42.10\n",
      "    4   6829 234T 235R C   N    1851 1853  -67.24  -72.10    4.87    0.37  -63.00  177.31   24.24\n",
      "    4        235R 235R N   CA   1853 1854  141.64  141.90                  -41.10\n",
      "    5   6830 235R 236G C   N    1862 1864  171.00 -167.20   21.83    0.39   82.20 -170.74   13.19\n",
      "    5        236G 236G N   CA   1864 1865  175.64  174.60                    8.50\n",
      "    6   6833 238P 239K C   N    1881 1883  -84.12  -62.90   60.95    9.18  -62.90   60.95    9.18\n",
      "    6        239K 239K N   CA   1883 1884  -97.94  -40.80                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  -96.22  -68.20   72.00    6.70  -62.50  124.52   19.06\n",
      "    7        240A 240A N   CA   1892 1893   78.97  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   339  590   2.272\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    1    3    7   23   54   72   99  107  162  221  261  310\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3tk6_MODLL.B99990001.pdb       205.50807\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4a7i\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26589    24886\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24886     385\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      159\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          49.6013\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.004   0.004     0.43583       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.839   1.839      6.4511       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  54.309  54.309      17.287       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   2.751   2.751      2.5478       1.000\n",
      " 5 Soft-sphere overlap restraints     :     159       0      1   0.015   0.015      4.1146       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      43       0      0   0.213   0.213      1.1873       1.000\n",
      "10 Distance restraints 2 (N-O)        :      46       0      0   0.364   0.364      2.6532       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.201   3.201      1.0877       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  59.319  59.319     0.88441       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  86.785  86.785      2.5359       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  70.155  70.155      1.7299       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  57.784  57.784      2.5713       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      29       0      0   0.286   0.286     0.50434E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  25.282 112.089      6.0358       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       2       0      0   0.427   0.427     0.29245E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4a7i_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1366.6768\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -143.67 -125.20   48.18    2.62  -63.00  159.16   18.01\n",
      "    1        235R 235R N   CA   1853 1854   96.10  140.60                  -41.10\n",
      "    2   6831 236G 237L C   N    1866 1868 -102.50 -108.50   10.95    0.51  -63.50  169.10   21.57\n",
      "    2        237L 237L N   CA   1868 1869  123.34  132.50                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -81.82  -70.20   13.01    0.82  -62.90  173.98   23.67\n",
      "    3        239K 239K N   CA   1883 1884  146.25  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -70.89  -68.20   13.32    0.98  -62.50  160.97   26.80\n",
      "    4        240A 240A N   CA   1892 1893  158.35  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   10   17   44   90  109  116  186  191  243  311\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4a7i_MODLL.B99990001.pdb        49.60130\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4bti\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26268    24565\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.6972\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24565     387\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       93\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          51.2497\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.13203       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.841   1.841      6.2868       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.423  47.423      12.017       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.719   2.719      5.2727       1.000\n",
      " 5 Soft-sphere overlap restraints     :      93       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.257   0.257      1.7627       1.000\n",
      "10 Distance restraints 2 (N-O)        :      42       0      0   0.277   0.277      1.3261       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   4.583   4.583      2.2289       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  78.944  78.944      2.0319       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  47.738  47.738    -0.27831       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  53.637  53.637      1.9671       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  97.342  97.342      3.1012       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      32       0      0   1.043   1.043      1.0745       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      1  23.544 128.432      14.001       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       6       0      0   0.753   0.753     0.32623       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4bti_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1359.8024\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -135.00 -125.20   20.96    0.76  -63.00  175.24   28.60\n",
      "    1        235R 235R N   CA   1853 1854  159.13  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -80.61  -80.20    6.12    0.26   82.20 -132.09    7.21\n",
      "    2        236G 236G N   CA   1864 1865  167.99  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -68.02  -70.70    2.73    0.26  -63.50  176.76   24.80\n",
      "    3        237L 237L N   CA   1868 1869  142.10  141.60                  -41.20\n",
      "    4   6834 239K 240A C   N    1890 1892  -61.93  -68.20    6.28    0.54  -62.50  173.46   28.40\n",
      "    4        240A 240A N   CA   1892 1893  145.64  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    0    6   12   45   40   71  105  146  187  277  281\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4bti_MODLL.B99990001.pdb        51.24969\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4btt\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25480    23777\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1852 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.7643\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.8388\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.3487\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    23777     383\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          55.5277\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.20837       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.609   1.609      4.7337       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      1  43.733  43.733      11.106       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.764   2.764      5.5785       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.001   0.001     0.18665E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.297   0.297      2.7164       1.000\n",
      "10 Distance restraints 2 (N-O)        :      45       0      0   0.325   0.325      2.3215       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.384   2.384     0.60301       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  61.674  61.674     0.93472       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  77.397  77.397     0.67482       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  77.031  77.031      1.8646       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 103.230 103.230      2.1248       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      28       0      0   0.376   0.376     0.88764E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      2  38.121 138.317      22.554       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       1       0      0   0.021   0.021     0.36258E-04   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4btt_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1534.5254\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853  -71.83  -72.10    9.68    0.75  -63.00  167.56   23.23\n",
      "    1        235R 235R N   CA   1853 1854  151.57  141.90                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -74.68  -80.20    9.24    0.24   82.20 -137.21    6.94\n",
      "    2        236G 236G N   CA   1864 1865  166.69  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -95.78 -108.50   27.56    1.57  -63.50  165.04   24.88\n",
      "    3        237L 237L N   CA   1868 1869  156.95  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -127.41 -118.00   23.08    0.96  -62.90  171.61   26.14\n",
      "    4        239K 239K N   CA   1883 1884  160.18  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -70.72  -68.20   12.03    0.88  -62.50  162.25   27.00\n",
      "    5        240A 240A N   CA   1892 1893  157.07  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    3   31   38   45   58   89  148  232  319\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4btt_MODLL.B99990001.pdb        55.52773\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4btu\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26087    24384\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1824 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.9599\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24384     399\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       98\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          75.0564\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.19388       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.775   1.775      5.7852       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  49.228  49.228      13.845       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       1      1   3.502   3.502      9.1013       1.000\n",
      " 5 Soft-sphere overlap restraints     :      98       0      0   0.003   0.003     0.73647E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.350   0.350      4.1132       1.000\n",
      "10 Distance restraints 2 (N-O)        :      52       0      0   0.331   0.331      3.1750       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.921   3.921      1.6316       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  95.299  95.299      2.8619       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  90.280  90.280      2.5749       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0   6.297   6.297     0.67287       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  43.287  43.287      2.6755       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      33       0      0   1.007   1.007     0.96299       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       6      2  28.940 132.666      25.495       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       3       0      0   2.775   2.775      1.8952       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4btu_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1689.9879\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6574 232M 232M C   CA   1835 1830   15.42    0.00   15.42    5.43    0.00   15.42    5.43\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6826 231S 232M C   N    1827 1829  -83.46  -63.40   57.72    7.34  -63.40   57.72    7.34\n",
      "    1        232M 232M N   CA   1829 1830   13.62  -40.50                  -40.50\n",
      "    2   6829 234T 235R C   N    1851 1853  -78.01  -72.10    8.84    0.77  -63.00  177.05   22.94\n",
      "    2        235R 235R N   CA   1853 1854  135.32  141.90                  -41.10\n",
      "    3   6830 235R 236G C   N    1862 1864   64.18   78.70   20.25    0.47   82.20  161.50    8.74\n",
      "    3        236G 236G N   CA   1864 1865 -151.99 -166.10                    8.50\n",
      "    4   6831 236G 237L C   N    1866 1868 -116.20 -108.50    8.45    0.37  -63.50 -175.15   23.23\n",
      "    4        237L 237L N   CA   1868 1869  135.98  132.50                  -41.20\n",
      "    5   6833 238P 239K C   N    1881 1883 -119.25 -118.00   19.48    0.93  -62.90  170.25   25.47\n",
      "    5        239K 239K N   CA   1883 1884  158.55  139.10                  -40.80\n",
      "    6   6834 239K 240A C   N    1890 1892  -65.51  -68.20    3.96    0.25  -62.50  176.74   29.11\n",
      "    6        240A 240A N   CA   1892 1893  142.39  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    3   14   40   51   48   91  127  193  258  304\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4btu_MODLL.B99990001.pdb        75.05643\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4y6d\n",
      "{'num_gaps': 2, 'gap_lengths': [4, 7], 'gap_list': [[62, 65], [235, 241]], 'gap_window': [[60, 67], [233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     66     1   76    81      E     A    9.333\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25863    24160\n",
      "128 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.0911\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.5934\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.8482\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     128\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24160     856\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      208\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         156.9906\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     130       0      0   0.005   0.005     0.80516       1.000\n",
      " 2 Bond angle potential               :     174       0      0   1.604   1.604      8.5204       1.000\n",
      " 3 Stereochemical cosine torsion poten:      62       0      3  50.537  50.537      26.134       1.000\n",
      " 4 Stereochemical improper torsion pot:      42       0      2   2.071   2.071      5.8559       1.000\n",
      " 5 Soft-sphere overlap restraints     :     208       0      0   0.001   0.001     0.23136E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     128       2      4   1.008   1.008      37.724       1.000\n",
      "10 Distance restraints 2 (N-O)        :     133       2      3   1.209   1.209      35.141       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      18       0      0   3.765   3.765      3.0085       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  74.535  74.535      2.9510       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  48.580  48.580     0.82805       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  82.966  82.966      8.5999       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0   4.407   4.407      1.3400       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      78       0      0   0.579   0.579     0.58447       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19       7      4  33.149 110.635      22.690       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      35       0      1   1.031   1.031      2.7842       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4y6d_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2210.8845\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14337  66E 141T N   O     506 1115   13.08    9.89    3.19    4.84    9.89    3.19    4.84\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6656  61Q  62E C   N     478  480 -119.06 -117.80   22.23    1.14  -63.60  164.54   19.48\n",
      "    1         62E  62E N   CA    480  481  114.61  136.80                  -40.30\n",
      "    2   6658  63E  64G C   N     496  498 -150.31 -167.20   93.22    3.41   82.20  147.62   11.52\n",
      "    2         64G  64G N   CA    498  499   82.92  174.60                    8.50\n",
      "    3   6659  64G  65G C   N     500  502   61.92   78.70   17.07    0.92   82.20  178.89    9.69\n",
      "    3         65G  65G N   CA    502  503 -169.23 -166.10                    8.50\n",
      "    4   6829 234T 235R C   N    1851 1853  -60.52  -72.10   12.23    1.01  -63.00  173.09   23.22\n",
      "    4        235R 235R N   CA   1853 1854  145.83  141.90                  -41.10\n",
      "    5   6830 235R 236G C   N    1862 1864  -69.54  -80.20   23.90    0.56   82.20 -150.66    6.74\n",
      "    5        236G 236G N   CA   1864 1865  152.71  174.10                    8.50\n",
      "    6   6831 236G 237L C   N    1866 1868 -126.20 -108.50   20.02    0.88  -63.50 -172.30   29.94\n",
      "    6        237L 237L N   CA   1868 1869  141.88  132.50                  -41.20\n",
      "    7   6833 238P 239K C   N    1881 1883  -81.21  -70.20   11.92    0.96  -62.90  177.59   22.12\n",
      "    7        239K 239K N   CA   1883 1884  135.85  140.40                  -40.80\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    2   15   38   70   69   98  143  200  265  263\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4y6d_MODLL.B99990001.pdb       156.99063\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4y71\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26212    24509\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1836 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.9411\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24509     365\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      107\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          38.6448\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.77039E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.629   1.629      4.7685       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  47.462  47.462      11.804       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   1.969   1.969      2.6286       1.000\n",
      " 5 Soft-sphere overlap restraints     :     107       0      0   0.000   0.000     0.12106E-03   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      41       0      0   0.203   0.203      1.4920       1.000\n",
      "10 Distance restraints 2 (N-O)        :      42       0      0   0.243   0.243      1.0058       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.164   3.164      1.0628       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  76.881  76.881     0.80874       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  70.431  70.431     0.26056       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  58.248  58.248      1.8028       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 112.945 112.945      3.0310       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      17       0      0   0.463   0.463     0.10640       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  22.466 139.131      9.7967       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4y71_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1476.0525\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -110.12 -125.20   15.32    0.55  -63.00 -174.88   22.23\n",
      "    1        235R 235R N   CA   1853 1854  137.93  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -79.50  -80.20    4.23    0.13   82.20 -131.51    7.15\n",
      "    2        236G 236G N   CA   1864 1865  169.93  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -93.29 -108.50   17.90    0.79  -63.50  166.95   21.66\n",
      "    3        237L 237L N   CA   1868 1869  123.07  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -83.59  -70.20   17.06    1.03  -62.90  169.50   23.20\n",
      "    4        239K 239K N   CA   1883 1884  150.97  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -71.01  -68.20    8.57    0.59  -62.50  165.93   27.62\n",
      "    5        240A 240A N   CA   1892 1893  153.40  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    5   12   24   72   66   93  137  197  274  272\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4y71_MODLL.B99990001.pdb        38.64484\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4y76\n",
      "{'num_gaps': 2, 'gap_lengths': [2, 7], 'gap_list': [[46, 47], [235, 241]], 'gap_window': [[44, 49], [233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25656    23953\n",
      "124 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     124\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    23953     972\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      208\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          93.8136\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     127       0      0   0.003   0.003     0.41566       1.000\n",
      " 2 Bond angle potential               :     170       0      0   1.718   1.718      9.7574       1.000\n",
      " 3 Stereochemical cosine torsion poten:      72       0      2  49.879  49.879      25.480       1.000\n",
      " 4 Stereochemical improper torsion pot:      46       0      1   2.068   2.068      5.4565       1.000\n",
      " 5 Soft-sphere overlap restraints     :     208       0      0   0.002   0.002     0.71584E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     164       0      0   0.164   0.164      2.9229       1.000\n",
      "10 Distance restraints 2 (N-O)        :     138       0      0   0.501   0.501      7.3757       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      16       0      0   4.542   4.542      3.2217       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      1  66.251  66.251      6.2635       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  86.905  86.905      3.1005       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  93.239  93.239      5.0209       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       6       0      0  42.649  42.649      3.3463       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     101       0      0   0.626   0.626     0.81596       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17       7      5  32.084 103.995      17.380       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      84       0      0   0.608   0.608      3.1847       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4y76_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2627.3074\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6640  45Y  46Q C   N     343  345 -145.58 -121.10   88.83    3.76  -63.80  125.06   22.32\n",
      "    1         46Q  46Q N   CA    345  346 -134.91  139.70                  -40.30\n",
      "    2   6641  46Q  47A C   N     352  354  -60.34  -68.20   24.99    1.74  -62.50  162.49   26.74\n",
      "    2         47A  47A N   CA    354  355  121.58  145.30                  -40.90\n",
      "    3   6829 234T 235R C   N    1851 1853 -120.30 -125.20   37.01    1.88  -63.00  152.78   24.57\n",
      "    3        235R 235R N   CA   1853 1854  177.28  140.60                  -41.10\n",
      "    4   6830 235R 236G C   N    1862 1864  -60.18  -62.40    2.90    0.42   82.20  151.43   11.56\n",
      "    4        236G 236G N   CA   1864 1865  -43.06  -41.20                    8.50\n",
      "    5   6831 236G 237L C   N    1866 1868  -76.44  -70.70   10.19    0.63  -63.50  169.27   24.26\n",
      "    5        237L 237L N   CA   1868 1869  150.03  141.60                  -41.20\n",
      "    6   6833 238P 239K C   N    1881 1883  -77.76  -70.20   13.50    0.82  -62.90  168.28   22.69\n",
      "    6        239K 239K N   CA   1883 1884  151.58  140.40                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  -64.23  -68.20    4.36    0.42  -62.50  172.02   28.28\n",
      "    7        240A 240A N   CA   1892 1893  147.09  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    3    6   31   51   84  101  128  171  259  264\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4y76_MODLL.B99990001.pdb        93.81358\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4y79\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25807    24104\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.9634\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.3201\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24104     370\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      160\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         213.4046\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      2   0.015   0.015      6.5601       1.000\n",
      " 2 Bond angle potential               :      94       0      2   3.687   3.687      23.123       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  49.946  49.946      18.504       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.615   2.615      3.9299       1.000\n",
      " 5 Soft-sphere overlap restraints     :     160       9     15   0.086   0.086      138.04       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      42       0      0   0.252   0.252      1.5184       1.000\n",
      "10 Distance restraints 2 (N-O)        :      45       0      0   0.215   0.215      1.0985       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   5.435   5.435      3.1347       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  86.197  86.197      1.7408       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  64.274  64.274     0.60504       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  13.333  13.333      1.2833       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  85.722  85.722      3.1431       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      16       0      0   0.330   0.330     0.43503E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      3  28.464 117.797      10.657       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       2       0      0   0.342   0.342     0.19010E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4y79_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1814.7489\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -178.32 -125.20   62.02    1.84  -63.00 -173.73   32.63\n",
      "    1        235R 235R N   CA   1853 1854  172.62  140.60                  -41.10\n",
      "    2   6831 236G 237L C   N    1866 1868  -65.09  -70.70    6.33    0.46  -63.50  179.87   24.88\n",
      "    2        237L 237L N   CA   1868 1869  138.66  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883 -114.51 -118.00    6.06    0.23  -62.90 -177.60   21.44\n",
      "    3        239K 239K N   CA   1883 1884  134.15  139.10                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -86.91  -68.20   37.26    2.41  -62.50  143.67   24.81\n",
      "    4        240A 240A N   CA   1892 1893  177.52  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    1   10   11   34   57   75  113  122  188  249  295\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4y79_MODLL.B99990001.pdb       213.40456\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4y7a\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25767    24064\n",
      "70 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24064     364\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      107\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          34.7819\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.003   0.003     0.13790       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.661   1.661      5.0842       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  43.177  43.177      9.3119       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   1.989   1.989      2.5450       1.000\n",
      " 5 Soft-sphere overlap restraints     :     107       0      0   0.001   0.001     0.87681E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      40       0      0   0.236   0.236      1.9047       1.000\n",
      "10 Distance restraints 2 (N-O)        :      42       0      0   0.225   0.225      1.3590       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.864   2.864     0.87079       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  48.798  48.798     0.95491       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  85.107  85.107      1.2405       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  57.869  57.869      1.9975       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0   4.222   4.222      1.3286       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      17       0      0   0.389   0.389     0.77623E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  17.944 115.608      7.9604       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4y7a_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1339.8361\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6830 235R 236G C   N    1862 1864 -171.59 -167.20   32.76    1.33   82.20  170.70   12.75\n",
      "    1        236G 236G N   CA   1864 1865  142.14  174.60                    8.50\n",
      "    2   6831 236G 237L C   N    1866 1868  -75.94  -70.70    5.27    0.49  -63.50  178.13   25.46\n",
      "    2        237L 237L N   CA   1868 1869  141.11  141.60                  -41.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -67.95  -70.20    2.25    0.16  -62.90  178.99   23.52\n",
      "    3        239K 239K N   CA   1883 1884  140.28  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -76.59  -68.20   23.17    1.58  -62.50  152.86   25.77\n",
      "    4        240A 240A N   CA   1892 1893  166.90  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    0    2    9   24   47   90   97  115  188  236  275\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4y7a_MODLL.B99990001.pdb        34.78190\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4y7b\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25795    24092\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.6486\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24092     371\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       93\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          43.6502\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.11903       1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.678   1.678      5.2795       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  45.627  45.627      12.666       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   2.257   2.257      3.2976       1.000\n",
      " 5 Soft-sphere overlap restraints     :      93       0      0   0.001   0.001     0.15942E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      44       0      0   0.141   0.141     0.74846       1.000\n",
      "10 Distance restraints 2 (N-O)        :      46       0      0   0.353   0.353      2.2305       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.679   3.679      1.4369       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  62.519  62.519      1.9532       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  67.056  67.056     0.46829       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      1  37.575  37.575      2.4680       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  64.390  64.390      2.7538       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      16       0      0   0.277   0.277     0.34700E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  21.648 131.117      10.178       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4y7b_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1489.3455\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -139.44 -125.20   18.38    0.55  -63.00 -176.63   29.99\n",
      "    1        235R 235R N   CA   1853 1854  152.22  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864   83.70   78.70   17.89    0.32   82.20  168.22    8.24\n",
      "    2        236G 236G N   CA   1864 1865  176.72 -166.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -104.52 -108.50   21.78    1.21  -63.50  169.91   26.11\n",
      "    3        237L 237L N   CA   1868 1869  153.92  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883  -58.63  -70.20   14.80    1.24  -62.90  169.63   21.80\n",
      "    4        239K 239K N   CA   1883 1884  149.63  140.40                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -62.79  -68.20    8.31    0.52  -62.50  179.89   29.47\n",
      "    5        240A 240A N   CA   1892 1893  139.00  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   10   15   61   76   92  128  176  248  277\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4y7b_MODLL.B99990001.pdb        43.65023\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4zh8\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    25635    23932\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1850 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.5463\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    23932     374\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      158\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         287.5354\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      2   0.018   0.018      9.6230       1.000\n",
      " 2 Bond angle potential               :      94       1      2   3.725   3.725      23.963       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  50.649  50.649      18.789       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   2.423   2.423      2.6164       1.000\n",
      " 5 Soft-sphere overlap restraints     :     158      11     17   0.103   0.103      198.84       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      45       0      0   0.162   0.162     0.90861       1.000\n",
      "10 Distance restraints 2 (N-O)        :      47       0      0   0.157   0.157     0.66734       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      2   7.449   7.449      5.8885       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  71.635  71.635      3.1443       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  56.904  56.904     0.82333       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  53.866  53.866      3.7396       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  93.362  93.362      3.4336       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      17       0      0   0.806   0.806     0.31441       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  29.740 132.907      14.782       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4zh8_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1949.1107\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4514 235R 235R CD  NE   1857 1858  142.19  120.00   22.19    5.63  120.00   22.19    5.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -143.59 -125.20   40.71    1.49  -63.00  163.26   27.48\n",
      "    1        235R 235R N   CA   1853 1854  176.92  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864 -113.34  -80.20   38.74    2.85   82.20 -140.39   17.00\n",
      "    2        236G 236G N   CA   1864 1865  154.04  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868  -74.63  -70.70   23.46    1.63  -63.50  154.47   22.10\n",
      "    3        237L 237L N   CA   1868 1869  164.73  141.60                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883 -109.79 -118.00   25.83    1.12  -62.90  162.33   19.06\n",
      "    4        239K 239K N   CA   1883 1884  114.61  139.10                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -91.93  -68.20   28.82    2.83  -62.50  172.37   26.96\n",
      "    5        240A 240A N   CA   1892 1893  128.94  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   559 1858   2.225\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    2    2    5   14   26   47   83   94  123  185  233  269\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4zh8_MODLL.B99990001.pdb       287.53540\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 4zha\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26132    24429\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1853 atoms and 1 coordinate set(s) were parsed in 0.01s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.3820\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.8855\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24429     376\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      168\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         333.0228\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      1   0.018   0.018      9.5387       1.000\n",
      " 2 Bond angle potential               :      94       1      3   3.889   3.889      26.008       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      0  48.586  48.586      14.173       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      1   3.202   3.202      5.3349       1.000\n",
      " 5 Soft-sphere overlap restraints     :     168      10     22   0.114   0.114      257.83       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      46       0      0   0.157   0.157     0.88908       1.000\n",
      "10 Distance restraints 2 (N-O)        :      48       0      0   0.386   0.386      3.3332       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   3.033   3.033     0.97615       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  44.361  44.361      1.3878       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0 105.449 105.449      1.6105       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  49.283  49.283      1.9402       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  96.566  96.566      2.9189       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      17       0      0   0.353   0.353     0.63223E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       4      2  24.288 131.433      7.0182       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4zha_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1783.9524\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4514 235R 235R CD  NE   1857 1858  146.60  120.00   26.60    6.75  120.00   26.60    6.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6831 236G 237L C   N    1866 1868  -63.99  -70.70   10.55    0.66  -63.50  174.66   24.22\n",
      "    1        237L 237L N   CA   1868 1869  133.46  141.60                  -41.20\n",
      "    2   6832 237L 238P C   N    1874 1876  -55.06  -58.70    3.74    0.51  -64.50  177.08   12.72\n",
      "    2        238P 238P N   CA   1876 1877  -29.63  -30.50                  147.20\n",
      "    3   6833 238P 239K C   N    1881 1883  -73.06  -70.20    3.28    0.27  -62.90  179.88   22.82\n",
      "    3        239K 239K N   CA   1883 1884  138.79  140.40                  -40.80\n",
      "    4   6834 239K 240A C   N    1890 1892  -72.87  -68.20    6.95    0.68  -62.50  179.25   29.89\n",
      "    4        240A 240A N   CA   1892 1893  140.15  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   566 1858   2.209\n",
      "  serious non-bonded atom clash:   736 1858   2.287\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    1    5    6   18   38   56   99   92  144  197  241  300\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4zha_MODLL.B99990001.pdb       333.02277\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHEKGRQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDSGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMKT\n",
      "Modelando proteína 5k0h\n",
      "{'num_gaps': 1, 'gap_lengths': [7], 'gap_list': [[235, 241]], 'gap_window': [[233, 241]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26343    24640\n",
      "70 (of 1906 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1831 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.7997\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906      70\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24640     369\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       99\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          35.7136\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      71       0      0   0.002   0.002     0.96169E-01   1.000\n",
      " 2 Bond angle potential               :      94       0      0   1.551   1.551      4.4194       1.000\n",
      " 3 Stereochemical cosine torsion poten:      39       0      2  54.280  54.280      15.533       1.000\n",
      " 4 Stereochemical improper torsion pot:      21       0      0   1.179   1.179     0.65014       1.000\n",
      " 5 Soft-sphere overlap restraints     :      99       0      0   0.001   0.001     0.23654E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      38       0      0   0.176   0.176     0.87887       1.000\n",
      "10 Distance restraints 2 (N-O)        :      37       0      0   0.162   0.162     0.48756       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       9       0      0   2.472   2.472     0.64870       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  77.976  77.976      1.1744       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  47.305  47.305    -0.31691       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0   8.555   8.555     0.62800       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  77.192  77.192      2.1812       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      29       0      0   0.252   0.252     0.30119E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       9       5      1  20.306 132.311      9.2789       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5k0h_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1457.4452\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6829 234T 235R C   N    1851 1853 -121.86 -125.20    8.93    0.49  -63.00  179.93   28.37\n",
      "    1        235R 235R N   CA   1853 1854  148.87  140.60                  -41.10\n",
      "    2   6830 235R 236G C   N    1862 1864  -65.46  -80.20   43.55    1.08   82.20 -166.78    6.68\n",
      "    2        236G 236G N   CA   1864 1865  133.12  174.10                    8.50\n",
      "    3   6831 236G 237L C   N    1866 1868 -117.76 -108.50   16.75    0.79  -63.50 -179.32   28.44\n",
      "    3        237L 237L N   CA   1868 1869  146.46  132.50                  -41.20\n",
      "    4   6833 238P 239K C   N    1881 1883   50.84   56.60    6.61    0.56  -62.90  140.60   24.41\n",
      "    4        239K 239K N   CA   1883 1884   41.85   38.60                  -40.80\n",
      "    5   6834 239K 240A C   N    1890 1892  -63.96  -68.20    5.44    0.35  -62.50  177.22   29.11\n",
      "    5        240A 240A N   CA   1892 1893  141.89  145.30                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2    5   12   40   65  107  126  158  222  236  278\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5k0h_MODLL.B99990001.pdb        35.71359\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHERQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDAGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 5voe\n",
      "{'num_gaps': 2, 'gap_lengths': [2, 8], 'gap_list': [[136, 137], [233, 240]], 'gap_window': [[134, 139], [231, 240]]}\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26282    24579\n",
      "126 (of 1906 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.4807\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.7796\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     126\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24579    1129\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      457\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         491.5378\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     130       0      0   0.017   0.017      10.831       1.000\n",
      " 2 Bond angle potential               :     174       3      9   5.312   5.312      87.769       1.000\n",
      " 3 Stereochemical cosine torsion poten:      68       0      5  58.493  58.493      33.665       1.000\n",
      " 4 Stereochemical improper torsion pot:      45       0      0   1.726   1.726      3.3747       1.000\n",
      " 5 Soft-sphere overlap restraints     :     457       2      8   0.029   0.029      46.383       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     196       7     16   0.762   0.762      121.86       1.000\n",
      "10 Distance restraints 2 (N-O)        :     200       3     12   0.706   0.706      74.491       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      18       2      3  11.428  11.428      27.723       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      13       0      0  94.406  94.406      8.9389       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      1  83.317  83.317      6.5459       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  88.812  88.812      6.9536       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0  64.528  64.528      3.2550       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     107       0      0   0.677   0.677      1.1273       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19       7      7  41.919 124.745      46.313       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     134       0      0   1.731   1.731      12.307       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5voe_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3902.5801\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4466 230R 231S C   N    1821 1823  142.32  120.00   22.32    5.07  120.00   22.32    5.07\n",
      "    2   4469 231S 231S N   CA   1823 1824  126.14  107.00   19.14    5.50  107.00   19.14    5.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6730 135E 136K C   N    1068 1070 -101.42 -118.00   32.89    1.30  -62.90  156.32   18.55\n",
      "    1        136K 136K N   CA   1070 1071  110.70  139.10                  -40.80\n",
      "    2   6732 137G 138R C   N    1081 1083 -123.71 -125.20   29.31    1.45   57.30 -137.68   27.05\n",
      "    2        138R 138R N   CA   1083 1084  169.87  140.60                   38.00\n",
      "    3   6826 231S 232M C   N    1827 1829  -89.18  -63.40   34.65    6.97  -63.40   34.65    6.97\n",
      "    3        232M 232M N   CA   1829 1830  -63.65  -40.50                  -40.50\n",
      "    4   6827 232M 233K C   N    1835 1837   86.89   56.60   99.00    9.43  -62.90 -130.67   38.47\n",
      "    4        233K 233K N   CA   1837 1838  132.86   38.60                  -40.80\n",
      "    5   6828 233K 234T C   N    1844 1846 -122.79 -124.80   64.35    2.70  -63.20  135.13   14.40\n",
      "    5        234T 234T N   CA   1846 1847   79.18  143.50                  -42.10\n",
      "    6   6833 238P 239K C   N    1881 1883 -136.65 -118.00   27.87    1.42  -62.90  175.45   20.18\n",
      "    6        239K 239K N   CA   1883 1884  118.39  139.10                  -40.80\n",
      "    7   6834 239K 240A C   N    1890 1892  154.11 -134.00   72.19    2.65  -62.50 -130.88   45.03\n",
      "    7        240A 240A N   CA   1892 1893  140.40  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 1831 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    8   20   57   64   94  124  158  218  287  293\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5voe_MODLL.B99990001.pdb       491.53775\n",
      "\n",
      "IVGGQECKDGECPWQALLINEENEGFCGGTILSEFYILTAAHCLYQAKRFKVRVGDRNTEQEEGGEAVHEVEVVIKHNRFTKETYDFDIAVLRLKTPITFRMNVAPACLPERDWAESTLMTQKTGIVSGFGRTHERQSTRLKMLEVPYVDRNSCKLSSSFIITQNMFCAGYDTKQEDACQGDAGGPHVTRFKDTYFVTGIVSWGEGCARKGKYGIYTKVTAFLKWIDRSMK\n",
      "Modelando proteína 5vof\n",
      "{'num_gaps': 2, 'gap_lengths': [2, 8], 'gap_list': [[136, 137], [233, 240]], 'gap_window': [[134, 139], [231, 240]]}\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "patch_s_522_> Number of disulfides patched in MODEL:        4\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "patch___481W> This disulfide was already patched.\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    26298    24595\n",
      "126 (of 1906 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1906     126\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    24595    1121\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      377\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         315.1132\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     130       0      1   0.015   0.015      7.4021       1.000\n",
      " 2 Bond angle potential               :     174       3      5   4.091   4.091      51.709       1.000\n",
      " 3 Stereochemical cosine torsion poten:      68       0      4  59.158  59.158      36.098       1.000\n",
      " 4 Stereochemical improper torsion pot:      45       1      2   2.792   2.792      11.451       1.000\n",
      " 5 Soft-sphere overlap restraints     :     377       0      4   0.024   0.024      24.572       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     194       0      6   0.461   0.461      36.085       1.000\n",
      "10 Distance restraints 2 (N-O)        :     200       0      2   0.542   0.542      29.915       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      18       1      1  16.880  16.880      60.484       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      13       0      0  76.617  76.617      4.7411       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  70.179  70.179      3.8572       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  86.736  86.736      7.5866       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0  99.915  99.915      2.8367       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     101       0      0   0.442   0.442     0.37112       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      10      5  35.625 135.307      28.764       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     134       0      0   1.656   1.656      9.2416       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5vof_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3822.6851\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4474 231S 232M C   N    1827 1829  149.76  120.00   29.76    6.76  120.00   29.76    6.76\n",
      "    2   4477 232M 232M N   CA   1829 1830  125.06  107.00   18.06    5.19  107.00   18.06    5.19\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7074 240A 240A CA  C    1893 1895 -110.84 -180.00   69.16   13.83 -180.00   69.16   13.83\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6729 134H 135E C   N    1059 1061  -59.91  -63.60    5.47    0.92  -69.30  179.01   13.33\n",
      "    1        135E 135E N   CA   1061 1062  -36.26  -40.30                  142.50\n",
      "    2   6730 135E 136K C   N    1068 1070  -59.41  -70.20   25.38    2.06  -62.90  155.87   20.05\n",
      "    2        136K 136K N   CA   1070 1071  163.37  140.40                  -40.80\n",
      "    3   6732 137G 138R C   N    1081 1083 -177.76 -125.20   64.85    1.91   57.30 -171.92   21.51\n",
      "    3        138R 138R N   CA   1083 1084  178.58  140.60                   38.00\n",
      "    4   6827 232M 233K C   N    1835 1837  -55.59  -70.20   14.78    1.03  -62.90  179.13   23.67\n",
      "    4        233K 233K N   CA   1837 1838  138.18  140.40                  -40.80\n",
      "    5   6828 233K 234T C   N    1844 1846 -107.88 -124.80   19.39    0.75  -63.20 -178.28   20.67\n",
      "    5        234T 234T N   CA   1846 1847  134.04  143.50                  -42.10\n",
      "    6   6829 234T 235R C   N    1851 1853 -142.02 -125.20   19.71    0.59  -63.00 -174.31   30.48\n",
      "    6        235R 235R N   CA   1853 1854  150.86  140.60                  -41.10\n",
      "    7   6830 235R 236G C   N    1862 1864   62.21   78.70   36.35    0.54   82.20  143.61    7.96\n",
      "    7        236G 236G N   CA   1864 1865 -133.71 -166.10                    8.50\n",
      "    8   6831 236G 237L C   N    1866 1868  -68.39  -70.70   15.98    1.30  -63.50  161.46   22.70\n",
      "    8        237L 237L N   CA   1868 1869  157.42  141.60                  -41.20\n",
      "    9   6833 238P 239K C   N    1881 1883 -176.12 -118.00   75.11    2.57  -62.90  174.31   29.21\n",
      "    9        239K 239K N   CA   1883 1884 -173.33  139.10                  -40.80\n",
      "   10   6834 239K 240A C   N    1890 1892 -161.20 -134.00   27.52    0.76  -62.50 -165.22   37.18\n",
      "   10        240A 240A N   CA   1892 1893  151.18  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   314 1458   2.153\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    1    2   13   12   46   67   86  114  137  242  289  302\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5vof_MODLL.B99990001.pdb       315.11316\n",
      "\n",
      "1562.8788454532623\n"
     ]
    }
   ],
   "source": [
    "import time\n",
    "start = time.time()\n",
    "\n",
    "# se lleva a cabo el modelado de todas las estructuras cristalográficas\n",
    "for pdb_code in pdbids_list:\n",
    "    run_modeller(pdb_code,\n",
    "                 seq_prot,\n",
    "                 num_res_window = 2, \n",
    "                 force_modeling = False,\n",
    "                 max_var_iterations = 1000, \n",
    "                 repeat_optimization = 2)\n",
    "\n",
    "end = time. time()\n",
    "print(end - start)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3.7.6 64-bit ('mds': conda)",
   "language": "python",
   "name": "python37664bitmdsconda8fcfe9b6c3484566880235af88776fef"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
